<html><h1><a href=https://www.ncbi.nlm.nih.gov/pubmed/27423261>Retention data of bile  and their oxo derivatives in characterization of pharmacokinetic properties and in silico ADME modeling.</a></h1><p>Information on ADME properties of examined bile  and their oxo derivatives are scarce, although the interest for bile  and their use in nanochemistry and macromolecular chemistry is increasing. The purpose of this research was to evaluate the lipophilicity, a crucial physicochemical parameter for describing ADME properties of selected bile  and their oxo derivatives, and to compare two approaches: experimentally determined hydrophobicity parameters and calculated logP values.Commercially available bile  - deoxycholic, chenodeoxycholic, hyodeoxycholic and ursodeoxycholic  were used to synthesize oxo derivatives. Lipophilicity was evaluated in two solvent systems: toluene/ethanol and toluene/butanol. Retention parameters were acquired by normal-phase TLC. The correlations between calculated logP values obtained using five different software and experimentally determined hydrophobicity parameters (RM(0)(tol/eth), RM(0)(tol/but), b(tol/eth) and b(tol/but)) were examined.Correlation analysis confirmed significant dependence between experimental RM(0) values and software calculated parameters. Results suggest satisfactory intestinal absorption after oral administration for all of the examined compounds as well as low volumes of distribution, and high affinity for binding with plasma proteins. Penetration through blood-brain barrier and skin is not satisfactory. All of the examined compounds show high affinity for binding with G-protein coupled receptors and consequently inhibition of ionic channels. Results also suggest possible binding with nuclear receptors.Established lipophilicity testing model of studied compounds showed excellent predictive ability and might represent significant tool in development of relations between chromatographic behavior and ADME properties. Compounds 3α-hydroxy-7,12-dioxo-5β-cholanoic and 12α-hydroxy-3,7-dioxo-5β-cholanoic  might be the most suitable candidates for further development studies (satisfactory pharmacokinetic properties and lowest haemolytic potential) followed by 3α-hydroxy-12-oxo-5β-cholanoic  and 3α-hydroxy-7-oxo-5β-cholanoic  (slightly higher haemolytic potential, but better ligand properties).Copyright © 2016 Elsevier B.V. All rights reserved.</p><h1><a href=https://www.ncbi.nlm.nih.gov/pubmed/24810629>Identification of 9α-hydroxy-17-oxo-1,2,3,4,10,19-hexanorandrost-6-en-5-oic  and β-oxidation products of the C-17 side chain in cholic  degradation by Comamonas testosteroni TA441.</a></h1><p>Comamonas testosteroni degrades testosterone into 9,17-dioxo-1,2,3,4,10,19-hexanorandrostan-5-oic  and 2-hydroxyhexa-2,4-dienoic  via aromatization of the A-ring. The former compound is suggested to be degraded further by β-oxidation, but the details of the process remain unclear. In this study, we identified 9α-hydroxy-17-oxo-1,2,3,4,10,19-hexanorandrost-6-en-5-oic  as an intermediate compound in the β-oxidation of this compound. ORF32, located in one of the two main steroid degradation gene clusters, was shown to be indispensable for the conversion of this compound. A homology search indicated that ORF32 encodes a hydratase for the CoA-ester, suggesting that ORF32 encodes a hydratase that adds a water molecule to a double bond at C-6 of the CoA-ester of 9α-hydroxy-17-oxo-1,2,3,4,10,19-hexanorandrost-6-en-5-oic . From the culture of an ORF32-disrupted mutant incubated with cholic  for a short period (around two days, when a considerable number of intermediate compounds were detected by HPLC), 7α,12α-dihydroxy-3-oxochola-1,4-dien-24-oic , 7α,12α-dihydroxy-3-oxochol-4-en-24-oic , 12α-hydroxy-3-oxochola-4,6-dien-24-oic , 7α,12α-dihydroxy-3-oxopregna-1,4-diene-20-carboxylic , 12α-hydroxy-3-oxopregna-4,6-diene-20-carboxylic , 7α,12α-dihydroxy-3-oxopregn-4-ene-20-carboxylic , 12α-hydroxy-3-oxopregna-4,6-diene-20-carboxylic , 7α-hydroxy-3-oxopregna-4,17(20)-diene-20-carboxylic , and 3-oxopregna-4,6,17(20)-triene-20-carboxylic  were isolated as intermediate compounds of C-17 side-chain degradation. The presence of these compounds implies that the process of degradation of the C-17 side chain in C. testosteroni will be similar to the process in Pseudomonas. The final two compounds, which have a double bond at the C-17(20) position, are here identified for the first time, to the best of our knowledge, as intermediate compounds in bacterial steroid degradation; their composition suggests that the remaining three carbons at the C-17 position would be removed oxidatively as a propionic  derivative.Copyright © 2014 Elsevier Ltd. All rights reserved.</p><h1><a href=https://www.ncbi.nlm.nih.gov/pubmed/12506972>Fusicoccins P and Q, and 3-epifusicoccins H and Q, new polar fusicoccins from isolate Niigata 2-A of a peach Fusicoccum canker fungus.</a></h1><p>Our search for new polar fusicoccins biosynthetically related to fusicoccin A from the culture filtrate of isolate Niigata 2-A of a peach Fusicoccum canker fungus resulted in the isolation of new fusicoccins named fusicoccins P and Q, and 3-epifusicoccins H and Q, together with 3'-deacetylfusicoccin A and 16-O-demethyl-3-epifusicoccin J. The structures of fusicoccins P and Q, and of 3-epifusicoccin Q were determined to be those of deisopentenylfusicoccin J, 12alpha-hydroxyfusicoccin H and -epifusicoccin H, respectively, by NMR spectrometry and chemical derivation from known fusicoccins. 3-Epifusicoccin H was identified by comparing its 400 MHz NMR spectra with those of fusicoccin H. The lettuce seed germination-stimulating activity of these new fusicoccins was examined in the presence of ABA: fusicoccin P was highly active, while 3-epifusicoccins H and Q were slightly active, and fusicoccins H and Q were almost inactive. Possible biosynthetic  incorporating these new fusicoccins and 3-epifusicoccins from geranylgeranyl diphosphate to 3'-deacetlyfusicoccin A and 16-O-demethyl-3-epifusicoccin J are discussed.</p><h1><a href=https://www.ncbi.nlm.nih.gov/pubmed/21551335>Hasllibacter halocynthiae gen. nov., sp. nov., a nutriacholic -producing bacterium isolated from the marine ascidian Halocynthia roretzi.</a></h1><p>A Gram-negative, aerobic, ovoid to rod-shaped bacterial strain, KME 002(T) was isolated from a marine ascidian, Halocynthia roretzi, off the coast of Gangneung, Korea. Phylogenetic analyses based on 16S rRNA gene sequences showed that this strain belonged to the family Rhodobacteraceae in the class Alphaproteobacteria and was closely related to the type strains of Dinoroseobacter shibae, Roseovarius crassostreae and Pseudoruegeria aquimaris with 95.0, 94.7 and 94.5% 16S rRNA gene sequence similarities, respectively. KME 002(T) was an obligately halophilic bacterium requiring 1 to 5% (w/v) NaCl, with an absolute requirement for magnesium chloride for growth. Cells were motile by means of a single polar flagellum and showed budding fission. The predominant cellular fatty  of the isolate was C(18:1)ω7c and Q-10 was the major ubiquinone. The DNA G+C content of the strain was 71.6 mol%. The major secondary metabolites from cultures in liquid medium were cholic  derivatives, including 3α,12α-hydroxy-3-keto-glycocholanic , 12-hydroxy-3-keto-glycocholanic , nutriacholic  and deoxycholic . These characteristics determined in this polyphasic study suggest that strain KME 002(T) represents a novel species in a new genus of the family Rhodobacteraceae. The name Hasllibacter halocynthiae gen. nov., sp. nov. is proposed for this isolate, and the type strain is KME 002(T) (=JCM 16214(T)=KCCM 90082(T)).</p><h1><a href=https://www.ncbi.nlm.nih.gov/pubmed/22520047>Metabolite profiling of plasma and urine from rats with TNBS-induced acute colitis using UPLC-ESI-QTOF-MS-based metabonomics--a pilot study.</a></h1><p>The incidence of inflammatory bowel disease, a relapsing intestinal condition whose precise etiology is still unclear, has continually increased over recent years.  profiling is an effective method with high sample throughput that can detect and identify potential biomarkers, and thus may be useful in investigating the pathogenesis of inflammatory bowel disease. In this study, using a metabonomics approach, a pilot study based on ultra-performance liquid chromatography coupled with electrospray ionization quadrupole time-of-flight mass spectrometry (UPLC-ESI-QTOF-MS) was performed to characterize the  profile of plasma and urine samples of rats with experimental colitis induced by 2,4,6-trinitrobenzene sulfonic . Acquired  profile data were processed by multivariate data analysis for differentiation and screening of potential biomarkers. Five metabolites were identified in urine: two tryptophan metabolites [4-(2-aminophenyl)-2,4-dioxobutanoic  and 4,6-cihydroxyquinoline], two gut microbial metabolites (phenyl-acetylglycine and p-cresol glucuronide), and the bile  12α-hydroxy-3-oxocholadienic . Seven metabolites were identified in plasma: three members of the bile /alcohol group (cholic , 12α-hydroxy-3-oxocholadienic  and cholestane-3,7,12,24,25-pentol) and four lysophosphatidylcholines [LysoPC(20:4), LysoPC(16:0), LysoPC(18:1) and LysoPC(18:0)]. These metabolites are associated with damage of the intestinal barrier function, microbiota homeostasis, immune modulation and the inflammatory response, and play important roles in the pathogenesis of inflammatory bowel disease. Our results positively support application of the metabonomic approach in study of the pathophysiological mechanism of inflammatory bowel disease.© 2012 The Authors Journal compilation © 2012 FEBS.</p><h1><a href=https://www.ncbi.nlm.nih.gov/pubmed/29907951>Analysis of human C24 bile  metabolome in serum and urine based on enzyme digestion of conjugated bile  and LC-MS determination of unconjugated bile .</a></h1><p>Host-gut microbiota  interactions are closely associated with health and disease. A manifestation of such co- is the vast structural diversity of bile  (BAs) involving both oxidative stereochemistry and conjugation. Herein, we describe the development and validation of a LC-MS-based method for the analysis of human C24 BA metabolome in serum and urine. The method has high throughput covering the discrimination of oxidative stereochemistry of unconjugated species in a 15-min analytical cycle. The validated quantitative performance provided an indirect way to ascertain the conjugation patterns of BAs via enzyme-digestion protocols that incorporated the enzymes, sulfatase, β-glucuronidase, and choloylglycine hydrolase. Application of the method has led to the detection of at least 70 unconjugated BAs including 27 known species and 43 newly found species in the post-prandial serum and urine samples from 7 nonalcoholic steatohepatitis patients and 13 healthy volunteers. Newly identified unconjugated BAs included 3α, 12β-dihydroxy-5β-cholan-24-oic , 12α-hydroxy-3-oxo-5β-cholan-24-oic , and 3α, 7α, 12β-trihydroxy-5β-cholan-24-oic . High-definition negative fragment spectra of the other major unknown species were acquired to facilitate future identification endeavors. An extensive conjugation pattern is the major reason for the "invisibility" of the newly found BAs to other common analytical methods. Metabolomic analysis of the total unconjugated BA profile in combination with analysis of their conjugation patterns and urinary excretion tendencies have provided substantial insights into the interconnected roles of host and gut microbiota in maintaining BA homeostasis. It was proposed that the urinary total BA profile may serve as an ideal footprint for the functional status of the host-gut microbial BA co-. In summary, this work provided a powerful tool for human C24 BA metabolome analysis that bridges the gap between GC-MS techniques in the past age and LC-MS techniques currently prevailing in biomedical researches. Further applications of the present method in clinical, translational research, and other biomedical explorations will continue to boost the construction of a host-gut microbial co- network of BAs and thus facilitate the decryption of BA-mediated host-gut microbiota crosstalk in health and diseases. Graphical abstract ᅟ.</p><h1><a href=https://www.ncbi.nlm.nih.gov/pubmed/10985>3alpha-, 7alpha- and 12alpha-hydroxysteroid dehydrogenase activities from Clostridium perfringens.</a></h1><p>25 strains of Clostridium perfringens were screened for hydroxysteroid dehydrogenase activity; 19 contained NADP-dependent 3alpha-hydroxysteroid dehydrogenase and eight contained NAD-dependent 12alpha-hydroxysteroid dehydrogenase active against conjugated and unconjugated bile salts. All strains containing 12alpha-hydroxysteroid dehydrogenase also contained 3alpha-hydroxysteroid dehydrogenase although 12alpha-hydroxysteroid dehydrogenase was invariably in lesser quantity than the 3alpha-hydroxysteroid dehydrogenase. In addition, 7alpha-hydroxysteroid dehydrogenase activity was evident only when 3alpha, 7alpha, 12alpha-trihydroxy-5beta-cholanoate was substrate but notably absent when 3alpha, 7alpha-dihydroxy-5beta-cholanoate was substrate. The oxidation product , 7-diketo-5beta-cholanoate is rapidly further degraded to an unknown compound devoid of either 3alpha- or 7alpha-OH groups. Group specificity of these enzymes was confirmed by thin-layer chromatography studies of the oxidation products. These enzyme systems appear to be constitutive rather than inducible. In contrast to C. perfringens. Clostridium paraputrificum (five strains tested) contained no measurable hydroxysteroid dehydrogenase activity. pH studies of the C. perfringens enzymes revealed a sharp pH optimum at pH 11.3 and 10.5 for the 3alpha-OH- and 12alpha-OH-oriented activities, respectively. Kinetic studies gave Km estimates of approx. 5 X 10(-5) and 8 X 10(-4) M with 3alpha, 7a-dihydroxy-5beta-cholanoate and 3alpha, 12alpha-dihydroxy-5beta-cholanoate as substrates for two respective enzymes. 3alpha-hydroxysteroid dehydrogenase was active against 3alpha-OH-containing steroids such as androsterone regardless of the sterochemistry of the 5H (Both A/B cis and A/B trans steroides were substrates). There was no activity against 3beta-OH-containing steroids. The 3alpha- and 12alpha-hydroxysteroid dehydrogenase activities, although differing in cofactor requirements cannot be distinguished by their appearance in the growth curve, their mobility on disc gel electrophoresis, elution volume on passage through Sephadex G-200 or heat inactivation studies.</p></html>