<html><h2><a href=https://www.ncbi.nlm.nih.gov/pubmed/18922195>The effect of short-term hyperammonaemia on milk synthesis in dairy cows.</a></h2><p>To test the hypothesis that ammonia detoxification in ruminants consumes amino acids to the detriment of milk protein production, we infused four lactating dairy cows with ammonium acetate or sodium acetate in switchback experiments. Plasma ammonia concentrations increased to 411 microm within 1 h of the start of infusion of ammonium acetate at 567 mmol/h. The rate constant for ammonia clearance from plasma was 0 x 054/min and the half-life was 12 x 9 min. Infusion at 567 mmol/h for 1 h followed by 1 h without infusion, repeated four times between am- and pm-milking, caused a decrease in feed intake. Compared with sodium acetate, continuous infusion of ammonium acetate at 360 mmol/h throughout an entire 10-h milking interval increased plasma ammonia concentrations to 193 microm and caused a 20% decrease in milk, protein and lactose production with no effect on percentage composition of milk or the yield of milk fat. Arterial concentrations of glucose and non-esterified fatty acids tended to increase; there was no effect on arterial acetate, beta-hydroxybutyrate or triacylglcerol, and branched-chain amino acids, Lys and Thr decreased. Mammary plasma flow, estimated by assuming 100% uptake/output of Phe+, was significantly correlated with milk yield. Mammary uptakes of acetate tended to be reduced by hyperammonaemia, but uptakes of other  metabolites and amino acids were not affected. Thus, while an increase in amino acid consumption during hyperammonaemia was apparent from the drop in circulating concentrations of Leu, , Val, Lys and Thr, there was no evidence to support the hypothesis that milk yield is affected by the lower concentrations. An ammonia-induced depression in feed intake may have caused the decrease in milk synthesis.</p><h2><a href=https://www.ncbi.nlm.nih.gov/pubmed/8601843>Amino acid pair interchanges at spatially conserved locations.</a></h2><p>Here we study the pattern of amino acid interchanges at spatially, locally conserved regions in globally dissimilar and unrelated proteins. By using a method which completely separates the amino acid sequence from its respective structure, this work addresses the question of which properties of the amino acids are the most crucial for the stability of conserved structural motifs. The proteins are taken from a structurally non-redundant dataset. The spatially conserved substructural motifs are defined as consisting of a "large enough" number of Calpha atoms found to provide a geometric match between two proteins, regardless of the order of the Calpha atoms in the sequence, or of the sequence composition of the substructures. This approach can apply to proteins with little or no sequence similarity but with sufficient structural similarity, and is unique in its ability to handle local, non-topological matches between pairs of dissimilar proteins. The method uses a computer-version based algorithm, the Geometric Hashing. Since the Geometric Hashing ignores sequence information it lends itself to answer the question posed above. The interchanges at geometrically similar positions that have been obtained with our method demonstrate the expected behaviour. Yet, a closer inspection reveals some distant characteristics, as compared with interchanges based upon sequence-order based techniques, or from -contact-based considerations. First, a pronounced division of the amino acids into two classes is displayed: Lys, Glu, Arg, Gln, Asp, Asn, Pro, Gly, Thr, Ser and His on the one hand, and , Val, Leu, Phe, Met, , Trp, Cys and Ala on the other. These groups further cluster into subgroups: Lys, Glu, Arg, Gln; Asp Asn; Pro, Gly; , Val, Leu, Phe. The other amino acids stand alone. Analysis of the conservation among amino acids indicates proline to be consistently, by far, the most conserved. Next are Asp, Glu, Lys and Gly. Cys is also highly conserved. Interestingly, oppositely charged amino acids are interchanged roughly as frequently as those of the same charge. These observations can be explained in terms of the three-dimensional structures of the proteins. Most of all, there is a clear distinction between residues which prefer to be on the protein surfaces, compared to those frequently buried in the interiors. Analysis of the interchanges indicates their low information content. This, together with the separation into two groups, suggest that the predictive value of the spatial positions of the Calpha+ atoms is not much greater than the sequence alone, aside from their hydrophobicity/hydrophillicity classification.</p><h2><a href=https://www.ncbi.nlm.nih.gov/pubmed/27695803>Digestibility of , amino acids, and phosphorus in a novel source of soy protein concentrate and in soybean meal fed to growing pigs.</a></h2><p>Three experiments were conducted to determine standardized ileal digestibility (SID) of CP and AA, DE and ME, and apparent total tract digestibility (ATTD) and standardized total tract digestibility (STTD) of P in a new source of soy protein concentrate (SPC) and in soybean meal (SBM). In Exp. 1, 9 barrows (initial BW: 13.08 ± 1.98 kg) were prepared with a T-cannula in the distal ileum and allotted to a triplicated 3 × 3 Latin square design with 3 diets and 3 periods. A nitrogen-free diet and 2 diets that contained corn starch and SPC or SBM as the sole source of CP and AA were formulated. Each period lasted 7 d, and ileal digesta were collected on d 6 and 7 of each period. The SID for , Leu, Phe, Pro, and  was greater ( < 0.05) in SPC than in SBM, but for CP and all other AA, no difference between SPC and SBM was observed. In Exp. 2, 24 barrows (initial BW: 13.94 ± 1.34 kg) were housed individually in metabolism crates and randomly allotted to 1 of 3 diets. A corn-based diet (96.9% corn) and 2 diets that contained corn and SPC or corn and SBM were formulated. Each diet was fed to 8 pigs. Feces and urine samples were collected using the marker to marker method with 5-d adaptation and 5-d collection periods. The DE and ME in SPC and SBM were calculated using the difference procedure. Results indicated that the ATTD of GE was lower ( < 0.05) in SBM than in corn and the DE and ME in SPC were greater ( < 0.01) than in corn and SBM. In Exp. 3, 40 barrows (initial BW: 14.12 ± 2.08 kg) were placed in metabolism crates and allotted to 4 diets in a randomized complete block design with 10 pigs per diet. Two diets were based on SPC or SBM as the sole source of P. Two additional diets were formulated by adding microbial phytase to diets that were otherwise similar to the 2 initial diets. Feces were collected for 5 d after a 5-d adaptation period and values for ATTD and STTD of P were calculated. No differences were observed in ATTD and STTD of P between SPC and SBM, but the ATTD and STTD of P of both SPC and SBM were greater ( < 0.01) if microbial phytase was added to the diets. In conclusion, the concentrations of DE and ME are greater in SPC than in SBM, but the SID of most AA and the STTD of P are not different between SPC and SBM.</p><h2><a href=https://www.ncbi.nlm.nih.gov/pubmed/7410430>Allophycocyanin from Nostoc sp. phycobilisomes. Properties and amino acid sequence at the NH2 terminus of the alpha and beta subunits of allophycocyanins I, II, and III.</a></h2><p>Allophycocyanin from Nostoc sp. phycobilisomes was separated into four spectrally distinct components designated allophycocyanin I, B, II, and III by adsorption chromatography on brushite columns. Allophycocyanins I and B had red-shifted fluorescence emission maxima, and on this basis, may function in transfer of excitation  from phycobilisomes to chlorophyll a. Allophycyanins II and III, which together comprise 70% of the total allophycocyanin, have absorption maxima at 648 nm and 650 nm, respectively, and probably transfer excitation  from phycocyanin to allophycocyanins I and B, in addition to serving a structural function. Allophycocyanin I was resolved into alpha, beta, and gamma subunits with apparent molecular weights of 18,000, 17,000, and 35,000, respectively, whereas allophycocyanin B was resolved into two subunits with apparent molecular weights of 16,100 and 15,300, using a modified Weber and Osborn gel electrophoresis system (Brown, A. S., and Troxler, R. F. (1977) Biochem. J. 163, 571-581). In the same gel system, allophycocyanins II and III were each resolved into alpha and beta subunits with apparent molecular weights of 18,000 and 17,000, respectively. The subunits of allophycocyanins I, II, and III were isolated by gel filtration and ion exchange chromatography and the amino acid compositions determined. Automated sequence analysis demonstrated that the first 30 amino acids at the NH2 terminus of alpha subunits, and the beta subunits, of allophycocyanins I to III were identical. alpha Subunits: Ser--Val-Thr-Lys-Ser--Val-Asn-Ala-Asp-Ala-Glu-Ala-Arg--Leu-Ser-Pro-Gly -Glu-Leu-Asp-Arg--Lys-Ser-Phe-Val-Thr- beta Subunits: Ala-Gln-Asp-Ala--Thr-Ala-Val--Asn-Ala-Ala-Asp-Val-Gln-Gly-Lys--Leu-Asp-Ala-Thr-Ala-Leu-Ser-Lys-Leu-Lys-Ala-- The gamma subunit of allophycocyanin I and both subunits of allophycocyanin B appeared to be blocked at the NH2 terminus, suggesting that the allophycocyanin B subunits may be different gene products than those of allophycocyanins I to III, or if the same, the subunits of allophycocyanin B undergo proteolytic modification after initial synthesis.</p><h2><a href=https://www.ncbi.nlm.nih.gov/pubmed/20460665>Metabolizable  and standardized ileal digestible amino acid contents of expeller-extracted canola meal fed to broiler chicks.</a></h2><p>An experiment was conducted to determine the standardized ileal digestible amino acid (AA) and AMEn contents of expeller-extracted canola meal (EECM) fed to broiler chicks. One hundred forty-four broiler chicks were divided into 24 groups of 6 birds balanced for BW and fed 4 diets in a completely randomized design (6 groups per diet) from d 14 to 21 of age. The diets were a corn-soybean meal-based basal diet formulated to meet NRC nutrient requirements, the basal diet with - and AA-yielding ingredients replaced by 30% of either solvent-extracted canola meal (SECM) or EECM, and a low-protein casein-cornstarch-based diet. The SECM, which is commonly used in formulating poultry diets, was fed for comparison with EECM, whereas the casein-cornstarch-based diet was fed to estimate basal endogenous AA losses for determining standardized ileal digestibility of AA. All 4 diets contained titanium oxide (0.3%) as an indigestible marker, and nutrient digestibility and retention were determined by the substitution method. From d 19 to 21, excreta samples were collected for AMEn determination. On d 21, the birds were killed by cervical dislocation, and contents of the whole ileum were obtained for determination of AA digestibility. Expeller-extracted canola meal had greater (P<0.05) standardized ileal digestible contents of Gly, Leu, Ser, Thr, Asp, and Glu and tended to have greater (P<0.10) standardized ileal digestible contents of , Lys, Phe, Ala, and  than SECM. Compared with SECM, EECM had greater (P<0.001) AMEn (2,694 vs. 1,801 kcal/kg). The results show that the EECM evaluated in the current study had greater standardized ileal digestible AA and AMEn contents than SECM, and hence, it may be a better source of protein and  for broiler chicks than SECM. The standardized ileal digestible AA and AMEn values of EECM used in the current study could be used when formulating broiler chicken diets using the same to minimize the N excretion and feeding cost.</p><h2><a href=https://www.ncbi.nlm.nih.gov/pubmed/30005070>Fetal Huanjiang mini-pigs exhibit differences in nutrient composition according to body weight and gestational period.</a></h2><p>Low birth weight may negatively affect  storage and nutrient metabolism, and impair fetal growth and development. We analyzed effects of body weight (BW) and gestational period on nutrient composition in fetal Huanjiang mini-pigs. Fetuses with the lowest BW (LBW), middle BW (MBW), and highest BW (HBW) were collected at days 45, 75, and 110 of gestation. Crude protein (CP), crude fat, amino acid (AA), and fatty acid (FA) concentrations were determined. The BW gain, carcass weight, fat percentage, and uterus weight of sows increased as gestation progressed, as did litter weight, average individual fetal weight, fetal body weight, and dry matter (DM). The concentrations of Ala, Arg, crude fat, Gly, Pro, , C14:0, C16:0, C16:1, C18:1n9c, C18:2n6c, C18:3n3, C18:3n6, C20:0, C20:3n6, saturated FA (SFA), and monounsaturated FA (MUFA) increased significantly as gestation progressed. The percentage of skeleton, and the ratio of the liver, lung, and stomach to BW decreased as gestation progressed. There were also significant reductions in the concentrations of CP, Asp, Glu, His, , Leu, Lys, Phe, Ser, Thr, essential AA (EAA), acidic AA, C17:0, C20:4n6, C22:6n3, unsaturated FA (UFA), polyunsaturated FA (PUFA), n-3PUFA, n-6PUFA as gestation progressed, and reductions in EAA/total AA (TAA), PUFA/SFA, and n-3/n-6 PUFA. The LBW fetuses exhibited the lowest BW and crude fat, C14:0, C16:1, C17:0, C18:2n6c, and MUFA concentrations at days 75 and 110 of gestation. They also exhibited lower  concentration at day 45 of gestation and lower Glu concentration at day 75 of gestation than HBW fetuses. These findings suggest that LBW fetuses exhibit lower amounts of crude fat and several FAs during mid-gestation and late-gestation, which may in turn affect adaptability, growth, and development.</p><h2><a href=https://www.ncbi.nlm.nih.gov/pubmed/18488495>[Spatial structure of isoleucine pentapeptides Glu-Phe-Leu-Arg--NH2 and Pro-Phe--Arg--NH2].</a></h2><p>The spatial structure of two cardioactive isoleucine pentapeptides Glu-Phe-Leu-Arg--NH2 (I) and Pro-Phe--Arg--NH2 (II) have been investigated using the theoretical conformational analysis. The low- conformations of these molecules were found, the values of dihedral angles of the backbone and side chains of the amino acid residues constituting these peptides were determined, and the energies of intra- and interresidual interactions were estimated. It was revealed that the spatial structure of molecule I can exist as five and that of molecule II as seven stable backbone forms.</p><h2><a href=https://www.ncbi.nlm.nih.gov/pubmed/2474609>Structures of histamine-releasing peptides formed by the action of acid proteases on mammalian albumin(s).</a></h2><p>The acid proteases, pepsin, rennin and cathepsin D, were shown to generate mast cell histamine releasing peptides (HRP) when incubated with the albumin fraction of mammalian plasmas. Significant histamine release was observed using less than 1 microliter equivalent of pepsin-treated plasma. Histamine release was rapid, dependent on calcium and , and accompanied by degranulation. The major HRP present in pepsin-treated human and canine plasma was identified as H--Ala-Arg-Arg-His-Pro--Phe-OH whereas that from rat plasma had valine substituted for isoleucine. Cathepsin D-treated BSA gave rise to the human octapeptide (above) as well as to an extended decapeptide with H--Glu- at the N-terminus. These peptides were apparently derived from one region of serum albumin, residues 139 to 149 of the human, canine, or bovine sequence. We hypothesize that cathepsin D, released from leukocyte lysosomes, might generate HRP during the delayed phase of an inflammatory response.</p><h2><a href=https://www.ncbi.nlm.nih.gov/pubmed/8580841>Conservative substitutions in the hydrophobic core of Rhodobacter sphaeroides thioredoxin produce distinct functional effects.</a></h2><p>The internal residue Phe 25 in Rhodobacter sphaeroides thioredoxin was changed to five amino acids (Ala, Val, Leu, , ) by site-directed mutagenesis, and the mutant proteins were characterized in vitro and in vivo using the mutant trxA genes in an Escherichia coli TrxA- background. The substitution F25A severely impaired the functional properties of the enzyme. Strains expressing all other mutations can grow on methionine sulfoxide with growth efficiencies of 45-60% that of the wild type at 37 degrees, and essentially identical at 42 degrees. At both temperatures, however, strains harboring the substitutions F25V and F25Y had lower growth rates and formed smaller colonies. In another in vivo assay, only the wild type and the F25I substitution allowed growth of phage T3/7 at 37 degrees, demonstrating that subtle modifications of the protein interior at position 25 /Leu or Phe/) can produce significant biological effects. All F25 mutants were good substrates for E. coli thioredoxin reductase. Although turnover rates and apparent Km values were significantly lower for all mutants compared to the wild type, catalytic efficiency of thioredoxin reductase was similar for all substrates. Determination of the free  of unfolding showed that the aliphatic substitutions (Val, Leu, ) significantly destabilized the protein, whereas the F25Y substitution did not affect protein stability. Thus, thermodynamic stability of R. sphaeroides thioredoxin variants is not correlated with the distinct functional effects observed both in vivo and in vitro.</p><h2><a href=https://www.ncbi.nlm.nih.gov/pubmed/2180743>Fluorescence-based continuous assay for the aspartyl protease of human immunodeficiency virus-1.</a></h2><p>5-Dimethylaminoaphthalene-1-sulfonyl-Ser-Gln-Asn--Pro--Val-T rp (Dns-SQNYPIVW) is a fluorescent substrate for the aspartyl protease of human immunodeficiency virus-1. In intact substrate, fluorescence of Trp (lambda ex 290 nm, lambda em 360 nm) was 60% quenched by  transfer to the dansyl group. Protease-catalyzed cleavage at the -Pro bond abolished the  transfer, and the consequent increase in Trp fluorescence was used to follow the enzymatic reaction. At substrate concentrations less than 60 microM, initial reaction velocity increased as a linear function of substrate concentration, with kcat/KM = 9700 M-1 s-1. Limited solubility and internal fluorescence quenching precluded a determination of KM for Dns-SQNYPIVW, but this was clearly greater than 100 microM.</p><h2><a href=https://www.ncbi.nlm.nih.gov/pubmed/8589252>Conformational study of angiotensin II.</a></h2><p>Conformational free  calculations using an empirical potential ECEPP/3 (Empirical Conformational  Program for Peptides, Version 3) were carried out on angiotensin II (AII) of sequence Asp-Arg-Val---His-Pro-Phe to find the stable conformations of the free state in the unhydrated and the hydrated states. A conformational analysis of the unhydrated state was carried out using the buildup procedure. The free  calculation using the hydration shell model was also carried out to obtain the stable conformation of the hydrated state. The calculated stable conformations of AII in both states have a partially right-handed alpha-helical structure stabilized by short- and medium-range interactions. The similarity between the lowest free  conformations of the unhydrated and hydrated states suggests that the hydration might not be important to stabilize the overall conformation of AII in a free state. The absence of any intramolecular interaction of the  side chain suggests the possible interaction of this residue with the receptor. In this study, we found that the low free  conformations contain both the parallel-plate and the perpendicular-plate geometries of the His and Phe rings, suggesting the coexistence of both conformations.</p><h2><a href=https://www.ncbi.nlm.nih.gov/pubmed/12113341>Ion-pair reversed-phase liquid chromatography-electrospray mass spectrometry for the analysis of underivatized small peptides.</a></h2><p>The single run analysis of 23 small peptides (principally glycyl and lysyl dipeptides) is performed by ion-pair reversed-phase liquid chromatography coupled with evaporative light scattering detection or electrospray (tandem) mass spectrometry. Several perfluorinated carboxylic acid homologues are evaluated with an octadecyl silica stationary phase (Supelcosil ABZ+ Plus). Among the perfluorocarboxylic acids tested the nonafluoropentanoic acid and the tridecafluoroheptanoic acid gave the best results. Special attention was paid to the separation of isomer/isobar dipeptides (e.g., Gly-, Gly-Leu, Leu-Gly, as well as Gly-Gln, Gly-Lys, etc.) which is usually necessary in spite of the high specificity of mass spectrometry. Before LC-MS analysis, ion-spray fragmentation as well as optimization of MS parameters of the analysed peptides was investigated. The optimum collision  of glysyl peptides, Ala-Gln, Asp-Asp and Asp-Asp-Asp (13-18 eV) was different from that of the lysyl peptides, -Glu and oxidised glutathione (25-32 eV). Limits of detection varied from 0.1 to 1.2 mg l(-1) for simple MS and 0.05 to 25 mg l(-1) for tandem MS.</p><h2><a href=https://www.ncbi.nlm.nih.gov/pubmed/20969427>Sequence composition and environment effects on residue fluctuations in protein structures.</a></h2><p>Structure fluctuations in proteins affect a broad range of cell phenomena, including stability of proteins and their fragments, allosteric transitions, and  transfer. This study presents a statistical-thermodynamic analysis of relationship between the sequence composition and the distribution of residue fluctuations in protein-protein complexes. A one-node-per-residue elastic network model accounting for the nonhomogeneous protein mass distribution and the interatomic interactions through the renormalized inter-residue potential is developed. Two factors, a protein mass distribution and a residue environment, were found to determine the scale of residue fluctuations. Surface residues undergo larger fluctuations than core residues in agreement with experimental observations. Ranking residues over the normalized scale of fluctuations yields a distinct classification of amino acids into three groups: (i) highly fluctuating-Gly, Ala, Ser, Pro, and Asp, (ii) moderately fluctuating-Thr, Asn, Gln, Lys, Glu, Arg, Val, and Cys, and (iii) weakly fluctuating-, Leu, Met, Phe, , Trp, and His. The structural instability in proteins possibly relates to the high content of the highly fluctuating residues and a deficiency of the weakly fluctuating residues in irregular secondary structure elements (loops), chameleon sequences, and disordered proteins. Strong correlation between residue fluctuations and the sequence composition of protein loops supports this hypothesis. Comparing fluctuations of binding site residues (interface residues) with other surface residues shows that, on average, the interface is more rigid than the rest of the protein surface and Gly, Ala, Ser, Cys, Leu, and Trp have a propensity to form more stable docking patches on the interface. The findings have broad implications for understanding mechanisms of protein association and stability of protein structures.</p><h2><a href=https://www.ncbi.nlm.nih.gov/pubmed/22573843>Triennial Lactation Symposium: Mammary metabolism of amino acids in dairy cows.</a></h2><p>Although in dairy cows the mammary gland (MG) is the major net user of essential AA (EAA) supply, milk protein synthesis from absorbed EAA is not a straightforward process. Early studies identified 2 groups of EAA based on different pattern of mammary utilization: group 1 [Met, Phe (+), Trp], where MG uptake was similar to secretion in milk protein, and group 2 (Arg, , Leu, Lys, Thr, and Val), where uptake exceeded milk protein output. This review examines the validity of this classification under variable protein supply through a meta-analysis, with the outcomes then explained with studies in which the fates of individual EAA were monitored using isotope approaches. For the meta-analysis, the Fick principle, based on stoichiometric transfer of Phe+ uptake to milk protein, was used to estimate mammary plasma flow across all studies. This approach was judged acceptable because doubling Phe supply did not result in mammary oxidation of Phe+ and either limited or no contribution of peptides to Phe and  mammary supply could be detected. The AA content of proteins synthesized by the MG was estimated from milk protein composition, and the uptake-to-output ratio (U:O) for individual AA was re-calculated based on these assumptions. Analysis of individual samples by isotopic dilution resulted in reduced variance compared with analysis on pooled samples performed with an AA analyzer. Globally, the U:O of His and Met is maintained close to unity under variable protein supply. The group 2 AA could be subdivided. First, the U:O for group "2v" AA (, Leu, Val, and Lys) is greater than 1 and varied with protein supply. Accordingly, the increased U:O of Leu, induced by duodenal casein infusion, led to extra-mammary Leu oxidation. Decreasing Lys supply decreased Lys U:O and the associated transfer of N to non-EAA, mainly to Glx, Asx, Ser, and Ala. Second, the U:O of group "2nv" AA, Arg and Thr, does not vary with protein supply. The Arg U:O averages 2.5, whereas the Thr U:O, albeit averaging 1.2, does not differ from unity. Excess of both these AA is probably directed toward the synthesis of non-EAA rather than  supply. Overall, the ability of the MG to use excess EAA-N supply offers alternative sources of N and C for  provision, lactose synthesis and non-EAA synthesis. The latter function spares dietary non-EAA for other necessary processes, such as gluconeogenesis and  supply, in other tissues to support lactation.</p><h2><a href=https://www.ncbi.nlm.nih.gov/pubmed/15723532>Role of hydrogen bond networks and dynamics in positive and negative cooperative stabilization of a protein.</a></h2><p>Cooperativity mediated through hydrogen bond networks in yeast iso-1-cytochrome c was studied using a thermodynamic triple mutant cycle. Three known stabilizing mutations, Asn 26 to His, Asn 52 to , and  67 to Phe, were used to construct the triple mutant cycle. The side chain of His 26, a wild-type residue, forms two hydrogen bonds that bridge two substructures of the wild-type protein, and  67 and Asn 52 are part of an extensive buried hydrogen bond network. The stabilities of all variants in the triple mutant cycle were determined by guanidine hydrochloride denaturation methods and used to determine the pairwise, Delta(2)G(int), and triple interaction energies. His 26 and  52 interact cooperatively (Delta(2)G(int) is 1-2 kcal/mol), whereas the two other pairs of mutations interact anticooperatively (Delta(2)G(int) is -0.5 to -1.5 kcal/mol). Previously reported structural data for iso-1-cytochrome c variants containing these mutations show that changes in the strength of the His 26 to Glu 44 hydrogen bond, apparently caused by changes in main chain dynamics, provide a mechanism for the long distance (His 26 to Phe 67 and His 26 to  52) propagation of pairwise interaction energies. Opposing changes in the strength of the His 26 to Glu 44 hydrogen bond caused by the N52I and Y67F mutations generate a negative triple interaction  (-0.9 +/-0.7 kcal/mol) that combined with cancellation of cooperative and anticooperative pairwise interactions produce apparent additivity for the stabilizing effects of the single mutations in the triple mutant variant.</p><h2><a href=https://www.ncbi.nlm.nih.gov/pubmed/7227760>Augmented utilization of branched-chain amino acids by skeletal muscle in decompensated liver cirrhosis in special relation to ammonia detoxication.</a></h2><p>Femoral arterio-venous (A-V) differences of blood free amino acids and plasma ammonia (NH3) were simultaneously determined after an overnight fast in 16 patients with decompensated liver cirrhosis in the absence and presence of encephalopathy, as compared with those in 8 control subjects. In spite of increased releases of phenylalanine (Phe) and tyrosine () from the peripheral tissue, releases of isoleucine () and leucine (Leu) as well as alanine (Ala) were found to be significantly reduced in decompensated liver cirrhosis, particularly in the presence of hepatic encephalopathy. Furthermore, NH3 was found to be significantly taken up by the skeletal muscle of these patients, and a positive correlation was observed between arterial NH3 level and the A-V differences of Leu, of  and of Ala. These findings strongly suggest that net degradation (or utilization) of branched-chain amino acids (in particular, Leu and ) is enhanced in the muscle for detoxication of ammonia (i.e., glutamine synthesis) by supplying the carbon skeleton and  in cirrhosis of the liver.</p><h2><a href=https://www.ncbi.nlm.nih.gov/pubmed/14633226>Changes in the level of endogenous leptin, FSH, 17beta-oestradiol and metabolites during lupin-induced increase in ovulation rate in ewes.</a></h2><p>The aim of the study was to determine the changes in the plasma concentration of leptin during lupin feeding-induced increase in the ovulation rate (OR) in ewes. Additionally, alterations in the plasma level of glycogenic amino acids and glucose (as the factors influencing leptin secretion) and the levels of follicle-stimulating hormone (FSH) and 17beta-oestradiol (E-2) (as the hormones regulated by leptin and engaged in recruitment, selection and development of ovulatory follicles) were analysed. Ninety-six female Polish Lowland Sheep were used. All ewes were cyclic and synchronized with PGF2alpha. The ewes were divided into two groups: control (n = 48), fed only with hay, and experimental (n = 48), received additionally lupin (Lupinus angustifolius) grain as a high-protein and a high- supplement. They were given lupin from the second to 13th day of the oestrous cycle at increasing doses (150-750 g/day per ewe). On the 11th day of cycle blood samples for analysis of hormones, amino acids and total glucose concentration, were collected from the jugular vein. OR was determined by laparoscopy of ovaries on the sixth day of the following oestrous cycle. Mean OR of ewes supplemented with lupin grain (1.687 +/- 0.463) was 30.67% higher than that of control (1.291 +/- 0.454). In spite of the unchanged body mass, a significant increase (P < or = 0.05) in mean concentration of plasma leptin in the experimental ewes [2.17 +/- 0.15 ng/ml human equivalent (HE)] was found in comparison with control (1.42 +/- 0.12 ng/ml HE). A significantly (P < or = 0.05) higher plasma FSH level in the ewes fed lupin (105.21 +/- 5.87 ng/ml) compared with those fed hay (67.88 +/- 6.03 ng/ml) was also found. However, plasma level of E-2 decreased after lupin feeding. Moreover, in the ewes fed lupin the plasma concentrations of glucose and nine glycogenic amino acids (Gly, Ala, Val, Met, Leu, , , Phe and Arg) were increased. It can be concluded that lupin feeding exerts the stimulatory effect on the OR in Polish Lowland Sheep. The increase in OR is connected with significantly higher plasma leptin level and coincident with rise in FSH, glycogenic amino acids and glucose concentration. In contrast, the level of plasma E-2 was significantly decreased in lupin-fed ewes.</p><h2><a href=https://www.ncbi.nlm.nih.gov/pubmed/24189289>Allyl/propenyl phenol synthases from the creosote bush and engineering production of specialty/commodity chemicals, eugenol/isoeugenol, in Escherichia coli.</a></h2><p>The creosote bush (Larrea tridentata) harbors members of the monolignol acyltransferase, allylphenol synthase, and propenylphenol synthase gene families, whose products together are able to catalyze distinct regiospecific conversions of various monolignols into their corresponding allyl- and propenyl-phenols, respectively. In this study, co-expression of a monolignol acyltransferase with either substrate versatile allylphenol or propenylphenol synthases in Escherichia coli established that various monolignol substrates were efficiently converted into their corresponding allyl/propenyl phenols, as well as providing proof of concept for efficacious conversion in a bacterial platform. This capability thus potentially provides an alternate source to these important plant phytochemicals, whether for flavor/fragrance and fine chemicals, or ultimately as commodities, e.g., for renewable  or other intermediate chemical purposes. Previous reports had indicated that specific and highly conserved amino acid residues 84 (Phe or Val) and 87 ( or ) of two highly homologous allyl/propenyl phenol synthases (circa 96% identity) from a Clarkia species mainly dictate their distinct regiospecific catalyzed conversions to afford either allyl- or propenyl-phenols, respectively. However, several other allyl/propenyl phenol synthase homologs isolated by us have established that the two corresponding amino acid 84 and 87 residues are not, in fact, conserved.Copyright © 2013 Elsevier Inc. All rights reserved.</p><h2><a href=https://www.ncbi.nlm.nih.gov/pubmed/23933241>Synthesis and characterization of novel fluorogenic substrates of coagulation factor XIII-A.</a></h2><p>Further development of our recently published Glu(pNA)-containing peptides (Anal. Biochem. 428 (2012) 73-80) provided new fluorogenic substrates for the activated blood coagulation factor XIII. A first series was designed by incorporation of Glu(AMC) at the penultimate position from the N terminus. For the best derivative H--Glu(AMC)-Val-Lys-Val--NH2, a moderate kcat/Km value of 34s(-1)M(-1) was determined, which is more than 100-fold reduced compared with the previously reported Glu(pNA) substrates. Furthermore, two fluorescence resonance  transfer (FRET) substrates were prepared by incorporation of an N-methyl-anthraniloyl fluorophore and a 2,4-dinitrophenyl quencher. Both substrates were excellently cleaved by FXIII-A2(∗), which is generated from its zymogen by activation of thrombin in the presence of calcium ions. In the absence and presence of H-Gly-ethyl ester, kcat/Km values of 8010 and 8660s(-)(1)M(-)(1), respectively, were found for the conversion of H-Lys(N(Me)Abz)-Glu(NH-(CH2)4-NH-Dnp)-Val-Lys-Val--Gly-NH2 (substrate 8). These values are more than 200-fold improved compared with the Glu(AMC) substrates. Substrate 8 is suitable for the measurement of FXIII-A2(∗) activities in plasma samples as well as for in vitro measurements. Furthermore, it was used for the determination of the inhibitory potency of a newly synthesized chloromethyl ketone derivative, Cbz-Phe-Glu(CMK)-Val-Lys-Val--Gly-NH2, which was found to be a potent irreversible inhibitor of FXIII-A2(∗).Copyright © 2013 Elsevier Inc. All rights reserved.</p><h2><a href=https://www.ncbi.nlm.nih.gov/pubmed/25375824>Conformations of a model cyclic hexapeptide, CYIQNC: (1)H-NMR and molecular dynamics studies.</a></h2><p>Solution conformation of the cyclic hexapeptide sequence, [cyclo-S-Cys---Gln-Asn-Cys-S] (CYIQNC) - a disulfide-linked fragment of a neurohypophyseal peptide hormone oxytocin (OT) - has been investigated by high-field one-dimensional (1D) and two-dimensional (2D) NMR spectroscopic methods and compared with the results obtained from computer simulation studies. (1)H-NMR results based on temperature dependence of amide proton chemical shifts and nuclear Overhauser effect indicate that peptide in solution populates different conformations, characterized by two fused β-turns. The segment (3)-Gln(4)-Asn(5)-Cys(6) yields a preferred type-III β-turn at residues 4, 5 (HB, 3HN → 6CO), while the segment Cys(6), Cys(1)-(2)-(3) exhibits inherently weaker, flexible β-turn either of type I/II'/III/half-turn at residues 1, 2 (HB, 6HN → 3CO). The computer simulation studies using a mixed protocol of distance geometry-simulated annealing followed by constrained minimization, restrained molecular dynamics, and  minimization showed the possibility of existence of additional conformations with the hydrogen bonds, (a) 5HN → 3CO and (b) 2HN → 6CO. These results, therefore, indicate that the additional conformations obtained from both NMR and simulation studies can also be possible to the peptide. These additional conformations might have very small population in the solution and did not show their signatures in these conditions. These findings will be helpful in designing more analogs with modifications in the cyclic moiety of OT.</p><h2><a href=https://www.ncbi.nlm.nih.gov/pubmed/7795524>Conformationally constrained analogs of protein kinase inhibitor (6-22)amide: effect of turn structures in the center of the peptide on inhibition of cAMP-dependent protein kinase.</a></h2><p>The high-affinity interaction between protein kinase inhibitor (PKI)(6-22)amide(Thr6--Ala-Asp-Phe--Ala-Ser-Gly-Arg-Thr-Gly- Arg-Arg-Asn- Ala-Ile22-NH2) and the catalytic subunit of cAMP-dependent protein kinase requires both the N-terminal Thr6 to Ile11 sequence of the inhibitor peptide and its C-terminal pseudosubstrate site comprised of Arg15 to Ile22. Small angle X-ray scattering data indicate that PKI(6-22)amide has a compact, rather than extended, structure in solution (Reed J et al., 1989, Biochem J 264:371-380). CD spectroscopic analysis of the PKI peptide led to the suggestion that a beta-turn structure might be located in the -Ala12-Ser-Gly-Arg15-connecting sequence in the middle of the molecule (Reed J, Kinzel V, Cheng HC, Walsh DA, 1987, Biochemistry 26:7641-7647). To investigate this possibility further, conformationally constrained and flexible analogs of PKI(6-22)amide were synthesized and used to study the structure-function relationships of this central portion of the inhibitor. (Des12-14)PKI(6-22) amide exhibited over a 200-fold loss in inhibitory activity. Replacement of the omitted -Ala12-Ser-Gly14-sequence with aminocaprylic acid yielded an analog that regained more than 90% of the lost binding . The D-alanine14 PKI analog was as potent as the parent peptide, whereas the beta-alanine14 and the sarcosine14 analogs were only 10-fold less active. Several peptides that promoted a beta-turn structure at residues 12-15 showed about 200-fold decreases in inhibitory activity. Two constrained analogs that could not assume a beta-turn conformation were only 30-fold less potent than PKI(6-22)amide. Thus, the structure of the central connecting portion of the PKI peptide, encompassing residues 12-15, greatly influences its ability to effectively bind to and inhibit the catalytic subunit. We conclude, however, that a formal beta-turn at this position is not required and is actually detrimental for a high-affinity interaction of PKI(6-22)amide with the enzyme. These results are interpreted in light of the Fourier-transform infrared spectra of the peptide analogs and the crystal structure of the peptide bound at the active site of the protein kinase (Knighton DR et al., 1991b, Science 253:414-420).</p><h2><a href=https://www.ncbi.nlm.nih.gov/pubmed/22453000>Identification of εPKC targets during cardiac ischemic injury.</a></h2><p>Epsilon-protein kinase C (εPKC) protects the heart from ischemic injury. However, the mechanism(s) of εPKC cardioprotection is still unclear. Identification of the εPKC targets may aid in elucidating the εPKC-mediated cardioprotective mechanisms. Previous studies, using εPKC transgenic mice and difference in gel electrophoresis, identified proteins involved in glucose metabolism, the expression of which was modified by εPKC. Those studies were accompanied by metabolomic analysis, suggesting that increased glucose oxidation may be responsible for the cardioprotective effect of εPKC. Whether these εPKC-mediated alterations were because of differences in protein expression or phosphorylation was not determined.In the present study, we used an εPKC -specific activator peptide, ψεRACK, combined with phosphoproteomics, to find εPKC targets, and identified that the proteins whose phosphorylation was altered by selective activation of εPKC were mostly mitochondrial proteins. Analysis of the mitochondrial phosphoproteome led to the identification of 55 spots, corresponding to 37 individual proteins, exclusively phosphorylated, in the presence of ψεRACK. The majority of the proteins identified were involved in glucose and lipid metabolism, components of the respiratory chain as well as mitochondrial heat shock proteins.The protective effect of εPKC during ischemia involves phosphorylation of several mitochondrial proteins involved in glucose and lipid metabolism and oxidative phosphorylation. Regulation of these metabolic pathways by εPKC phosphorylation may lead to εPKC-mediated cardioprotection induced by ψεRACK.</p><h2><a href=https://www.ncbi.nlm.nih.gov/pubmed/18937371>Binding and release of cholesterol in the Osh4 protein of yeast.</a></h2><p>Sterols have been shown experimentally to bind to the Osh4 protein (a homolog of the oxysterol binding proteins) of Saccharomyces cerevisiae within a binding tunnel, which consists of antiparallel beta-sheets that resemble a beta-barrel and three alpha-helices of the N-terminus. This and other Osh proteins are essential for intracellular transport of sterols and ultimately cell life. Molecular dynamics (MD) simulations are used to study the binding of cholesterol to Osh4 at the atomic level. The structure of the protein is stable during the course of all MD simulations and has little deviation from the experimental crystal structure. The conformational stability of cholesterol within the binding tunnel is aided in part by direct or water-mediated interactions between the 3-hydroxyl (3-OH) group of cholesterol and Trp(46), Gln(96), (97), Asn(165), and/or Gln(181) as well as dispersive interactions with Phe(42), Leu(24), Leu(39), (167), and (203). These residues along with other nonpolar residues in the binding tunnel and lid contribute nearly 75% to the total binding . The strongest and most populated interaction is between Gln(96) and 3-OH with a cholesterol/Gln(96) interaction  of -4.5 +/- 1.0 kcal/mol. Phe(42) has a similar level of attraction to cholesterol with -4.1 +/- 0.3 kcal/mol. A MD simulation without the N-terminus lid that covers the binding tunnel resulted in similar binding conformations and binding energies when compared with simulations with the full-length protein. Steered MD was used to determine details of the mechanism used by Osh4 to release cholesterol to the cytoplasm. Phe(42), Gln(96), Asn(165), Gln(181), Pro(211), and (206) are found to direct the cholesterol as it exits the binding tunnel as well as Lys(109). The mechanism of sterol release is conceptualized as a molecular ladder with the rungs being amino acids or water-mediated amino acids that interact with 3-OH.</p><h2><a href=https://www.ncbi.nlm.nih.gov/pubmed/9517542>Mapping the active site of factor Xa by selective inhibitors: an NMR and MD study.</a></h2><p>The structure of two selective inhibitors, Ac---Arg--Pro-NH2 and Ac-(4-Amino-Phe)-(Cyclohexyl-Gly)-Arg-NH2, in the active site of the blood clotting enzyme factor Xa was determined by using transferred nuclear Overhauser effect nuclear magnetic resonance (NMR) spectroscopy. They represent a family of peptidic inhibitors obtained by the screening of a vast combinatorial library. Each structure was first calculated by using standard computational procedures (distance geometry, simulated annealing,  minimization) and then further refined by systematic search of the conformation of the inhibitor docked in the active site and repeating the simulated annealing and  minimization. The final structure was optimized by molecular dynamics simulations of the inhibitor-complex in water. The NMR restraints were kept throughout the refinement. The inhibitors assume a compact, very well defined conformation, embedded into the substrate binding site not in the same way as a substrate, blocking thus the catalysis. The model allows to explain the mode of action, affinity, and specificity of the peptides and to map the active site.</p><h2><a href=https://www.ncbi.nlm.nih.gov/pubmed/20042596>Structural basis for the association of the redox-sensitive target of rapamycin FATC domain with membrane-mimetic micelles.</a></h2><p>The target of rapamycin (TOR) is a conserved eukaryotic Ser/Thr kinase that regulates cellular growth in response to the nutrient and  state. TOR signaling plays an important role in the development of diseases such as cancer, obesity, and diabetes and in different redox-sensitive processes (hypoxia, apoptosis, and aging). Because TOR has been detected at different cellular membranes and in the nucleus, its localization may influence the specific signaling readout. To better understand how TOR can associate with different membranes, the lipid-binding properties of the redox-sensitive yeast TOR1 FATC domain (y1fatc) have been characterized by solution NMR spectroscopy. Binding studies with different lipids indicate that y1fatc interacts specifically with a membrane-mimetic environment but appears not to recognize a specific lipid headgroup. In both, the structures of oxidized and reduced micelle-bound y1fatc, residues -2456 to Trp-2470 of the lipid-binding motif form a hydrophobic bulb that has a rim of charged residues. The diffusion constants for both micelle-bound states are consistent with the rotational correlation times from the analysis of the (15)N relaxation data. Based on the K(d) values, the oxidized form (K(d) approximately 0.31 mm) binds dodecyl phosphocholine micelles slightly tighter than the reduced form (K(d) approximately 1.86 mM). Binding studies with y1fatc in which one or both tryptophans (Trp-2466 and Trp-2470) were replaced by alanine suggest that these residues are important for the exact positioning in the membrane and that the other aromatic (His-2462, -2463, and Phe-2469) and aliphatic residues (-2456, Leu-2459, -2464, and Pro-2468) in the lipid-binding motif contribute significantly to the affinity.</p><h2><a href=https://www.ncbi.nlm.nih.gov/pubmed/10413483>Effects of structure on alpha C-H bond enthalpies of amino acid residues: relevance to H transfers in enzyme mechanisms and in protein oxidation.</a></h2><p>The bond dissociation enthalpies (BDE) of all of the amino acid residues, modeled by HC(O)NHCH(R)C(O)NH(2) (PH(res)), were determined at the B3LYP/6-31G//B3LYP/6-31G level, coupled with isodesmic reactions. The results for neutral side chains with phi, psi angles approximately 180 degrees, approximately 180 degrees in ascending order, to an expected accuracy of +/-10 kJ mol(-)(1), are Asn 326; cystine 330; Asp 332; Gln 334; Trp 337; Arg 340; Lys 340; Met 343; His 344; Phe 344;  344; Leu 344; Ala 345; Cys 346; Ser 349; Gly 350;  351; Val 352; Glu 354; Thr 357; Pro-cis 358; Pro-trans 369. BDEs calculated at the ROMP2/6-31G//B3LYP/6-31G level exhibit the same trends but are approximately 7 kJ mol(-)(1) higher. All BDEs are smaller than those of typical secondary or tertiary C-H bonds due to the phenomenon of captodative stabilization. The stabilization is reduced by changes in the phi,psi angles. As a result the BDEs increase by about 10 kJ mol(-)(1) in beta-sheet and 40 kJ mol(-)(1) in alpha-helical environments, respectively. In effect the alpha C-H BDEs can be "tuned" from about 345 to 400 kJ mol(-)(1) by adjusting the local environment. Some very significant effects of this are seen in the current literature on H-transfer processes in enzyme mechanisms and in oxidative damage to proteins. These observations are discussed in terms of the findings of the present study.</p><h2><a href=https://www.ncbi.nlm.nih.gov/pubmed/12112677>Monte Carlo simulations of the peptide recognition at the consensus binding site of the constant fragment of human immunoglobulin G: the  landscape analysis of a hot spot at the intermolecular interface.</a></h2><p>Monte Carlo simulations of molecular recognition at the consensus binding site of the constant fragment (Fc) of human immunoglobulin G (Ig) protein have been performed to analyze structural and thermodynamic aspects of binding for the 13-residue cyclic peptide DCAWHLGELVWCT. The  landscape analysis of a hot spot at the intermolecular interface using alanine scanning and equilibrium-simulated tempering dynamics with the simplified, knowledge-based  function has enabled the role of the protein hot spot residues in providing the thermodynamic stability of the native structure to be determined. We have found that hydrophobic interactions between the peptide and the Met-252, -253, His-433, and His-435 protein residues are critical to guarantee the thermodynamic stability of the crystallographic binding mode of the complex. Binding free  calculations, using a molecular mechanics force field and a solvation  model, combined with alanine scanning have been conducted to determine the energetic contribution of the protein hot spot residues in binding affinity. The conserved Asn-434, Ser-254, and -436 protein residues contribute significantly to the binding affinity of the peptide-protein complex, serving as an energetic hot spot at the intermolecular interface. The results suggest that evolutionary conserved hot spot protein residues at the intermolecular interface may be partitioned in fulfilling thermodynamic stability of the native binding mode and contributing to the binding affinity of the complex.Copyright 2002 Wiley-Liss, Inc.</p><h2><a href=https://www.ncbi.nlm.nih.gov/pubmed/8075386>Conformation and sweet tastes of L-aspartyl dipeptide methyl esters.</a></h2><p>In order to investigate the conformational preferences to elicit tastes, conformational free  calculations using an empirical potential (ECEPP/2) and the hydration shell model were carried out on the L-aspartyl dipeptide methyl esters, L-(+)HAsp(-)-L-Xaa-OMe, in the hydrated state, where Xaa includes sweet (Phe, , Met, and Gly), bitter (Ala, Trp, Val, Leu, and ), and tasteless (Ser, Thr, and Abu) residues. The refined preferred conformation of the Phe dipeptide (aspartame) with side chain chi 1/2 conformation g- is g-Fg- in the hydrated state, which is consistent with the structure deduced from 1H-nmr experiments. Irrespective of the Xaa and taste, all the dipeptides have the same conformation for the Asp residue, which is attributable to the hydrogen bond between protonated amino hydrogen and carboxylate oxygen and the favored hydration of the carboxylate group. This implies that the L-aspartyl residue is a necessary factor for the dipeptides to be sweet not a sufficient factor. The computed conformational preferences for sweet, bitter, and tasteless dipeptides in the hydrated state indicate to us that the conformation about the N--C alpha bond of the Xaa residue, i.e., the orientation of the hydrophobic moiety with respect to the AH/B functionalities in the aspartyl moiety, seems to be crucial to elicit the tastes. In addition, the hydrophobicity and the size of the Xaa residue are found to play a major role in determining the tastes. These well accord with the related works reported previously.</p><h2><a href=https://www.ncbi.nlm.nih.gov/pubmed/25707212>[The influence of immunization of rats with BSA-conjugated peptide 269-280 of type 3 melanocortin receptor on the metabolic parameters and thyroid functions].</a></h2><p>One of the approaches to study the role of the brain hormonal signaling systems in the regulation of biochemical and physiological processes is their shutdown using the antibodies generated to peptides corresponding to extracellular regions of receptors. The brain type 3 melanocortin receptors (M3R) play an important role in the central regulation of the metabolism and the endocrine system. However, the influence of prolonged inhibition of M3R on  metabolism, insulin resistance, and thyroid gland (TG) function is practically not studied. The aim of the study was to investigate the influence of prolonged repeated immunization of male rats with the BSA-conjugated peptide Ala-[Pro-Thr-Asn-Pro--Cys--Cys-Thr-Thr-Ala-His269-280]-Ala (A[269- 280]A) corresponding to the third extracellular loop of M3R on their metabolic parameters and functional activity of TG. 9 months after the first immunization, the weight of rats was reduced and after 12-13 months was significantly lower than in controls. The weight of abdominal and brown adipose tissues, on the contrary, increased. At the same timeline there was an increase in the fasting glucose and insulin levels, and increase of the HOMA-IR index (by 75%) indicating that immunized animals develop insulin resistance. The rats have increased glucose utilization due to an increase of insulin production by pancreatic β-cells. 12 months after the first immunization, the increase of the triglycerides level (by 74%) and the ratio of LDL- and HDL-cholesterol (by 36%) were revealed. 13 months after the start of immunization, the levels of free and total thyroxine and total triiodothyronine significantly decreased. In the TG plasma membranes of immunized rats the weakening adenylyl cyclase stimulating effect of thyroid-stimulating hormone was detected. Thus, long-term decrease in the bra- in M3R activity due to repeated immunization of rats with BSA-conjugated peptide A[269-280]A induces the disturbances of the peripheral metabolism and TG function.</p><h2><a href=https://www.ncbi.nlm.nih.gov/pubmed/1814697>Simulation analysis of the stability mutants R96H of bacteriophage T4 lysozyme and I96A of barnase.</a></h2><p>Free  simulation methods are used to analyse the effects of the mutation Arg-96----His on the stability of bacteriophage T4 lysozyme and of -96----Ala on the stability of barnase. By use of thermodynamic integration, the contributions of specific interactions to the free  change are evaluated. It is shown that a number of contributions that stabilize the wild-type or the mutant partially cancel in the overall free  difference; some of these involve the unfolded state. Comparison of the results with conclusions based on structural and thermodynamic data leads to new insights into the origin of the stability difference between wild-type and mutant proteins. For the charged-to-charged amino acid mutation in T4 lysozyme, the importance of the contributions of more distant residues, solvent water and the covalent linkage involving the mutated amino acid are of particular interest. Also, the analysis of the Arg-96 to His mutation with respect to the interactions with the C-terminal end of a helix (residues 82-90) indicates that the nearby carbonyl groups (-88 and Asp-89) make the dominant contribution, that the amide groups do not contribute significantly and that the helix dipole model is inappropriate for this case. For the non-polar-to-non-polar amino acid mutation in barnase, the solvent contribution is unimportant, and covalent terms are shown to be significant because they do not cancel between the folded and unfolded state.</p><h2><a href=https://www.ncbi.nlm.nih.gov/pubmed/2183216>Understanding structural relationships in proteins of unsolved three-dimensional structure.</a></h2><p>The locations of functionally important sequences and general structural motifs have been assigned to -tRNA synthetase. However, a function has not been established for some segments of the protein (e.g., CP1). The method of structural modeling described here cannot establish the details of a 3 A crystal structure, and, in contrast to a crystal structure, the precision of the model varies according to the extent of a sequence similarity or the functional importance of a region. In -tRNA synthetase, the signature sequence and the flanking regions are likely to be similar in structure to the proteins on which the model is based. For other regions, it may be possible to build a three-dimensional model by connecting well defined regions and refining the positions of the connecting elements by  minimization. Structural modelling of this kind must be done cautiously, because the order and orientation of the elements of a structural motif can change in subtle ways. In the case of -tRNA synthetase, the beta-strand nearest the N-terminus is the outermost strand of the nucleotide binding fold; in Met-tRNA synthetase, the same strand is innermost. Furthermore, the orientation of this strand may be antiparallel (-tRNA synthetase) or parallel (Met-tRNA synthetase). Because multiple structures that differ in their orientations of structural elements are possible, the structural analogies between proteins should not be naively extrapolated without independent experimental support. As described above, some regions of proteins tolerate internal deletions and insertions. This provides further experimental support for the practice of allowing for gaps in computer-generated sequence alignments. Nevertheless, because some regions are more tolerant of insertions and deletions than others, the structural and functional significance of a region of broken alignment must be assessed carefully. All gaps in sequence alignments cannot be treated equally, and each must be evaluated within its own context. In the synthetases of known structure, structural analogy can be used to identify important functional elements. For example, the amino acid binding site of Met-tRNA synthetase might be formed, at least in part, by a peptide that encompasses Ala50; this amino acid aligns with Gly94 of the -tRNA synthetase. This is an example in which results on a protein of unknown structure (-tRNA synthetases) can lead to identification of a potential substrate binding site in a protein of known structure (Met-tRNA synthetase).</p><h2><a href=https://www.ncbi.nlm.nih.gov/pubmed/15905365>tRNA actively shuttles between the nucleus and cytosol in yeast.</a></h2><p>Previous evidence suggested that transfer RNAs (tRNAs) cross the nuclear envelope to the cytosol only once after maturing in the nucleus. We now present evidence for nuclear import of tRNAs in yeast. Several export mutants accumulate mature tRNAs in the nucleus even in the absence of transcription. Import requires  but not the Ran cycle. These results indicate that tRNAs shuttle between the nucleus and cytosol.</p><h2><a href=https://www.ncbi.nlm.nih.gov/pubmed/20023126> and amino acid utilization in expeller-extracted canola meal fed to growing pigs.</a></h2><p>Two experiments were conducted to determine the nutritive value of expeller-extracted canola meal (EECM) for growing pigs. In Exp. 1, a total of 6 ileally cannulated barrows (average initial BW = 26.8 kg) were fed 3 diets in a replicated 3 x 3 Latin square design to determine the apparent and standardized ileal digestibility (SID) values of N and AA and the SID AA contents of EECM. The 3 diets were a cornstarch-based diet with either solvent-extracted canola meal (SECM) or EECM as the sole source of protein, and a low-casein cornstarch-based diet, which was used to estimate basal endogenous N and AA losses to determine the SID of N and AA. All 3 diets contained chromic oxide as an indigestible marker for determining nutrient digestibility by the indicator method. In Exp. 2, a total of 18 intact barrows (average initial BW = 25.9 kg) were fed 3 diets in a completely randomized design (6 pigs per diet) to determine apparent total tract digestibility and retention of nutrients and the DE and ME contents of EECM. The diets were a basal corn-based diet or the basal diet with corn replaced by 35% SECM or EECM. The basal diet was used for determining the total tract digestible nutrient content by the difference method. Solvent-extracted canola meal, which is commonly used in the formulation of swine diets, was fed in both experiments for comparison with EECM. The SECM and EECM were similar in CP content (41.8 vs. 41.4%). Expeller-extracted canola meal was, however, greater in ether extract content (12.03 vs. 5.54%) and decreased in NDF content (23.8 vs. 29.9%) compared with SECM. The EECM also had a greater content of all the AA except Met, Cys, and Ser, by approximately 6.6%; Cys was greater in SECM, whereas Met and Ser were similar between the 2 meals. The EECM had greater (P < 0.05) SID of N, Arg, , Leu, Phe, Glu, and Pro. The SID contents of Arg, His, , Leu, Phe, Val, Ala, Asp, Glu, Gly, Pro, and  were also greater (P < 0.05) for EECM than for SECM by an average of 15%. The EECM had greater (P < 0.01) DE (4,107 vs. 3,790 kcal/kg) and ME (3,978 vs. 3,564 kcal/kg) values compared with SECM. The results show that the EECM used in the current study had greater digestible AA and  and a greater ME content than the SECM; hence, it may be a better source of protein and  for growing pigs than SECM.</p><h2><a href=https://www.ncbi.nlm.nih.gov/pubmed/17313369>The role of Tyr605 and Ala607 of thimet oligopeptidase and Tyr606 and Gly608 of neurolysin in substrate hydrolysis and inhibitor binding.</a></h2><p>The physicochemical properties of TOP (thimet oligopeptidase) and NEL (neurolysin) and their hydrolytic activities towards the FRET (fluorescence resonance  transfer) peptide series Abz-GFSXFRQ-EDDnp [where Abz is o-aminobenzoyl; X=Ala, , Leu, Phe, , Trp, Ser, Gln, Glu, His, Arg or Pro; and EDDnp is N-(2,4-dinitrophenyl)-ethylenediamine] were compared with those of site-mutated analogues. Mutations at Tyr605 and Ala607 in TOP and at Tyr606 and Gly608 in NEL did not affect the overall folding of the two peptidases, as indicated by their thermal stability, CD analysis and the pH-dependence of the intrinsic fluorescence of the protein. The kinetic parameters for the hydrolysis of substrates with systematic variations at position P1 showed that Tyr605 and Tyr606 of TOP and NEL respectively, played a role in subsite S1. Ala607 of TOP and Gly608 of NEL contributed to the flexibility of the loops formed by residues 600-612 (GHLAGGYDGQYYG; one-letter amino acid codes used) in NEL and 599-611 (GHLAGGYDAQYYG; one-letter amino acid codes used) in TOP contributing to the distinct substrate specificities, particularly with an isoleucine residue at P1. TOP Y605A was inhibited less efficiently by JA-2 {N-[1-(R,S)-carboxy-3-phenylpropyl]Ala-Aib--p-aminobenzoate}, which suggested that the aromatic ring of Tyr605 was an important anchor for its interaction with wild-type TOP. The hydroxy groups of Tyr605 and Tyr606 did not contribute to the pH-activity profiles, since the pKs obtained in the assays of mutants TOP Y605F and NEL Y606F were similar to those of wild-type peptidases. However, the pH-kcat/Km dependence curve of TOP Y605A differed from that of wild-type TOP and from TOP Y606F. These results provide insights into the residues involved in the substrate specificities of TOP and NEL and how they select cytosolic peptides for hydrolysis.</p><h2><a href=https://www.ncbi.nlm.nih.gov/pubmed/24706971>Dietary electrolyte balance influences ileal endogenous amino acid losses in broiler chickens.</a></h2><p>The objective of the current study was to determine the effect of nitrogen-free diets formulated with 2 ratios of corn starch and dextrose and 2 levels of dietary electrolyte balance (DEB; calculated as Na + K - Cl(-), in mEq/kg of diet) on ileal endogenous amino acid (EAA) losses in 48-d-old broiler chickens. On d 43, 240 broiler chickens were allotted to 4 dietary treatments with 6 replicate cages per diet in a completely randomized design. Each experimental diet was fed for 5 d (d 43-48). All diets were free of nitrogen with 2 ratios of corn starch-to-dextrose (0.31 and 1.04) and 2 levels of DEB (108 and 219 mEq/kg of diet). Final BW and BW loss between birds fed the 2 corn starch-to-dextrose ratios and between birds fed 2 levels of DEB were not different (P > 0.05). Birds fed the diet containing a high level of DEB consumed less (P < 0.05) feed than birds on the low-DEB diet. Birds on the high-corn starch-to-dextrose (1.04) diet showed a tendency for higher (P = 0.08) feed intake. High dietary dextrose level (corn starch-to-dextrose ratio of 0.31) resulted in higher (P < 0.05) ileal DM and  digestibility. A high level of DEB resulted in higher (P < 0.05) ileal endogenous nitrogen loss. Arginine, , Leu, Phe, Val, Ala, Glu, Gly, Pro, and  secretion into the gut increased (P ≤ 0.05) with an increasing level of DEB. Endogenous His, Lys, Thr, Asp, Cys, and Ser showed a tendency for increased (P ≤ 0.1) losses with a high level of DEB. Mean ileal EAA losses for the indispensable and dispensable amino acids for the low-DEB diet were 81 and 82%, respectively, that of the birds fed the high-DEB diet. The 4 amino acids with the lowest endogenous flow were Met, His, , and Cys, whereas the highest endogenous flow was found in Glu, Asp, Val, and Leu. Data from the current study showed that EAA losses in the ileum of broiler chickens are significantly affected by DEB, but not by the ratio of corn starch to dextrose.</p><h2><a href=https://www.ncbi.nlm.nih.gov/pubmed/17620608>Potent cross-reactive neutralization of SARS coronavirus isolates by human monoclonal antibodies.</a></h2><p>The severe acute respiratory syndrome coronavirus (SARS-CoV) caused a worldwide epidemic in late 2002/early 2003 and a second outbreak in the winter of 2003/2004 by an independent animal-to-human transmission. The GD03 strain, which was isolated from an index patient of the second outbreak, was reported to resist neutralization by the human monoclonal antibodies (hmAbs) 80R and S3.1, which can potently neutralize isolates from the first outbreak. Here we report that two hmAbs, m396 and S230.15, potently neutralized GD03 and representative isolates from the first SARS outbreak (Urbani, Tor2) and from palm civets (SZ3, SZ16). These antibodies also protected mice challenged with the Urbani or recombinant viruses bearing the GD03 and SZ16 spike (S) glycoproteins. Both antibodies competed with the SARS-CoV receptor, ACE2, for binding to the receptor-binding domain (RBD), suggesting a mechanism of neutralization that involves interference with the SARS-CoV-ACE2 interaction. Two putative hot-spot residues in the RBD (-489 and -491) were identified within the SARS-CoV spike that likely contribute to most of the m396-binding . Residues -489 and -491 are highly conserved within the SARS-CoV spike, indicating a possible mechanism of the m396 cross-reactivity. Sequence analysis and mutagenesis data show that m396 might neutralize all zoonotic and epidemic SARS-CoV isolates with known sequences, except strains derived from bats. These antibodies exhibit cross-reactivity against isolates from the two SARS outbreaks and palm civets and could have potential applications for diagnosis, prophylaxis, and treatment of SARS-CoV infections.</p><h2><a href=https://www.ncbi.nlm.nih.gov/pubmed/30566595>Effect of inclusion level of corn germ meal on the digestible and metabolizable  and evaluation of ileal AA digestibility of corn germ meal fed to growing pigs.</a></h2><p>Two experiments were conducted to investigate the effect of inclusion level of corn germ meal (CGM) on the DE and ME values of CGM and to evaluate the ileal AA digestibility of CGM fed to growing pigs. In Exp. 1, 42 barrows (63.8 ± 2.1 kg BW) were allotted to seven diets in a completely randomized design with six replicates per diet. Diets included a corn-soybean meal (SBM) diet and six additional diets containing 4.85%, 9.70%, 19.40%, 29.10%, 38.80%, or 48.50% CGM. Pigs were fed twice daily, at 0730 and 1630 hours, at a level of 4% of BW, and feces and urine were collected for 5 d. The apparent total tract digestibility (ATTD) of DM, OM, CP, acid-hydrolyzed ether extract, and the DE and ME in diets linearly decreased (P < 0.01) as dietary CGM increased. Inclusion level of CGM had no effect on the DE and ME values of CGM. On a DM basis, the concentration of DE and ME varied from 3,396 to 3,747 kcal/kg and 3,107 to 3,502 kcal/kg, respectively. In Exp. 2, 11 crossbred barrows (30.4 ± 2.9 kg BW) with a T-cannula in the distal ileum were allotted to an 11 × 6 Youden square design with 11 diets and 6 period, which included an N-free diet and 10 CGM test diets. Chromic oxide (0.3%) was included in all diets as an indigestible marker. Pigs were fed daily at 4% of BW during each period, which consisted of 5 d of diet adaptation followed by 2 d of digesta collection. The apparent ileal digestibility (AID) of , Thr, and Ala and the standardized ileal digestibility (SID) of CP and  varied (P < 0.01) among the 10 CGM. The AID of CP and all AA except Pro and , and the SID of all AA except Pro were the greatest (P < 0.05) in sample 7. The AID and SID of CP averaged 40.47% and 64.75%, respectively, and varied from 32.30% to 54.87% and from 57.48% to 72.15%, respectively. The average SID of Lys, Met, Thr, and Trp was 65.61%, 76.15%, 65.29%, and 60.17%, respectively, with a SEM of 4.49, 2.40, 5.95, and 6.82, respectively. The average SID of Pro was 101.76%, with an SEM of 17.26. Increasing dietary CGM decreased the ATTD of nutrients, the DE and ME values of diets but CGM. The AID and SID of AA in CGM are low but source dependent, and CGM may be fed to pigs as an protein-rich feedstuff.</p><h2><a href=https://www.ncbi.nlm.nih.gov/pubmed/7612610>Interactions of hirudin-based inhibitor with thrombin: critical role of the IleH59 side chain of the inhibitor.</a></h2><p>Hirudin is the most potent and specific thrombin inhibitor from medicinal leech with a Ki value of 2.2 x 10(-14) M. It consists of an active site inhibitor segment, hirudin1-48, a fibrinogen-recognition exosite inhibitor segment, hirudin55-65, and a linker, hirudin49-54, connecting these inhibitor segments. The role of the side chain of the hirudin 59th residue, , is studied by using a series of synthetic bivalent thrombin inhibitors, which mimic the binding mode of hirudin. The synthetic inhibitors based on the hirudin sequence have a general sequence of Ac-(D-Phe)-Pro-Arg-Pro-(4-aminobutyric acid)-(7-amino-heptanoic acid)-Asp-Phe-Glu-Glu-Xaa-Pro-Glu-Glu--Leu-Gln-OH, in which the 59th residue, Xaa, is substituted by various natural and unnatural L-amino acids. For example, substitution of IleH59 by Val, which is equivalent to removing the delta-methyl group of IleH59, reduces the affinity of the inhibitor 5.7-fold (delta delta G0 = 1.0 kcal/mol) to a Ki value of 4.7 nM compared to that (Ki = 0.82 nM) of the corresponding inhibitor with IleH59. Removal of the entire side chain of IleH59, i.e., a substitution of IleH59 by Gly, reduces the affinity of the inhibitor 6300-fold, revealing the critical role of the IleH59 side chain in the inhibitor binding. Theoretical free  calculation successfully reproduces the binding free  of most of the analogs. It suggests that intra- and intermolecular van der Waals interactions of delta-CH3, gamma-CH3, and gamma-CH2 of IleH59 play the major role in the binding affinity.(ABSTRACT TRUNCATED AT 250 WORDS).</p><h2><a href=https://www.ncbi.nlm.nih.gov/pubmed/12068020>The thrombin epitope recognizing thrombomodulin is a highly cooperative hot spot in exosite I.</a></h2><p>The functional epitope of thrombin recognizing thrombomodulin was mapped using Ala-scanning mutagenesis of 54 residues located around the active site, the Na(+) binding loop, the 186-loop, the autolysis loop, exosite I, and exosite II. The epitope for thrombomodulin binding is shaped as a hot spot in exosite I, centered around the buried ion quartet formed by Arg(67), Lys(70), Glu(77), and Glu(80), and capped by the hydrophobic residues (76) and (82). The hot spot is a much smaller subset of the structural epitope for thrombomodulin binding recently documented by x-ray crystallography. Interestingly, the contribution of each residue of the epitope to the binding free  shows no correlation with the change in its accessible surface area upon formation of the thrombin-thrombomodulin complex. Furthermore, residues of the epitope are strongly coupled in the recognition of thrombomodulin, as seen for the interaction of human growth hormone and insulin with their receptors. Finally, the Ala substitution of two negatively charged residues in exosite II, Asp(100) and Asp(178), is found unexpectedly to significantly increase thrombomodulin binding.</p><h2><a href=https://www.ncbi.nlm.nih.gov/pubmed/12146486>Evaluation of isoleucine, leucine, and valine as a second-limiting amino acid for milk production in dairy cows fed grass silage diet.</a></h2><p>Five Finnish ruminally cannulated Ayrshire cows were used in a 5 x 5 Latin square trial with 14-d periods to determine whether branched-chain amino acids (AA) are the second- or colimiting AA for milk protein synthesis on grass silage-cereal based diet. Mammary metabolism of AA as well as AA supply from the basal diet were also studied. Grass silage (17.5% crude protein) was given ad libitum with 9 kg/d as a cereal-based concentrate (13.8% crude protein). Treatments were basal diet without AA infusion (Control), abomasal infusion of AA mixture of His, , Leu, and Val at 8.5, 14.9,27.9, and 18.3 g/d, respectively, AA mixture minus , AA mixture minus Leu, and AA mixture minus Val. Glucose was infused on all treatments at 250 g/d. Amino acid infusions had no effect on dry matter intake (mean 19.2 kg/d), yields of milk (mean 25.3 kg/d), -corrected milk (mean 25.9 kg/d), milk protein (mean 807 g/d), lactose (mean 1261 g/d), or fat (mean 1056 g/d). Milk composition was not affected by the treatments. Plasma concentrations of His and Val responded to AA infusions but concentration of  increased only on treatment AA mixture minus Leu, and concentration of Leu only on treatment AA mixture minus . Infusion of AA mixture of His, , Leu, and Val decreased plasma concentrations of Arg, Lys, Met, Phe, and . Amino acid infusions did not affect concentrations of plasma urea and  metabolites or AA utilization by the mammary gland. Based on unchanged production parameters, the supply of His or branched-chain AA seemed not to be limiting under the current dietary conditions. Changes in plasma AA concentrations suggest either antagonism between individual AA in absorption or increased partitioning of AA into the muscle tissues. About 75% of omasal canal nonammonia nitrogen flow (427 g/d) was of microbial origin, and AA profiles of microbial protein and omasal canal digesta were fairly similar. Postruminal AA supply seems to be dependent on the basal diet, but variation may exist even within the similar basal diets.</p><h2><a href=https://www.ncbi.nlm.nih.gov/pubmed/10993733>II. Structure and specificity of the interaction between the FHA2 domain of Rad53 and phosphotyrosyl peptides.</a></h2><p>The forkhead-associated (FHA) domain is a protein module found in many proteins involved in cell signaling in response to DNA damage. It has been suggested to bind to pThr sites of its target protein. Recently we have determined the first structure of an FHA domain, FHA2 from the yeast protein Rad53, and demonstrated that FHA2 binds to a pTyr-containing peptide (826)EDI(pY)YLD(832) from Rad9, with a moderate affinity (K(d) ca. 100 microM). We now report the solution structure of the complex of FHA2 bound with this pTyr peptide. The structure shows that the phosphate group of pTyr interacts directly with three arginine residues (605, 617, and 620), and that the leucine residue at the +2 position from the pTyr interacts with a hydrophobic surface on FHA2. The sequence specificity of FHA2 was determined by screening a combinatorial pTyr library. The results clearly show that FHA2 recognizes specific sequences C-terminal to pTyr with the following consensus: XX(pY)N(1)N(2)N(3), where N(1)=Leu, Met, Phe, or , N(2)=, Phe, Leu, or Met, and N(3)=Phe, Leu, or Met. Two of the selected peptides, GF(pY)LYFIR and DV(pY)FYMIR, bind FHA2 with K(d) values of 1.1 and 5.0 microM, respectively. The results, along with other recent reports, demonstrate that the FHA domain is a new class of phosphoprotein-binding domain, capable of binding both pTyr and pThr sequences.Copyright 2000 Academic Press.</p><h2><a href=https://www.ncbi.nlm.nih.gov/pubmed/12082163>Thermodynamic characterization of variants of mesophilic cytochrome c and its thermophilic counterpart.</a></h2><p>Thermal stability was measured for variants of cytochrome c-551 (PA c-551) from a mesophile, Pseudomonas aeruginosa, and a thermophilic counterpart, Hydrogenobacter thermophilus cytochrome c-552 (HT c-552), by differential scanning calorimetry (DSC) at pH 3.6. The mutated residues in PA c-551, selected with reference to the corresponding residues in HT c-552, were located in three spatially separated regions: region I, Phe7 to Ala/Val13 to Met; region II, Glu34 to /Phe43 to ; and region III, Val78 to . The thermodynamic parameters determined indicated that the mutations in regions I and III caused enhanced stability through not only enthalpic but also entropic contributions, which reflected improved packing of the side chains. Meanwhile, the mutated region II made enthalpic contributions to the stability through electrostatic interactions. The obtained differences in the Gibbs free  changes of unfolding [Delta(DeltaG)] showed that the three regions contributed to the overall stability in an additive manner. HT c-552 had the smallest heat capacity change (DeltaC(P)), resulting in higher DeltaG values over a wide temperature range (0-100 degrees C), compared to the PA c-551 variants; this contributed to the highest stability of HT c-552. Our DSC measurement results, in conjunction with mutagenesis and structural studies on the homologous mesophilic and thermophilic cytochromes c, provided an extended thermodynamic view of protein stabilization.</p><h2><a href=https://www.ncbi.nlm.nih.gov/pubmed/28657434>Quick elucidation of cyclodepsipeptide sequence from sacoglossan Elysia grandifolia using electrospray ionisation-tandem mass spectrometry.</a></h2><p>Butanol fraction of sacoglossan Elysia grandifolia was investigated for identifying peptides using electrospray ionisation-tandem mass spectrometry (ESI-MS/MS). Without prior isolation, the structural determination is achieved on the basis of mass fragmentation pattern and comparison with the previously established data. The ESI-MS of the fraction in the positive ion mode gave clusters of singly and doubly charged molecular ion peaks. The ESI-MS spectrum showed peaks for the presence of the peptides kahalalides F, G, R and S reported earlier. In addition, it also showed molecular ion peaks at m/z 1557.8 [M+H] and doubly charged ions at m/z 779.4 [M+2H], 790.4 [M+Na] and 796.4 [M+K]. The MS/MS at m/z 779.4 [M+2H] at collision  40 V obtained series of b and y fragment ions. The MS/MS spectrum showed identical fragment ion y at m/z 643 which revealed that cyclic part is identical with kahalalide F, R and S. Careful examination of the fragment ions b to b with their corresponding y fragment ions y to y, respectively and by comparison of MS/MS pattern of kahalalide S, established that proline can be replaced by tyrosine amino acid residue. The mass difference between b ( m/z 511) and b ( m/z 674) is equal to 163 which is equivalent to mass residue of tyrosine. Their y fragment ions also quickly helped in fixing the puzzle. This resulted in the identification of the peptide sequence cyclo-[Val-(5-MeHex-Val-Thr-Val-Val--Lys-)Thr--Val-Phe-Dhb)] for the new cyclodepsipeptide, kahalalide Z. Thus, ESI-MS/MS has set a trend in quick identification of new marine molecules.</p><h2><a href=https://www.ncbi.nlm.nih.gov/pubmed/17718594>Modeling study of Rusticyanin-Cytochrome C(4) complex: an insight to possible H-bond mediated recognition and electron--transfer process.</a></h2><p>Rusticyanin (RCy) mediated transfer of electron to Cytochrome C(4) (Cytc(4)) from the extracellular Fe(+2) ion is primarily involved in the Thiobacillus ferrooxidans induced bio-leaching of pyrite ore and also in the metabolism of this acidophilic bacteria. The modeling studies have revealed the two possible mode of RCy-Cytc(4) complexation involving nearly the same stabilization  approximately -15 x 10(3) kJ/mol, one through N-terminal Asp 15 and another -C terminal Glu 121 of Cytc(4) with the Cu-bonded His 143 of RCy. The Asp 15:His 143 associated complex (DH) of Cytc(4)-RCy was stabilized by the intermolecular H-bonds of the carboxyl oxygen atoms O(delta1) and O(delta2) of Asp 15 with the Nepsilon-atom of His 143 and O(b) atoms of Ala 8 and Asp 5 (of Cytc(4)) with the Thr 146 and Phe 51 (of RCy). But the other Glu 121:His 143 associated complex (EH) of Cytc(4)-RCy was stabilized by the H-bonding interaction of the oxygen atoms O(epsilon1) and O(epsilon2) of Glu 121 with the Nepsilon and Ogamma atoms of His 143 and Thr 146 of RCy. The six water molecules were present in the binding region of the two proteins in the  minimized autosolvated DH and EH-complexes. The MD studies also revealed the presence of six interacting water molecules at the binding region between the two proteins in both the complexes. Several residues Gly 82 and 84, His 143 (RCy) were participated through the water mediated (W 389, W 430, W 413, W 431, W 373, and W 478) interaction with the Asp 15,  82, and 62,  63 (Cytc(4)) in DH complex, whereas in EH complex the Phe 51, Asn 80,  146 (RCy) residues were observed to interact with Asn 108, Met 120, Glu 121 (of Cytc(4)) through the water molecules W 507, W 445, W 401, W 446, and W 440. The direct water mediated (W 478) interaction of His 143 (RCy) to Asp 15 (of Cytc(4)) was observed only in the DH complex but not in EH. These direct and water mediated H-bonding between the two respective proteins and the binding free  with higher interacting buried surface area of the DH complex compare to other EH complex have indicated an alternative possibility of the electron transfer route through the interaction of His 143 of RCy and the N-terminal Asp 15 of Cytc(4).</p><h2><a href=https://www.ncbi.nlm.nih.gov/pubmed/21257055>Deletion of arginine from an abomasal infusion of amino acids does not decrease milk protein yield in Holstein cows.</a></h2><p>This study was undertaken to determine if a limited supply of Arg would alter milk and milk protein yields, as well as mammary uptake of AA and energetic substrates. Six lactating Holstein cows (199 ± 5 d in milk) were used in a replicated 3 × 3 Latin square balanced for residual effects with 14-d periods. The diet was formulated to supply 100% of the National Research Council net  requirement and 72% of the metabolizable protein requirement. The treatments were randomly distributed as abomasal infusions of (1) water (CTL), (2) a mixture of essential AA (EAA) excluding Arg (ARG-), or (3) a mixture of EAA including Arg (ARG+). The profile of EAA in the infusates was the same as that found in casein with the exception that methionine was increased to maintain a 3:1 ratio of digestible lysine:methionine (total dietary+infusion). Milk protein yield was increased by the ARG+ compared with the CTL treatment and deletion of Arg in the infusate (ARG-) did not impair this response. Deletion of Arg from the EAA mixture decreased the mammary uptake of Arg relative to that of the CTL treatment, and although the uptake:output ratio decreased from 2.52 (ARG+) to 2.12 (ARG-), it was still largely in excess of Arg secretion in milk protein. Otherwise, Arg deletion did not affect any of the measured parameters (no significant difference between ARG- and ARG+) except Arg and urea arterial concentrations. In support of the increased yields of milk protein and lactose, mammary uptake of the group 2 AA (, Leu, Lys, and Val) increased and the uptake:output ratio tended to increase from 1.04 to 1.23. The mammary uptake:milk protein output ratio was not different from 1 and not different among treatments for the group 1 AA (His, Met, Phe+, Trp). Mammary uptake of energetic substrates did not vary across treatments, although milk lactose yield increased with the ARG+ treatment relative to CTL. These results indicate that deletion of Arg has minimal effects on milk and milk component yields when the remaining EAA are supplied in sufficient amounts despite decreased mammary Arg uptake and that group 2 AA seem to be involved in the mammary gland to support the lactose yield response to EAA infusion.Copyright Â© 2011 American Dairy Science Association. Published by Elsevier Inc. All rights reserved.</p><h2><a href=https://www.ncbi.nlm.nih.gov/pubmed/31098784>Discrimination power of knowledge-based potential dictated by the dominant energies in native protein structures.</a></h2><p>Extracting a well-designed energy function is important for protein structure evaluation. Knowledge-based potential functions are one type of the energy functions which can be obtained from known protein structures. The pairwise potential between atom types is approximated using Boltzmann's law which relates the frequency of atom types to its potential. The total energy is approximated as a summation of pairwise potential between the atomic pairs. In the present study, the performance of knowledge-based potential function was assessed based on the strength of interaction between groups of amino acids. The dominant energies involved in the pairwise potentials were revealed by eigenvalue analysis of the matrix, the elements of which represent the energy between amino acids. For this purpose, the matrix including the mean of the energies of residue-residue interaction types was constructed using 500 native protein structures. The matrix has a dominant eigenvalue and amino acids, with LEU, VAL, , PHE, , ALA and TRP having high values along the dominant eigenvector. The results show that the ranking of amino acids is consistent with the power of amino acids in discriminating native structures using K-alphabet reduced model. In the reduced interactions, only amino acids from a subset of all 20 amino acids, along with their interactions are considered to assess the energy. In the K-alphabet reduced model, the reduced structures are constructed based on only the K-amino acid types. The dominant K-alphabet reduced model derived for the k-first amino acids in the list [LEU, VAL, PHE, , , ALA, TRP] of amino acids has the best discrimination of native structure among all possible K-alphabet reduced models. Knowledge-based potentials might be improved with a new strategy.</p><h2><a href=https://www.ncbi.nlm.nih.gov/pubmed/17910473>NMR solution structure of subunit F of the methanogenic A1AO adenosine triphosphate synthase and its interaction with the nucleotide-binding subunit B.</a></h2><p>The A1AO adenosine triphosphate (ATP) synthase from archaea uses the ion gradients generated across the membrane sector (AO) to synthesize ATP in the A3B3 domain of the A1 sector. The  coupling between the two active domains occurs via the so-called stalk part(s), to which the 12 kDa subunit F does belong. Here, we present the solution structure of the F subunit of the A1AO ATP synthase from Methanosarcina mazei Gö1. Subunit F exhibits a distinct two-domain structure, with the N-terminal having 78 residues and residues 79-101 forming the flexible C-terminal part. The well-ordered N-terminal domain is composed of a four-stranded parallel beta-sheet structure and three alpha-helices placed alternately. The two domains are loosely associated with more flexibility relative to each other. The flexibility of the C-terminal domain is further confirmed by dynamics studies. In addition, the affinity of binding of mutant subunit F, with a substitution of Trp100 against  and  at the very C-terminal end, to the nucleotide-binding subunit B was determined quantitatively using the fluorescence signals of natural subunit B (Trp430). Finally, the arrangement of subunit F within the complex is presented.</p><h2><a href=https://www.ncbi.nlm.nih.gov/pubmed/24202884>Use of low-and high- collision-induced dissociation tandem mass spectrometry in the identification of an unusual amino acid in a semisynthetic polypeptide.</a></h2><p>During production of the semisynthetic eicosapeptide Phe-Pro-Arg-Pro-Gly-Gly-Gly-Gly-Asn-Gly-Asp-Phe-Glu-Glu--Pro-Glu-Glu--Leu, two minor impurities were isolated in which -15 had been replaced by other amino acids. The molecular weights of the two peptide minor products were 14 u lower and 14 u higher, respectively, than the major product with the foregoing amino acid sequence. It was determined that the first, lower molecular weight, impurity contained Val instead of  at position 15 of the sequence, whereas the second, higher molecular weight, impurity contained an unusual amino acid at the same position. This amino acid, which was characterized from the low-and high- collision-induced dissociation product ion tandem mass spectra of the second peptide impurity, is an α-amino acid with an asymmetric side chain branched at the β-carbon.</p><h2><a href=https://www.ncbi.nlm.nih.gov/pubmed/7668386>Increase in fluorescence upon the hydrolysis of tyrosine peptides: application to proteinase assays.</a></h2><p>The intrinsic fluorescence of tyrosine increases by a factor of approximately two when the carboxy group is liberated from a peptide bond by hydrolysis. The increase in fluorescence provides a novel way to monitor the hydrolysis of native tyrosine peptides that contain only proteinogenic amino acids. Thus, for example, the hydrolysis by HIV-1 proteinase of a heptapeptide viral protein fragment gag129-135, Ser-Gln-Asn--Pro--Val, was followed continuously at excitation and emission wavelengths 275 and 305 nm. The fluorescence increase is magnified by at least a factor of a thousand when a resonance  quencher, such as paranitrophenylalanine, is in the vicinity. For example, the peptide Lys-Ala-Arg-Val--Phe(p-NO2)-Glu-Ala-Nle-NH2 [Richards et al. (1990) J. Biol. Chem. 265, 7733], widely used for spectrophotometric assays of the HIV-1 proteinase, yields a substrate:product fluorescence ratio greater than 1:1000. Tyrosine-containing substrates of pepsin and trypsin showed similar behavior. The detection limit of the present method is at least one order of magnitude lower than absorbance assays of p-nitrophenylalanine peptides.</p><h2><a href=https://www.ncbi.nlm.nih.gov/pubmed/21866908>Volumetric characterization of interactions of glycine betaine with protein groups.</a></h2><p>We report the partial molar volumes and adiabatic compressibilities of N-acetyl amino acid amides and oligoglycines at glycine betaine (GB) concentrations ranging from 0 to 4 M. We use these results to evaluate the volumetric contributions of amino acid side chains and the glycyl unit (-CH(2)CONH-) as a function of GB concentration. We analyze the resulting GB dependences within the framework of a statistical thermodynamic model and evaluate the equilibrium constant for the reaction in which a GB molecule binds each of the functionalities under study replacing four water molecules. We calculate the free  of the transfer of functional groups from water to concentrated GB solutions, ΔG(tr), as the sum of a change in the free  of cavity formation, ΔΔG(C), and the differential free  of solute-solvent interactions, ΔΔG(I), in a concentrated GB solution and water. Our results suggest that the transfer free , ΔG(tr), results from a fine balance between the large ΔΔG(C) and ΔΔG(I) contributions. The range of the magnitudes and the shape of the GB dependence of ΔG(tr) depend on the identity of a specific solute group. The interplay between ΔΔG(C) and ΔΔG(I) results in pronounced maxima in the GB dependences of ΔG(tr) for the Val, Leu, , Trp, , and Gln side chains as well as the glycyl unit. This observation is in qualitative agreement with the experimental maxima in the T(M)-versus-GB concentration plots reported for ribonuclease A and lysozyme.© 2011 American Chemical Society</p><h2><a href=https://www.ncbi.nlm.nih.gov/pubmed/14691219>Effect of the N3 residue on the stability of the alpha-helix.</a></h2><p>N3 is the third position from the N terminus in the alpha-helix with helical backbone dihedral angles. All 20 amino acids have been placed in the N3 position of a synthetic helical peptide (CH(3)CO-[AAX AAAAKAAAAKAGY]-NH(2)) and the helix content measured by circular dichroism spectroscopy at 273 K. The dependence of peptide helicity on N3 residue identity has been used to determine a free  scale by analysis with a modified Lifson-Roig helix coil theory that includes a parameter for the N3  (n3). The most stabilizing residues at N3 in rank order are Ala, Glu, Met/, Leu, Lys, Ser, Gln, Thr, , Phe, Asp, His, and Trp. Free energies for the most destabilizing residues (Cys, Gly, Asn, Arg, and Pro) could not be fitted. The results correlate with N1, N2, and helix interior energies and not at all with N-cap preferences. This completes our work on studying the structural and energetic preferences of the amino acids for the N-terminal positions of the alpha-helix. These results can be used to rationally modify protein stability, help design helices, and improve prediction of helix location and stability.</p><h2><a href=https://www.ncbi.nlm.nih.gov/pubmed/3284834>Structure-conformation relationships of synthetic peptide inhibitors of human renin studied by resonance  transfer and molecular modeling.</a></h2><p>The structure-conformation relationships of a series of angiotensinogen6-13 (ANG6-13, His-Pro-Phe-His-Leu-Val--His) congeners substituted by Nin-For-Trp (Ftr), D-Ftr or Trp at the N-terminus,  at the C-terminus and Phe psi[CH2NH]Phe at the P1-P'1 cleavage site (i.e. Leu10-Val11) were studied using resonance  transfer coupled with molecular modeling of the peptide conformation using macromolecular  refinement and dynamics simulation. Average end-to-end intramolecular distances (r) of the peptides in solution were determined by fluorescence spectroscopy. For example, Ac-Ftr-Pro-Phe-His-Phe psi[CH2NH]Phe-Val--NH2 (U-70714E) gave an average intramolecular donor ()-acceptor (Ftr) distance of 16.3A in aqueous solution. This experimental value was consistent with a distance of 17.9 A determined by molecular modeling of U-70714E to a human renin 3-D structure (developed from known homologous aspartyl protease inhibitor X-ray crystallographic data) followed by simulation of the solution phase conformation of the peptide. An extended backbone secondary structure of U-70714E is suggested from these studies and the relationship(s) of structure-conformation to structure-activity was explored by analysis of several congeners of U-70714E, a potent (IC50 = 3.0 X 10(-9)M) inhibitor of human renin in vitro.</p><h2><a href=https://www.ncbi.nlm.nih.gov/pubmed/8289284>Determination of alpha-helix propensity within the context of a folded protein. Sites 44 and 131 in bacteriophage T4 lysozyme.</a></h2><p>To determine the effects of different amino acids on the structure and stability of an alpha-helix in the context of a globular protein, all 19 naturally-occurring amino acids were substituted for Ser44 in phage T4 lysozyme. A more restricted set of nine replacements was also made for Val131. Ser44 and Val131 are two of a very limited number of possible sites in T4 lysozyme that are well within alpha-helices, are solvent-exposed and relatively free of interactions with neighboring residues, and are not involved in crystal contacts. High resolution structures for the majority of the mutants, some of which crystallized non-isomorphously with wild-type, were determined. With the exception of proline, the amino acid substitutions caused little if any perturbation of the alpha-helix backbone. Also the beta-branched residues Thr, Val and  show no indication of either side-chain or backbone distortion. Therefore, other than proline, there is no evidence that differences in helix propensities are associated with different amounts of strain introduced into the helix. For reference, and also to allow estimates of side-chain entropy, a survey was made of side-chain conformations in 100 well-refined protein structures. As noted previously all side-chains within alpha-helices strongly avoid the g- conformation (chi 1 approximately 60 degrees). This restricts the beta-branched residues Thr, Val and  to a single conformer (g+, chi 1 approximately -60 degrees). Asp, Asn, Met and Ser within helices also overwhelmingly prefer the g+ conformation. For Arg, Cys, Gln, Glu, Leu and Lys the t (chi 1 approximately 180 degrees) and g+ conformers are populated roughly equally. Only the aromatic residues, His, , Trp and Phe prefer the t conformation. These preferences are the same whether the side-chain is buried or solvent-exposed. In general, the side-chain conformations adopted by the residues substituted at positions 44 and 131 correspond to the most commonly observed conformation for the same amino acid in helices in known protein structures. The changes in protein stability for the replacements at site 131 in general agree well with those at site 44 (correlation r = 0.97), suggesting that these may be representative of substitutions at fully solvent-exposed sites in the middle of alpha-helices. The free  values also agree quite well with those observed for equivalent replacements in a number of soluble alpha-helical model peptides and with data from "host-guest" studies and statistical surveys (r = 0.69 to 0.93).(ABSTRACT TRUNCATED AT 400 WORDS).</p><h2><a href=https://www.ncbi.nlm.nih.gov/pubmed/26733187>Molecular dynamics of sialic acid analogues complex with cholera toxin and DFT optimization of ethylene glycol-mediated zinc nanocluster conjugation.</a></h2><p>Cholera is an infectious disease caused by cholera toxin (CT) protein of bacterium Vibrio cholerae. A sequence of sialic acid (N-acetylneuraminic acid, NeuNAc or Neu5Ac) analogues modified in its C-5 position is modelled using molecular modelling techniques and docked against the CT followed by molecular dynamics simulations. Docking results suggest better binding affinity of NeuNAc analogue towards the binding site of CT. The NeuNAc analogues interact with the active site residues GLU:11, :12, HIS:13, GLY:33, LYS:34, GLU:51, GLN:56, HIE:57, :58, GLN:61, TRP:88, ASN:90 and LYS:91 through intermolecular hydrogen bonding. Analogues N-glycolyl-NeuNAc, N-Pentanoyl-NeuNAc and N-Propanoyl-NeuNAc show the least XPGscore (docking score) of -9.90, -9.16, and -8.91, respectively, and glide  of -45.99, -42.14 and -41.66 kcal/mol, respectively. Stable nature of CT-N-glycolyl-NeuNAc, CT-N-Pentanoyl-NeuNAc and CT-N-Propanoyl-NeuNAc complexes was verified through molecular dynamics simulations, each for 40 ns using the software Desmond. All the nine NeuNAc analogues show better score for drug-like properties, so could be considered as suitable candidates for drug development for cholera infection. To improve the enhanced binding mode of NeuNAc analogues towards CT, the nine NeuNAc analogues are conjugated with Zn nanoclusters through ethylene glycol (EG) as carriers. The NeuNAc analogues conjugated with EG-Zn nanoclusters show better binding  towards CT than the unconjugated nine NeuNAc analogues. The electronic structural optimization of EG-Zn nanoclusters was carried out for optimizing their performance as better delivery vehicles for NeuNAc analogues through density functional theory calculations. These sialic acid analogues may be considered as novel leads for the design of drug against cholera and the EG-Zn nanocluster may be a suitable carrier for sialic acid analogues.</p><h2><a href=https://www.ncbi.nlm.nih.gov/pubmed/11790849>Solution structure of a K(+)-channel blocker from the scorpion Tityus cambridgei.</a></h2><p>A new K(+)-channel blocking peptide identified from the scorpion venom of Tityus cambridgei (Tc1) is composed of 23 amino acid residues linked with three disulfide bridges. Tc1 is the shortest known toxin from scorpion venom that recognizes the Shaker B K(+) channels and the voltage-dependent K(+) channels in the brain. Synthetic Tc1 was produced using solid-phase synthesis, and its activity was found to be the same as that of native Tc1. The pairings of three disulfide bridges in the synthetic Tc1 were identified by NMR experiments. The NMR solution structures of Tc1 were determined by simulated annealing and -minimization calculations using the X-PLOR program. The results showed that Tc1 contains an alpha-helix and a 3(10)-helix at N-terminal Gly(4)-Lys(10) and a double-stranded beta-sheet at Gly(13)-(16) and Arg(19)-(23), with a type I' beta-turn at Asn(17)-Gly(18). Superposition of each structure with the best structure yielded an average root mean square deviation of 0.26 +/- 0.05 A for the backbone atoms and of 1.40 +/- 0.23 A for heavy atoms in residues 2 to 23. The three-dimensional structure of Tc1 was compared with two structurally and functionally related scorpion toxins, charybdotoxin (ChTx) and noxiustoxin (NTx). We concluded that the C-terminal structure is the most important region for the blocking activity of voltage-gated (Kv-type) channels for scorpion K(+)-channel blockers. We also found that some of the residues in the larger scorpion K(+)-channel blockers (31 to 40 amino acids) are not involved in K(+)-channel blocking activity.</p><h2><a href=https://www.ncbi.nlm.nih.gov/pubmed/31504604>Nutrient digestibility of soybean products in grower-finisher pigs.</a></h2><p>Solvent extraction of soybean creates soybean meal (SBM), but an array of other soybean products can be created using further processing of SBM or soybean. For accurate inclusion of these products in pig feed, characterization of digestible AA profile and  value is required. Soybean products from processes such as extrusion (EX) of soybean and thermo-mechanical (TM) treatment, bioconversion using fermentation or enzymes (BC), and ethanol-water extraction (EW) of soybean meal were collected together with SBM. These 9 soybean products were tested in cornstarch-based diets together with an N-free diet for a total of 10 diets. Ten ileal-cannulated barrows (30.4 ± 0.7 kg initial BW) were fed 10 diets at 2.8 times maintenance DE for six 9-day periods with a 6 (periods) × 10 (pigs) Youden square. The control SBM contained 47.0% CP, 1.4% ether extract, and ADF 6.0%. The 9 soybean products contained 35.6 to 66.4% CP, 0.9 to 21.6% ether extract, and 4.4 to 8.0% ADF. The EW soybean products were high in CP (> 61%), whereas the 2 EX soybean products were low in CP (< 36%) but high in ether extract (≥ 19%). Chemically-available Lys ranged from 92.6 to 100% of total Lys, indicating that minor Lys damage occurred during processing. The apparent total tract digestibility (ATTD) of  was lower (P < 0.05) for soybean products with greater ether extract and ADF content than SBM, and varied among soybean products. The standardized ileal digestibility (SID) did not differ (P > 0.05) among soybean products for most AA, except for lower SID of Arg, , Leu, Lys, Phe, and  (P < 0.05) for EX2 and BC1 than other soybean products. The DE and predicted NE value did not differ (P > 0.05) among soybean products. The greater SID AA content (P < 0.05) in EW, BC, and TM1 soybean products than SBM was mainly a result of greater total AA content due to removal of other macronutrients. In conclusion, extrusion of soybean creates soybean products with a greater  value but lower ATTD of  and lower SID AA content than SBM. Further processing of SBM creates soybean products with greater CP and SID AA content but similar SID of AA than SBM. Thus, new technologies to process SBM or soybean create high value ingredients to be included in pig diets, especially for young pigs with high nutritional requirements.© The Author(s) 2019. Published by Oxford University Press on behalf of the American Society of Animal Science. All rights reserved. For permissions, please e-mail: journals.permissions@oup.com.</p><h2><a href=https://www.ncbi.nlm.nih.gov/pubmed/2285793>Differential scanning calorimetric studies of E. coli aspartate transcarbamylase. III. The denaturational thermodynamics of the holoenzyme with single-site mutations in the catalytic chain.</a></h2><p>Aspartate transcarbamylase (EC 2.1.3.2) from E. coli is a multimeric enzyme consisting of two catalytic subunits and three regulatory subunits whose activity is regulated by subunit interactions. Differential scanning calorimetric (DSC) scans of the wild-type enzyme consist of two peaks, each comprised of at least two components, corresponding to denaturation of the catalytic and regulatory subunits within the intact holoenzyme (Vickers et al., J. Biol. Chem. 253 (1978) 8493; Edge et al., Biochemistry 27 (1988) 8081). We have examined the effects of nine single-site mutations in the catalytic chains. Three of the mutations (Asp-100-Gly, Glu-86-Gln, and Arg-269-Gly) are at sites at the C1: C2 interface between c chains within the catalytic subunit. These mutations disrupt salt linkages present in both the T and R states of the molecule (Honzatko et al., J. Mol. Biol. 160 (1982) 219; Krause et al., J. Mol. Biol. 193 (1987) 527). The remainder (Lys-164-, -165-Phe, Glu-239-Gln, Glu-239-Ala, -240-Phe and Asp-271-Ser) are at the C1: C4 interface between catalytic subunits and are involved in interactions which stabilize either the T or R state. DSC scans of all of the mutants except Asp-100-Gly and Arg-269-Gly consisted of two peaks. At intermediate concentrations, Asp-100-Gly and Arg-269-Gly had only a single peak near the Tm of the regulatory subunit transition in the holoenzyme, although their denaturational profiles were more complex at high and low protein concentrations. The catalytic subunits of Glu-86-Gln, Lys-164- and Asp-271-Ser appear to be significantly destabilized relative to wild-type protein while -165-Phe and -240-Phe appear to be stabilized. Values of delta delta G degree cr, the difference between the subunit interaction  of wild-type and mutant proteins, evaluated as suggested by Brandts et al. (Biochemistry 28 (1989) 8588) range from -3.7 kcal mol-1 for Glu-86-Gln to 2.4 kcal mol-1 for -165-Phe.</p><h2><a href=https://www.ncbi.nlm.nih.gov/pubmed/1503569>[Protein motifs and their structure-functional role].</a></h2><p>The review deals with repeating fragments of amino acid sequences, so-called "motifs", that are important in maintaining structural integrity and/or function of various proteins, especially those interacting with phospholipid aggregates. The occurrence of Phe-Leu-Gly motif characteristic for the amino acid sequence of the primate immuno-deficiency viruses fusion peptides is analysed in various proteins, as well as tripeptide fragments of general formula Xaa-Xah-Gly (Xaa-Phe, ; Xab-hydrophobic amino acids Ala, Val, Leu, ) homologous to the above motif and retro-sequences Gly-Xab-Xaa. These tripeptide repeats are characteristic for the amino acid sequences of complex membrane proteins, viral envelope proteins, proteinases and proteins connected with  transfer or interacting with lipids. These repeats are frequently met in conservative regions of amino acid sequences, in sites readily accessible for other molecules at the boundary of or between structured fragments, this being due to the backbone semi-coiled form's preference in the given amino acid fragment. This protein motif appears to play an important role at the initial stages of the large protein's interaction with the phospholipid membrane.</p><h2><a href=https://www.ncbi.nlm.nih.gov/pubmed/21677784>Biological responses to perfluorododecanoic acid exposure in rat kidneys as determined by integrated proteomic and metabonomic studies.</a></h2><p>Perfluorododecanoic acid (PFDoA) is a perfluorinated carboxylic chemical (PFC) that has broad applications and distribution in the environment. While many studies have focused on hepatotoxicity, immunotoxicity, and reproductive toxicity of PFCAs, few have investigated renal toxicity.Here, we used comparative proteomic and metabonomic technologies to provide a global perspective on renal response to PFDoA. Male rats were exposed to 0, 0.05, 0.2, and 0.5 mg/kg/day of PFDoA for 110 days. After 2-D DIGE and MALDI TOF/TOF analysis, 79 differentially expressed proteins between the control and the PFDoA treated rats (0.2 and 0.5 mg-dosed groups) were successfully identified. These proteins were mainly involved in amino acid metabolism, the tricarboxylic acid cycle, gluconeogenesis, glycolysis, electron transport, and stress response. Nuclear magnetic resonance-based metabonomic analysis showed an increase in pyruvate, lactate, acetate, choline, and a variety of amino acids in the highest dose group. Furthermore, the profiles of free amino acids in the PFDoA treated groups were investigated quantitatively by high-coverage quantitative iTRAQ-LC MS/MS, which showed levels of sarcosine, asparagine, histidine, 1-methylhistidine, , Leu, Val, Trp, , Phe, Cys, and Met increased markedly in the 0.5 mg dosed group, while homocitrulline, α-aminoadipic acid, β-alanine, and cystathionine decreased.These observations provide evidence that disorders in glucose and amino acid metabolism may contribute to PFDoA nephrotoxicity. Additionally, α(2u) globulin may play an important role in protecting the kidneys from PFDoA toxicity.</p><h2><a href=https://www.ncbi.nlm.nih.gov/pubmed/1518799>NMR docking of a substrate into the X-ray structure of staphylococcal nuclease.</a></h2><p>The conformation of the staphylococcal nuclease-bound metal-dTdA complex, previously determined by NMR methods [Weber, D.J., Mullen, G.P., Mildvan, A.S. (1991) Biochemistry 30:7425-7437] was docked into the X-ray structure of the enzyme-Ca(2+)-3',5'-pdTp complex [Loll, P.J., Lattman, E.E. (1989) Proteins: Struct., Funct., Genet. 5:183-201] by superimposing the metal ions, taking into account intermolecular nuclear Overhauser effects from assigned aromatic proton resonances of -85, -113, and -115 to proton resonances of the leaving dA moiety of dTdA, and  minimization to relieve small overlaps. The proton resonances of the Phe, , and Trp residues of the enzyme in the ternary enzyme-La(3+)-dTdA complex were sequence specifically assigned by 2D phase-sensitive NOESY, with and without deuteration of the aromatic protons of the  residues, and by 2D heteronuclear multiple quantum correlation (HMQC) spectroscopy and 3D NOESY-HMQC spectroscopy with 15N labeling. While resonances of most Phe,  and Trp residues were unshifted by the substrate dTdA from those found in the enzyme-La(3+)-3',5'-pdTp complex and the enzyme-Ca(2+)-3',5'-pdTp complex, proton resonances of -85, -113, -115, and Phe-34 were shifted by 0.08 to 0.33 ppm and the 15N resonance of -113 was shifted by 2.1 ppm by the presence of substrate. The optimized position of enzyme-bound dTdA shows the 5'-dA leaving group to partially overlap the inhibitor, 3',5'-pdTp (in the X-ray structure). The 3'-TMP moiety of dTdA points toward the solvent in a channel defined by -18, Asp-19, Thr-22, Lys-45, and His-46. The phosphate of dTdA is coordinated by the metal, and an adjacent inner sphere water ligand is positioned to donate a hydrogen bond to the general base Glu-43 and to attack the phosphorus with inversion. Arg-35 and Arg-87 donate monodentate hydrogen bonds to different phosphate oxygens of dTdA, with Arg-87 positioned to protonate the leaving 5'-oxygen of dA, thus clarifying the mechanism of hydrolysis. Model building of an additional 5'-dGMP onto the 3'-oxygen of dA placed this third nucleotide onto a surface cleft near residues Glu-80, Asp-83, Lys-84, and -115 with its 3'-OH group accessible to the solvent, thus defining the size of the substrate binding site as accommodating a trinucleotide.</p><h2><a href=https://www.ncbi.nlm.nih.gov/pubmed/26706905>Interaction of Quercetin of Onion with Axon Guidance Protein Receptor, NRP-1 Plays Important Role in Cancer Treatment: An In Silico Approach.</a></h2><p>Neuropilin-1 (NRP-1) is a transmembrane glycoprotein receptor whose distinct sites bind semaphorins and vascular endothelial growth factor family members to mediate the role of these ligands in neuronal axon guidance and angiogenesis, respectively. Similarly, Eph receptors and ephrin ligands play critical roles in various biological functions, and deregulated activation of Eph/ephrin signaling in humans is thought to lead to tumorigenesis. Therefore, in this paper, an attempt was made to elucidate the inhibition potential of nine bioactive compounds from four different native spices of Bangladesh against this couple of receptors via molecular docking study. The molecular docking study was carried out using Vina docking protocol. Finally, the receptor-ligand interaction analysis was carried out using the Discovery Studio Client package. Quercetin and diosgenin of onion showed favorable binding with NRP-1 with low binding  of -7.8 and -7.2 kcal/mol, respectively, in comparison with the control inhibitor (-6.1 kcal/mol). The study suggests that ligand interaction with the residues Asp 48, Thr 44, Thr 77,  81, Trp29,  143 of NRP-1 and Lys 653, Phe 765, Ser 763, Thr 699,  683 of Eph might be critical for the inhibitory activity of these receptors. The study provides evidence for consideration of quercetin and diosgenin of onion as valuable small ligand molecules for targeting NRP-1 receptor in treatment and prevention of neurological disorders as well as cancer.</p><h2><a href=https://www.ncbi.nlm.nih.gov/pubmed/20630242>Changes in production and mammary metabolism of dairy cows in response to essential and nonessential amino acid infusions.</a></h2><p>This study was undertaken to increase our understanding of the need of the mammary gland for the different types of AA and how the mammary gland alters its metabolism in response to a variable AA supply. Eight lactating Holstein cows (61+/-4 DIM) were used in a replicated 4 x 4 Latin square balanced for residual effects with 14-d periods. The diet was formulated to supply 100% of the net  requirement and 72% of the metabolizable protein requirement. The 4 treatments were 1) abomasal infusions of water, 2) essential AA at 359 g/d, 3) nonessential AA at 356 g/d, and 4) essential AA at 359 g/d+nonessential AA at 356 g/d (total of 715 g/d). The infusates had the same AA profile as casein with the exception that Met was increased to maintain a 3:1 ratio of digestible Lys to Met and because of solubility limitations all the  was replaced by Phe and part of the Glu was replaced by Gln. Milk yield and milk protein yield were increased by the essential AA treatments compared with the other treatments. Mammary uptake of beta-hydroxybutyrate plus lactate tended to increase with the essential AA treatments, whereas glucose mammary uptake tended to be higher with the nonessential AA treatments. With the essential AA treatments, the mammary uptake:milk protein output ratio for the group 1 AA (His, Met, Phe, Trp, and ) did not differ from 1 but tended to increase; the ratio for the group 2 AA (, Leu, Lys, and Val) did increase, significantly exceeding unity when essential AA were infused. These results indicate that the mammary gland alters differently its uptake of both AA and -yielding nutrients in response to the amount and profile of AA presented to it and that even under situations of protein deficiency nonessential AA supplementation does not enhance milk and milk protein synthesis.Copyright  (c) 2010 American Dairy Science Association. Published by Elsevier Inc. All rights reserved.</p><h2><a href=https://www.ncbi.nlm.nih.gov/pubmed/31669310>Dimerization of long Hibernation Promoting Factor from Staphylococcus aureus: structural analysis and biochemical characterization.</a></h2><p>Staphylococcus aureus hibernation promoting factor (SaHPF) is responsible for the formation of 100S ribosome dimers, which in turn help this pathogen to reduce  spent under unfavorable conditions. Ribosome dimer formation strongly depends on the dimerization of the C-terminal domain of SaHPF (CTD). In this study, we solved the crystal structure of CTD at 1.6 Å resolution and obtained a precise arrangement of the dimer interface. Residues Phe, Val, Thr, , ,  andThr in the dimer interface of SaHPF protein were mutated and the effects were analyzed for the formation of 100S disomes of ribosomes isolated from S. aureus. It was shown that substitution of any of single residues Phe, Val, ,  and  in the SaHPF homodimer interface abolished the ribosome dimerization in vitro.Copyright © 2019. Published by Elsevier Inc.</p><h2><a href=https://www.ncbi.nlm.nih.gov/pubmed/15889787>[Tertiary structure of myelopeptides. II. Conformational analysis of Phe-Arg-Pro-Arg--Met-Thr-Pro, Val-Val--Pro-Asp, and Val-Asp-Pro-Pro].</a></h2><p>Theoretical conformational analysis was used to study the spatial structure and conformational properties of myelopeptides, bone marrow peptide mediators. The low- conformations of myelopeptides MP-4 (Phe-Arg-Pro-Arg--Met-Thr-Pro), MP-5 (Val-Val--Pro-Asp), and MP-6 (Val-Asp-Pro-Pro) were found; the values of dihedral angles of backbone and side chains of the amino acid residues were determined; and the energies of intra- and interresidual interactions were estimated.</p><h2><a href=https://www.ncbi.nlm.nih.gov/pubmed/2930509>Purification and molecular properties of malate dehydrogenase from the marine diatom Nitzschia alba.</a></h2><p>Malate dehydrogenase (EC 1.1.1.37) was purified to homogeneity from the marine diatom Nitzschia alba. The purification steps consisted of (NH4)2SO4 precipitation, ion-exchange chromatography, Blue Sepharose affinity chromatography and gel filtration. A typical procedure provided 685-fold purification with 58% yield. The Mr of the holoenzyme was estimated to be 322,000 by gel filtration and 316,000 by ultracentrifugation. The enzyme migrated as a single polypeptide spot on two-dimensional polyacrylamide-gel electrophoresis with an Mr of 38,500, suggesting that the holoenzyme consists of eight identical subunits. This is the first case where malate dehydrogenase has been shown to be a homo-octamer; malate dehydrogenases from other sources are predominantly homodimers, with two homotetramers reported so far. The amino acid composition of the enzyme was determined and the N-terminal sequence of the subunit polypeptide was found to be Arg-Lys-Val-Ala-Val-Met-Gly-Ala-Ala-Gly-Gly--Gly-Gln-Pro-Leu-Ser-Leu- Leu-Leu - Lys-Leu-Ser-Pro-Gln-Val-Thr-Glu-Leu-Ser-Lys--. For the first 21 amino acid residues, near-identical sequences were reported for the enzymes isolated from pig heart, Escherichia coli, yeast and watermelon. Other physicochemical and catalytic properties, such as sedimentation coefficient, partial specific volume, Stokes radius, excitation and emission maxima, Michaelis constants, pH optima, pH stability range and activation , of this enzyme are also presented.</p><h2><a href=https://www.ncbi.nlm.nih.gov/pubmed/15565239>Structural scaffold of 18-crown-6 tetracarboxylic acid for optical resolution of chiral amino acid: X-ray crystal analyses and  calculations of complexes of D- and L-isomers of tyrosine, isoleucine, methionine and phenylglycine.</a></h2><p>To clarify the structural scaffold of (+)-18-crown-6 tetracarboxylic acid ((+)-18C6H4) for the optical resolution of a chiral amino acid, the crystal structures of its equimolar complexes with L- and D-isomers of tyrosine (), isoleucine (), methionine (Met) and phenylglycine (PheG) were analysed by X-ray diffraction methods. (+)-18C6H4 took very similar conformations for all complexes. Although the chemical structure of (+)-18C6H4 is C2-symmetric, it took a similar asymmetric ring conformation of radius ca. 6.0 A. In all complexes, the amino group of chiral amino acids was located near the center of the ring and formed three hydrogen bonds and five electrostatic interactions with eight oxygen atoms of the ether ring and carboxyl groups. Also, the Calpha atom of chiral amino acids participated in Calpha-H...O interaction with the oxygen atom of (+)-18C6H4. In contrast, the carboxyl group of chiral amino acids did not directly interact with (+)-18C6H4. These results indicate that the structural scaffold of (+)-18C6H4 for the optical resolution of chiral amino acids is mainly based on the mode of interaction of (+)-18C6H4 with the amino and Calpha-H groups of chiral amino acids. The differences in interaction pattern and binding  between the L- and D-isomers of each amino acid are discussed in relation to the chiral recognition of (+)-18C6H4.</p><h2><a href=https://www.ncbi.nlm.nih.gov/pubmed/22884342>Milk protein synthesis in response to the provision of an "ideal" amino acid profile at 2 levels of metabolizable protein supply in dairy cows.</a></h2><p>Providing a well-balanced supply of essential AA (EAA) can serve as an opportunity to reduce the protein intake for dairy cows by increasing the efficiency of metabolizable protein (or PDIE, its equivalent in the INRA feeding system) utilization for milk protein yield. Our objectives were to compare the effect of supplying an "ideal" EAA profile (EAA+) with an imbalanced AA profile (control) at 2 levels of PDIE/NE(L) (net  for lactation) supplies to study the interaction between PDIE and AA profiles, and to compare this ideal profile with a simple mixture of the 4 most deficient EAA (4 EAA) in the diets of dairy cows. Six lactating multiparous Holstein cows received 6 treatments with 2 different levels of PDIE supplied by diets and AA infusions in the duodenum according to a changeover design with 3-wk periods. Within each PDIE supply level, the cows received 3 different AA infusions in the duodenum according to a 3×3 Latin square design with 1-wk subperiods, which corresponded to the following treatment groups: control (Glu), 4EAA (Lys, Met, His, Leu), and EAA+ (4 EAA plus , Val, Phe, Trp, and ). In the low and high PDIE treatments, diets and infusions provided 54.7 and 64.0 g/Mcal of PDIE/NE(L), respectively, which corresponded to crude protein levels of 13.6 and 15.2%, respectively. High-PDIE supplies increased the milk protein yield by 163 g/d, the milk protein content by 1.4 g/kg, the milk yield by 4.1 kg/d, and the lactose yield by 178 g/d and decreased the PDIE efficiency of utilization by 12.4%, whereas the N efficiency of utilization remained unaffected. Supplying the 2 EAA profiles (4EAA and EAA+) increased the milk protein yield by 67 g/d, the milk protein content by 1.3g/kg, and the milk yield by 0.9 kg/d, whereas the milk fat and milk lactose contents were decreased by 2.4 and 1.6g/kg, respectively. The responses regarding milk yield and its composition were similar whether the cows received the 4 EAA or the EAA+ treatment. The responses were similar for the milk yield and composition whether the EAA were supplied by low- or high-PDIE supplies. In conclusion, the efficiency of PDIE utilization was improved by 6.6% and the N efficiency was improved by 7.0% by correcting the EAA profiles, independent of the level of PDIE supplied. In addition, the increased efficiency observed, associated with provision of the 4 EAA, was similar to the provision of all EAA (EAA+) in this experiment.Copyright © 2012 American Dairy Science Association. Published by Elsevier Inc. All rights reserved.</p><h2><a href=https://www.ncbi.nlm.nih.gov/pubmed/21472525>Effect of the identity of Xaa on the fragmentation modes of doubly-protonated Ala-Ala-Xaa-Ala-Ala-Ala-Arg.</a></h2><p>The product ion mass spectra resulting from collisional activation of doubly-protonated tryptic-type peptides Ala-Ala-Xaa-Ala-Ala-Ala-Arg have been determined for Xaa = Ala(A), Ser(S), Val(V), Thr(T), (I), Phe(F), (Y), Sar, Met(M), Trp(W), Pro(P), and Gln(Q). The major fragmentation reaction involves cleavage of the second amide bond (counting from the N-terminus) except for Xaa = Ser and Thr where elimination of H(2)O from the [M + 2H](+2) ion forms the base peak. In general, the extent of cleavage of the second amide bond shows little dependence on the identity of Xaa and little dependence on whether the bond cleavage involves symmetrical bond cleavage to form a y(5)/b(2) ion pair or asymmetrically to form y (5) (+2) and a neutral b(2) species. Notable exceptions to this generalization occur for Xaa equal to Pro or Sar. For Xaa = Pro only cleavage of the second amide bond is observed, consistent with a pronounced proline effect, i.e., cleavage N-terminal to Pro. When Xaa = Sar considerably enhanced cleavage of the second amide bond also is observed, suggesting that at least part of the proline effect relates to the tertiary nature of the amide nitrogen. In the competition between symmetric and asymmetric bond cleavage an attempt to establish a linear free  correlation in relating ln(y(5)(+2)/y(5)) to PA(H-Xaa-OH) did not lead to a reasonable correlation although the trend of increasing y(5)(+2)/y(5) ratio with increasing proton affinity of H-Xaa-OH was clear. Proline showed a unique behavior in giving a much higher y(5)(+2)/y(5) ratio than any of the other residues studied.© American Society for Mass Spectrometry, 2011</p><h2><a href=https://www.ncbi.nlm.nih.gov/pubmed/29471545>High-protein distillers dried grains with solubles produced using a novel front-end-back-end fractionation technology has greater nutritional value than conventional distillers dried grains with solubles when fed to growing pigs.</a></h2><p>Two experiments were conducted to determine the standardized ileal digestibility (SID) of CP and AA, apparent total tract digestibility (ATTD) of GE, and DE and ME in conventional distillers dried grains with solubles (DDGS-CV) and in a novel source of high-protein distillers dried grains with solubles (DDGS-HP) produced by Lincolnway  (Nevada, IA). In Exp. 1, 18 barrows (initial BW: 72.47 ± 9.16 kg) that had a T-cannula installed in the distal ileum were allotted to a completely randomized design with 3 diets and 6 replicate pigs per diet. A nitrogen-free diet and 2 diets that contained cornstarch and DDGS-CV or DDGS-HPLincolnway as the sole source of CP and AA were formulated. Diets were fed to pigs for 7 d, and ileal digesta were collected on days 6 and 7 of each period. The SID for Leu, Lys, Met, Phe, and Glu was greater (P < 0.05) in DDGS-HPLincolnway than in DDGS-CV, and the SID of , Val, and total indispensable AA, as well as the SID of , tended to be greater (P < 0.10) in DDGS-HPLincolnway than in DDGS-CV. No difference between DDGS-CV and DDGS-HPLincolnway was observed for the SID of CP and all other AA. In Exp. 2, 24 barrows (initial BW: 52.80 ± 2.55 kg) were housed individually in metabolism crates and randomly allotted to 1 of 3 diets. A corn-based basal diet (97.25% corn) and 2 diets that contained corn and DDGS-CV or corn and DDGS-HPLincolnway were formulated. Each diet was fed to 8 pigs. Feces and urine were collected using the marker to marker approach with 7-d adaptation and 5-d collection periods. The DE and ME in DDGS-CV and DDGS-HPLincolnway were calculated using the difference procedure. The DE and ME in DDGS-HPLincolnway on an as-fed basis were greater (P < 0.05) than in corn and DDGS-CV, but the ATTD of GE in DDGS-HPLincolnway and DDGS-CV was less (P < 0.01) than in corn. In conclusion, the SID of some AA and the DE and ME in DDGS-HPLincolnway were greater than in DDGS-CV.</p><h2><a href=https://www.ncbi.nlm.nih.gov/pubmed/2106161>Novel fluorogenic substrates for assaying retroviral proteases by resonance  transfer.</a></h2><p>The 11-kD protease (PR) encoded by the human immunodeficiency virus 1 (HIV-1) is essential for the correct processing of viral polyproteins and the maturation of infectious virus, and is therefore a target for the design of selective acquired immunodeficiency syndrome (AIDS) therapeutics. To facilitate the identification of novel inhibitors of HIV-1 PR, as well as to permit detailed studies on the enzymology and inhibition of this enzyme, a continuous assay for its activity was developed that was based on intramolecular fluorescence resonance  transfer (RET). The assay used the quenched fluorogenic substrate 4-(4-dimethylaminophenylazo)benzoic acid (DABCYL)--Ser Gln Asn  Pro  Val Gln--5-[(2-aminoethyl)amino]naphthalene-1 sulfonic acid (EDANS), whose peptide sequence is derived from a natural processing site for HIV-1 PR. Incubation of recombinant HIV-1 PR with the fluorogenic substrate resulted in specific cleavage at the -Pro bond and a time-dependent increase in fluorescence intensity that was linearly related to the extent of substrate hydrolysis. An internally quenched fluorogenic substrate was also designed that was selectively cleaved by the related PR from avian myeloblastosis virus (AMV). The fluorescence quantum yields of the HIV-1 PR and AMV PR substrates in the RET assay increased by 40.0- and 34.4-fold, respectively, per mole of substrate cleaved. Because of its simplicity, rapidity, and precision in the determination of reaction rates required for kinetic analysis, this method offers many advantages over the commonly used high-performance liquid chromatography- or electrophoresis-based assays for peptide substrate hydrolysis by retroviral PRs.</p><h2><a href=https://www.ncbi.nlm.nih.gov/pubmed/7966143>Binding of fluorescent and spin-labeled C-terminal hirudin analogs to thrombin.</a></h2><p>Synthetic peptides based on the sequence of the negatively charged carboxyl tail of hirudin exhibit anticoagulant activity. Several antithrombin agents are being developed by chemical and structural optimization of these "hirupeptides". The present work demonstrates the design and use of novel spin-labeled and fluorescent-labeled C-terminal hirudin analogs to study the interactions of these antithrombin agents with thrombin in solution. Three labeled hirulabels were synthesized based upon the amino acid sequence of the antithrombin agent MDL 28050, X-NH-(CH2)7-CO-Asp--Glu-Pro--Pro-Glu-Glu-Ala-Cha-D-Glu-OH, where X = anthraniloyl, 1,5-dansyl, or 3-carbamoyl-2,2,5,5-tetramethyl-3-pyrrolin-1-oxyl. The modifications did not significantly alter the potency of these inhibitors which showed Ki values of 100 nM. Their interactions with human and bovine thrombin were studied by ESR and fluorescence techniques. The spin-labeled hirupeptide was able to discern subtle differences in binding to human versus bovine thrombin. The 8-aminooctanoic acid spacer arm placed the nitroxide moieties near the active site, near regions of the autolysis loops which differentiates between human alpha- and gamma-thrombin. It was also able to discern paramagnetic quenching and fluorescence  transfer interactions, respectively, between covalently attached spin labels and fluorescent probes at the active site Ser 195 and the fluorophore on the hirupeptide.</p><h2><a href=https://www.ncbi.nlm.nih.gov/pubmed/25354707>Permethrin and its metabolites affect Cu/Zn superoxide conformation: fluorescence and in silico evidences.</a></h2><p>The proclivity of permethrin and its metabolites to affect the structure and activity of Cu/Zn superoxide dismutase (SOD) has been investigated by using intrinsic fluorescence and 8-ANS fluorescence techniques. In silico molecular docking investigations were carried out in order to assess the means of interaction at a molecular level between SOD and the considered ligands. Results show that both, permethrin and its metabolites are able to induce conformational variation on SOD. Permethrin and 3-phenoxybenzyl alcohol metabolite induce a blue shift toward the hydrophobic amino acids Leu-101, -102, Leu-104, -110 and -111, with a significant peak increase. An opposite effect was shown by 3-phenoxy benzaldehyde and 3-phenoxybenzoic acid with a progressive reduction of tyrosine fluorescence emission, without any shift. Computational findings confirm that all the molecules considered have more than one allosteric binding site but none of them interact with SOD at its catalytic Cu/Zn cleft. Moreover, all the binding poses found are very close in binding  thus demonstrating that there is not only a preferred interaction site but most of them are important due to their relative  in equilibrium with a population strictly connected to the ligand concentration. In the obtained complexes, all the ligands are involved in many hydrogen bonds through their polar oxygen moieties but due to the presence of a common aromatic hydrophobic core, many hydrophobic interactions are due to the SOD nature rich in apolar amino acids. Furthermore, for each ligand it can be pointed out the presence of a highly populated docked structure with a specific interaction of permethrin and its metabolites with -108, responsible for changes in fluorescence emission.</p><h2><a href=https://www.ncbi.nlm.nih.gov/pubmed/8085828>Growth requirements of hyperthermophilic sulfur-dependent heterotrophic archaea isolated from a shallow submarine geothermal system with reference to their essential amino acids.</a></h2><p>Three hyperthermophilic sulfur-dependent heterotrophs were isolated from a shallow submarine hydrothermal system at an inlet of Kodakara-jima island, Kagoshima, Japan. The isolates grew at 60 to 97 degrees C, with the optimum temperatures at 85 to 90 degrees C. Sensitivity to rifampin and the existence of ether lipids indicated that the isolates are hyperthermophilic archaea. Partial sequencing of the genes coding for 16S rRNA showed that the three isolates are closely related to the genus Thermococcus. They grew on proteinaceous mixtures, such as yeast extract, Casamino Acids, and purified proteins (e.g., casein and gelatin), but not on carbohydrates or organic acids as sole carbon and  sources. Nine amino acids were essential for growth of isolate KS-1 (Thr, Leu, , Val, Met, Phe, His, , and Arg). Isolate KS-2 required Lys in addition to the nine amino acids, and KS-8 required Lys instead of . In comparative studies, it was shown that Thermococcus celer DSM 2476 required 10 amino acids (Thr, Leu, , Val, Met, Phe, , Trp, Lys, and Arg) while Pyrococcus furiosus DSM 3638 required only  and Val. The hyperthermophilic fermentative eubacterium Thermotoga neapolitana DSM 4359 did not require any amino acids for growth.</p><h2><a href=https://www.ncbi.nlm.nih.gov/pubmed/1385425>Cooperative interactions among subunits of a voltage-dependent potassium channel. Evidence from expression of concatenated cDNAs.</a></h2><p>Four copies of the coding sequence for a voltage-dependent potassium channel (RBK1, rat Kv1.1) were ligated contiguously and transcribed in vitro. The resulting RNA encodes four covalently linked subunit domains ([4]RBK1). Injection of this RNA into Xenopus oocytes resulted in the expression of voltage-dependent potassium currents. A single amino acid substitution, -->Val, located within the outer mouth of the pore, introduced into the equivalent position of any of the four domains, reduced affinity for external tetraethylammonium by approximately the same amount. In constructs containing 0, 1, 2, 3, or 4  residues the free  of binding tetraethylammonium was linearly related to the number of  residues. A different amino acid substitution, Leu-->, located in the S4 region, was made in the equivalent position of one, two, three, or four domains. The depolarization required for channel activation increased approximately linearly with the number of  residues, whereas models of independent gating of each domain predict marked nonlinearity. Expression of this concatenated channel provides direct evidence that voltage-dependent potassium channels have four subunits positioned symmetrically around a central permeation pathway and that these subunits interact cooperatively during channel activation.</p><h2><a href=https://www.ncbi.nlm.nih.gov/pubmed/25595103>Computational identification and binding analysis of orphan human cytochrome P450 4X1 enzyme with substrates.</a></h2><p>Cytochrome P450s (CYPs) are important heme-containing proteins, well known for their monooxygenase reaction. The human cytochrome P450 4X1 (CYP4X1) is categorized as "orphan" CYP because of its unknown function. In recent studies it is found that this enzyme is expressed in neurovascular functions of the brain. Also, various studies have found the expression and activity of orphan human cytochrome P450 4X1 in cancer. It is found to be a potential drug target for cancer therapy. However, three-dimensional structure, the active site topology and substrate specificity of CYP4X1 remain unclear.In the present study, the three-dimensional structure of orphan human cytochrome P450 4X1 was generated by homology modeling using Modeller 9v8. The generated structure was accessed for geometrical errors and  stability using PROCHECK, VERFIY 3D and PROSA. A molecular docking analysis was carried out against substrates arachidonic acid and anandamide and the docked substrates were predicted for drug-likeness, ADME-Tox parameters and biological spectrum activity.The three-dimensional model of orphan human cytochrome P450 4X1 was generated and assessed with various structural validation programmes. Docking of orphan human cytochrome P450 4X1 with arachidonic acid revealed that  112, ALA 126,  222,  223, THR 312, LEU 315, ALA 316, ASP 319, THR 320, PHE 491 and  492 residues were actively participating in the interaction, while docking of CYP4X1 with anandamide showed that  112, GLN 114, PRO 118, ALA 126,  222,  223, SER 251, LEU 315, ALA 316 and PHE 491 key residues were involved in strong interaction.From this study, several key residues were identified to be responsible for the binding of arachidonic acid and anandamide with orphan human cytochrome P450 4X1. Both substrates obeyed Lipinski rule of five in drug-likeness test and biological spectrum prediction showed anticarcinogenic activity. Compared to anandamide, arachidonic acid showed strong interaction with cytochrome P450 4X1 and also less health effect in certain human system in ADME-Tox prediction. These findings provide useful information on the biological role and structure-based drug design of orphan human cytochrome P450 4X1.</p><h2><a href=https://www.ncbi.nlm.nih.gov/pubmed/12665544>The use of bioluminescence resonance  transfer 2 to study neuropeptide Y receptor agonist-induced beta-arrestin 2 interaction.</a></h2><p>The neuropeptide Y (NPY) family peptides NPY, peptide YY (PYY), and pancreatic polypeptide (PP) bind to four G protein-coupled receptors (GPCRs): Y1, Y2, Y4, and Y5. A key step in the desensitization and internalization of GPCRs is the association of the receptor with beta-arrestins. In the present study, these receptors were analyzed with respect to their ability to interact with GFP2-tagged beta-arrestin 2 using the new bioluminescence resonance  transfer 2 method. Agonists induced a concentration-dependent association of beta-arrestin 2 with all four receptors. Whereas the Y1 receptor exhibited the highest maximum response and rapid association (t(1/2) = 3.4 min), the maximal signals for the association of Y2 and Y4 receptors were less than half of that of Y1, and the association rates were much slower. Interestingly, when evaluated at the Y4 receptor, the Y4 agonist 1229U91 [(,Glu,Pro,Dpr,,Arg, Leu,Arg,Try-NH2)-2-cyclic(2,4'),(2',4)-diamide] was unable to provoke the same maximal response as human PP, suggesting that 1229U91 is a partial agonist. When stimulated by PYY, the Y5 receptor responded with a t(1/2) of 4.6 min and a maximal response approximately 60% of what was observed with Y1. Because beta-arrestins are key components in GPCR internalization, it is interesting to note that the receptor that is known to internalize rapidly (Y1) exhibits the most rapid association with beta-arrestin 2, whereas the receptor that is known to internalize slowly, or not at all (Y2) associates slowly with beta-arrestin 2.</p><h2><a href=https://www.ncbi.nlm.nih.gov/pubmed/12214994>Stimulatory and inhibitory effects of protein amino acids on growth rate and efficiency of mixed ruminal bacteria.</a></h2><p>Mixed ruminal bacteria were incubated in vitro with glucose, xylose, cellobiose, and various protein amino acids replaced isonitrogenously with 25% (i.e., 25 mg of N/L) of ammonia-N, to determine the growth rate and the amount of sugar consumed in the exponential growth phase. The growth rate and efficiency (grams of bacteria per gram of sugars) increased by 46 and 15%, respectively, when a mixture of 20 amino acids was added. On the other hand, neither growth rate nor efficiency increased when any one of these amino acids was added singly, except for Glu and Gln, each of which produced significant but small improvements. The stimulatory effect of the combined amino acids on bacterial growth declined when each of Leu, Trp, , Glu, Met, Phe, and Val was removed from the original group of 20. When a mixture of only these seven amino acids was used as a supplement, their stimulatory effects on growth rate and efficiency were only 21 and 25%, respectively, of the effects that the mixture of 20 amino acids showed. The effects increased to 76 and 72% on growth rate and efficiency, respectively, when Gly, Cys, and His were supplied in addition to the seven amino acids. The growth rate and efficiency of the ruminal bacteria were inhibited by an addition of each of , Thr, Cys, Phe, Leu, Lys, or Val to ammonia-N, and the effects of the first five of these amino acids were highly significant. Isoleucine, threonine, and phenylalanine were each inhibitory even at a low concentration (1 mg of NL), while cysteine and leucine showed inhibitory effects at higher concentrations (more than 10 mg of N/L). A higher growth rate of the ruminal bacteria when supplemented with amino acid mixtures was accompanied with a higher growth efficiency, which was attributable to a relatively smaller proportion of  expended on maintenance according to the Pirt derivation.</p><h2><a href=https://www.ncbi.nlm.nih.gov/pubmed/30949723>Metabolomic signature of type 1 diabetes-induced sensory loss and nerve damage in diabetic neuropathy.</a></h2><p>Diabetic-induced peripheral neuropathy (DPN) is a highly complex and frequent diabetic late complication, which is manifested by prolonged hyperglycemia. However, the molecular mechanisms underlying the pathophysiology of nerve damage and sensory loss remain largely unclear. Recently, alteration in metabolic flux has gained attention as a basis for organ damage in diabetes; however, peripheral sensory neurons have not been adequately analyzed with respect to metabolic dysfunction. In the present study, we attempted to delineate the sequence of event occurring in alteration of metabolic pathways in relation to nerve damage and sensory loss. C57Bl6/j wild-type mice were analyzed longitudinally up to 22 weeks in the streptozotocin (STZ) model of type 1 diabetes. The progression of DPN was investigated by behavioral measurements of sensitivity to thermal and mechanical stimuli and quantitative morphological assessment of intraepidermal nerve fiber density. We employed a mass spectrometry-based screen to address alterations in levels of metabolites in peripheral sciatic nerve and amino acids in serum over several months post-STZ administration to elucidate metabolic dysfunction longitudinally in relation to sensory dysfunction. Although hyperglycemia and body weight changes occurred early, sensory loss and reduced intraepithelial branching of nociceptive nerves were only evident at 22 weeks post-STZ. The longitudinal metabolites screen in peripheral nerves demonstrated that compared with buffer-injected age-matched control mice, mice at 12 and 22 weeks post-STZ showed an early impairment the tricaoboxylic acid (TCA cycle), which is the main pathway of carbohydrate metabolism leading to  generation. We found that levels of citric acid, ketoglutaric acid (2 KG), succinic acid, fumaric acid, and malic acid were observed to be significantly reduced in sciatic nerve at 22 weeks post-STZ. In addition, we also found the increase in levels of sorbitol and L-lactate in peripheral nerve from 12 weeks post-STZ injection. Amino acid screen in serum showed that the amino acids valine (Val), isoleucine (), and leucine (Leu), grouped together as BCAA, increased more than twofold from 12 weeks post-STZ. Similarly, the levels of tyrosine (), asparagine (Asn), serine (Ser), histidine (His), alanine (Ala), and proline (Pro) showed progressive increase with progression of diabetes. Our results indicate that the impaired TCA cycle metabolites in peripheral nerve are the primary cause of shunting metabolic substrate to compensatory pathways, which leads to sensory nerve fiber loss in skin and contribute to onset and progression of peripheral neuropathy.</p><h2><a href=https://www.ncbi.nlm.nih.gov/pubmed/10320051>Genetics of insulin resistance.</a></h2><p>Even among young, healthy individuals, there is more than a 10-fold variation in insulin sensitivity; however, taken in combination, all the known modifiers of insulin sensitivity - including obesity and a variety of environmental factors - explain less than one third of this variation. It is possible that genetic factors could account for the bulk of the variance observed, and hence play a major role in the development of impaired insulin sensitivity, ie insulin resistance. From the genetic point of view, insulin resistance is thought to be due to the inheritance of a number of mutations in a variety of genes. Three complementary approaches have been applied in the search for mutations: mutational analysis of candidate genes; linkage analysis of candidate genes or chromosomal regions for insulin resistance in familial type 2 diabetes; and random genome mapping with quantitative trait loci (QTL) analysis. Mutational analysis of the insulin signalling cascade has identified a glycine-arginine (Gly-Arg) substitution at codon 972 of the insulin receptor substrate-1 (IRS-1) gene with a carrier prevalence of 9% among Caucasians. Expression of this variant in 32-D cells is associated with a significant (20-30%) impairment of insulin-stimulated PI3-kinase activity, as well as reduced binding of IRS-1 to the p85 regulatory subunit of PI3-kinase. Genotype/phenotype studies stratified according to body mass index (BMI) indicate that obese subjects who are heterozygous for the mutant allele have a 50% decrease in insulin sensitivity, compared with wild-type obese subjects. This suggests that there may be an interaction between the mutant allele and obesity, such that, in the presence of obesity, the mutant variant may aggravate the obesity-associated insulin resistance. Mutational analysis has also shown that homozygous carriers of a codon Met 326  mutation in the p85 subunit of phosphatidylinositol-3 (PI3)-kinase (about 2% of the Caucasian population) have lower glucose tolerance, glucose effectiveness. A further Asp to  polymorphism has been identified at codon 905 of the gene encoding the regulatory subunit of glycogen-associated protein phosphatase-1 (PP1G). Individuals who are heterozygous for this polymorphism constitute 18% of the Caucasian population and appear to exhibit both tissue-specific and pathway-specific insulin resistance. It is likely that inherited insulin resistance will eventually prove to be related to subtle mutations in many such genes of the insulin signalling network and the numerous genetic components controlling  metabolism.</p><h2><a href=https://www.ncbi.nlm.nih.gov/pubmed/21177445>Changes in chemical composition in male turkeys during growth.</a></h2><p>In growing animals, requirements for many nutrients (and ) are determined by the retention of these nutrients. During growth, this retention changes in an absolute way and also between nutrients and , resulting in changing nutrient requirements. The objective of this study was to describe the changes in chemical composition in male growing turkeys. The serial slaughter technique was used to determine the composition of amino acids, lipid, ash, and water in feather-free body (FFB) and feathers in male turkeys offered feed ad libitum from 1 to 15 wk of age. Allometric relations were used to describe changes in body composition. The feather content in the body decreased from 6% at 1 wk of age to less than 3% at 15 wk of age. The water and protein content in FFB decreased with increasing FFB mass, with allometric scalars (b) of, respectively, 0.967 and 0.970, whereas the lipid content increased with increasing FFB mass (b = 1.388). The water, protein, and ash content in fat-free FFB was constant and represented, respectively, 71.6, 24.2, and 4.2% of the fat-free FFB mass. The amino acid content of FFB protein was relatively constant and only the Cys content decreased between 1 and 15 wk of age, whereas the  content increased. Feathers were mostly composed of protein, and the protein content did not change during growth. During growth, the Lys, Met, Trp, His, , Asp, and Glu contents in feather protein decreased, whereas the Cys, Val, and Ser contents increased. The contribution of feathers to whole-body amino acid retention ranged from 5% for His to 33% for Cys. On average, the weight gain of FFB contained 21.3% protein and 12.7% lipid, corresponding to an  content of 10.1 kJ/g. The weight gain of feathers contained 87.4% protein, corresponding to an  content of 20.4 kJ/g. The results of the present study can be used in a factorial approach to determine nutrient requirements in growing turkeys.</p><h2><a href=https://www.ncbi.nlm.nih.gov/pubmed/31079216>Novel stable analogues of the neurotensin C-terminal hexapeptide containing unnatural amino acids.</a></h2><p>Neurotensin (NT) (pGlu-Leu--Glu-Asn-Lys-Pro-Arg-Arg-Pro---Leu) exerts a dual function as a neurotransmitter/neuromodulator in the central nervous system and as a hormone/cellular mediator in periphery. This dual function of NT establishes a connection between brain and peripheral tissues that renders this peptide a central player in  homeostasis. Many biological actions of NT are mediated through its interaction with three types of NT receptors (NTS receptors). Despite its role in  homeostasis, NT has a short half-life that hampers further determination of the biological actions of this peptide and its receptors in brain and periphery. The short half-life of NT is due to the proteolytic degradation of its C-terminal side by several endopeptidases. Therefore, it is important to synthesize NT analogues with resistant bonds against metabolic deactivation. Based on these findings, we herein report the synthesis of ten linear, two cyclic and two dimeric analogues of NT with modifications in its structure that improve their metabolic stability, while retaining the ability to bind to NTS receptors. Modifications at position 11 (introduction of D-Tyrosine (OEthyl) [D-(Et)] or D-1-naphtylalanine [D-1-Nal] were combined with introduction of a L-Lysine or a D-Arginine at positions 8 or 9, and 1-[2-(aminophenyl)-2-oxoethyl]-1H-pyrrole-2-carboxylic acid (AOPC) at positions 7 or 8, resulting in compounds NT4-NT21. AOPC is an unnatural amino acid with promise in applications as a building block for the synthesis of peptidomimetic compounds. To biologically evaluate these analogues, we determined their plasma stability and their binding affinities to type 1 NT receptor (NTS1), endogenously expressed in HT-29 cells, Among the fourteen NT analogues, compounds, NT5, NT6, and NT8, which have D-(Et) at position 11, bound to NTS1 in a dose-response manner and with relatively high affinity but still lower than that of the natural peptide. Despite their lower binding affinities compared to NT, the NT5, NT6, and NT8 exhibited a remarkably higher stability, as a result of their chemistry, which provides protection from enzymatic activity. These results will set the basis for the rational design of novel NT molecules with improved pharmacological properties and enhanced enzymatic stability.</p><h2><a href=https://www.ncbi.nlm.nih.gov/pubmed/11389076>Cyclic hydroxamic-acid-containing peptide 31, a potent synthetic histone deacetylase inhibitor with antitumor activity.</a></h2><p>Cyclic hydroxamic-acid-containing peptide 1 (CHAP1), designed as a hybrid of trichostatin A and trapoxin, is a lead compound for the development of potent inhibitors of histone deacetylase (HDAC). In this study, we synthesized a series of CHAP derivatives and evaluated their biological activities by monitoring the potency of their inhibition of HDAC activity, their ability to augment the expression of MHC class-I molecules in B16/BL6 cells, and their effect on cell proliferation. A structure-activity relationship study using these three assay systems revealed several requirements of their structure for the strong inhibition of HDAC not only in the cell-free situation, but also in cells. When the structures of CHAP derivatives are represented as cyclo(-Asu(NHOH)-AA(2)-AA(3)-Pro or Pip-)(n), where Asu(NHOH) and Pip are zeta-hydroxamide-alpha-aminosuberic acid and pipecolic acid, respectively, (a) the tetrapeptide structure (n = 1) was better than the octapeptide one (n = 2); (b) AA(2) and AA(3) should be hydrophobic; and (c) the combination of amino acid chirality should be LDLD for the strongest inhibition of HDAC in cells (LDLD > LLLD, LDLL > LLDL). cyclo(-L-Asu(NHOH)-D-(Me)-L--D-Pro-) or CHAP31 was selected as one of the strongest CHAPs, and its biological activity was characterized further. CHAP31 was much more stable in the presence of cultured cells (t(1/2) > 3000 h) than trichostatin A (t(1/2) = 14.7 h) or trapoxin A (t(1/2) = 2.10 h). CHAP31 exhibited antitumor activity in C57BL x DBA/2 F(1) (BD2F(1)) mice bearing B16/BL6 tumor cells. Furthermore, CHAP31 inhibited the growth in four of five human tumor lines implanted into nude mice. These results suggest CHAP31 to be promising as a novel therapeutic agent for cancer treatment.</p><h2><a href=https://www.ncbi.nlm.nih.gov/pubmed/28046184>Nutritive value of extruded or multi-enzyme supplemented cold-pressed soybean cake for pigs.</a></h2><p>The objectives were to determine the standardized ileal digestibility (SID) of AA and NE value of cold-pressed soybean cake (CP-SBC), and the effect of extrusion or adding multi-enzyme to CP-SBC diet for growing pigs. Eight ileal-cannulated pigs (initial BW = 79.7 ± 3.97 kg) were fed 4 diets in a replicated 4 × 4 Latin square design to give 8 replicates per diet. Diets included a cornstarch-based diet with CP-SBC, extruded CP-SBC, and SBC plus multi-enzyme (1,200 U of xylanase, 150 U of glucanase, 500 U of cellulase, 60 U of mannanase, 700 U of invertase, 5,000 U of protease, and 12,000 U of amylase/kilogram of diet; Superzyme-CS, 0.5 g/kg); and a N-free diet. The CP-SBC was the sole source of protein in the CP-SBC-containing diets. The ratio of cornstarch to sugar and soybean oil in CP-SBC-containing diets was identical to the N-free diet to allow calculation of  digestibility of CP-SBC by the difference method. The evaluated CP-SBC had been produced by heating the soybean seed at 105°C for 60 min followed by pressing of the heated soybean seeds at less than 42°C (barrel temperature). On a DM basis, CP-SBC and extruded CP-SBC contained 47.8 and 47.1% CP, 15.6 and 10.5% ADF, 7.23 and 8.85% ether extract, 3.11 and 3.08% Lys, and 2.25 and 3.70 trypsin inhibitor units per mg, respectively. Extrusion increased ( < 0.001) the SID of AA for the CP-SBC by an average of 12%. Also, extrusion increased ( < 0.001) the NE value of the CP-SBC from 2,743 to 2,853 kcal/kg of DM. Supplementation of CP-SBC diet with the multi-enzyme increased ( < 0.05) the SID of Arg and Pro, and tended to increase ( < 0.1) the SID of  and . However, the multi-enzyme supplementation did not affect the NE value of CP-SBC. In conclusion, the CP-SBC evaluated in the present study could be an alternative source of AA and  in swine diets, and its nutritive value can be increased by extrusion following cold-pressing. The multi-enzyme used in this study improved the digestibility of some AA, but had limited effect on  digestibility and hence NE value of the CP-SBC.</p><h2><a href=https://www.ncbi.nlm.nih.gov/pubmed/7473728>Calculation of relative binding free energies and configurational entropies: a structural and thermodynamic analysis of the nature of non-polar binding of thrombin inhibitors based on hirudin55-65.</a></h2><p>Free  calculations were carried out on a series of exosite-binding inhibitors of thrombin. These inhibitors are based on the C-terminal fragment of hirudin and have the sequence Phe-Glu-Glu-IleH59-Pro-Glu-Glu-- Leu, where the superscript over  indicates its relative position in the natural sequence of hirudin. In this study, the effect of replacing IleH59 with ten other non-polar amino acids was examined. Three preferred interaction sites for methyl/methylene groups for the various XaaH59 side-chains in the complex were identified from conformational search calculations. The corresponding thermodynamic changes were determined using a combination of systematic search and  minimization in a manner that locates the local minima in the system and in the process simultaneously builds up the partition function. The free , internal  and entropic contributions are readily calculated from the partition function. Very good agreement in the resulting relative binding free energies was obtained between theory and experiment. The calculations allowed us to dissect out the enthalpic, entropic and solvation contributions to delta delta G. The contribution from desolvation was found to be relatively weak. The binding of these non-polar side-chains to thrombin is found to be driven mainly by favorable protein-ligand interactions rather than by the desire for non-polar groups to be desolvated. We also find that the configurational entropy contributes about 0.48 kcal/mol (0.81 kappa T) in average for each torsional angle "frozen" in binding.</p><h2><a href=https://www.ncbi.nlm.nih.gov/pubmed/19887450>Distinctive properties of the hyaluronan-binding domain in the lymphatic endothelial receptor Lyve-1 and their implications for receptor function.</a></h2><p>The lymphatic endothelial hyaluronan (HA) receptor Lyve-1 is a member of the Link protein superfamily most similar to the leukocyte HA receptor CD44. However, the structure of Lyve-1 and the nature of its interaction with ligand are obscure. Here we present new evidence that Lyve-1 is functionally distinct from CD44. Using truncation mutagenesis we confirm that Lyve-1 in common with CD44 contains an extended HA-binding unit, comprising elements flanking the N and C termini of the consensus lectin-like Link module, bridged by a third conserved disulfide linkage that is critical for HA binding. In addition, we identify six essential residues -87, -97, Arg-99, Asn-103, Lys-105, and Lys-108 that define a compact HA-binding surface on Lyve-1, encompassing the epitope for an adhesion-blocking monoclonal antibody 3A, in an analogous position to the HA-binding surface in CD44. The overtly electrostatic character of HA binding in Lyve-1 and its sensitivity to ionic strength (IC(50) of 150 mm NaCl) contrast markedly with CD44 (IC(50) > 2 m NaCl) in which HA binding is mediated by hydrogen bonding and hydrophobic interactions. In addition, unlike the extended Link module in CD44, which binds HA efficiently when expressed as a soluble monomer (K(d) = 65.7 mum), that of Lyve-1 requires artificial dimerization, although the full ectodomain is active as a monomer (K(d) = 35.6 mum). Finally, full-length Lyve-1 did not form stable dimers in binding-competent 293T transfectants when assessed using bioluminescent resonance  transfer. These results reveal that elements additional to the extended Link module are required to stabilize HA binding in Lyve-1 and indicate important structural and functional differences with CD44.</p><h2><a href=https://www.ncbi.nlm.nih.gov/pubmed/11563031>Restricted rotation in chiral peptide nucleic acid (PNA) monomers--influence of substituents studied by means of 1H NMR.</a></h2><p>The influence of amino acid side chains [derived from: Ala, Val, Leu, , Phe, (Bzl), Ser(Bzl), Thr(Bzl), Pro, Trp], incorporated into "aminoalkyl" part of PNA monomers, on the temperature-dependent distributions of rotamers about the tertiary amide bond was studied by means of 1H NMR at 0, 25 and 40 degrees C in CDCl3. The delta G0 values of the  differences between individual rotamers were calculated. The results may be helpful in the designing of monomers with desirable properties.</p><h2><a href=https://www.ncbi.nlm.nih.gov/pubmed/18555004>Yeast sterol C24-methyltransferase: role of highly conserved tyrosine-81 in catalytic competence studied by site-directed mutagenesis and thermodynamic analysis.</a></h2><p>The role of Try-81 in the reaction catalyzed by Saccharomyces cerevisiae sterol 24-C-methyltransferase (Erg6p) was investigated kinetically and for product differences against a panel of position-81 mutants in which  was substituted with Trp, Phe, , Leu, Val and Ala. The residue chosen for mutation is one that was reported previously to accept fecosterol and yield a set 24-ethyl (idene) sterol products typical of plants, showing the amino acid residue is located close to the transient C25 carbocation intermediate in the active site. One group of mutants (aromatic) tested with the natural substrate zymosterol accelerated the C-methylation reaction (k(cat)/K(m)) whereas the other group of mutants (aliphatics) decreased catalytic competence as the amino acid side chain was downsized. Mutating to aromatic and assaying with the substrate analog designed as a suicide substrate 26,27-dehydrozymosterol favored C26-monol formation, whereas mutating to the aliphatic of smaller size favored C26-diol formation (a measure of enzyme alkylation). In no case was zymosterol converted to an intermediate that formed a C25-diol. Thermodynamic analysis (determination of E(a), DeltaG(double dagger), DeltaH(double dagger) and TDeltaS(double dagger)) for the C-methylation reaction performed by these enzymes assayed with the substrate and its analog or zymosterol paired with the "charged' high  intermediate (HEI) analogs 24(R,S)25,epiminolanosterol and 25-azalanosterol or "neutral" membrane insert ergosterol showed that mutation to aromatics can reduce inhibitor potency (measured as K(m)/K(i)), yet catalysis can improve in Trp81 by the introduction of a gain in free  associated with stabilization of the transition state of a rate-controlling step directed toward turnover. Alternatively, mutation to the smaller aliphatic amino acid side chains led to a destabilization in the active site structure which was accompanied by increases in the partition ratios associated with abortive complex formation. The results are explained by consideration of the functional differences attributed to Tyr81 substitution to aromatics and aliphatics of different size involved with cation-pi or hydrogen bonding interactions and in the activation barriers required of differing side chain conformations to orient the reactants in the direction of turnover versus enzyme inactivation.</p><h2><a href=https://www.ncbi.nlm.nih.gov/pubmed/6487083>[Effect of the content of crude plant protein in feed on the utilization of urea in dairy cattle. 4. Passage of amino acids through the duodenum].</a></h2><p>Model investigations carried out by means of a re-entrant method at the duodenum with dairy cows and with -equivalent rations (600 EFU cattle/kg DM), which after the supplementation of 11 g urea/kg DM (150 g/animal and day) contained 13.8 (I), 16.7 (II) and 20.2 (III)% crude protein, had the following results: The amino acid profile of the duodenal protein differed from that of the applied rations and was similar to the microbial protein in the rumen. It only changed inconsiderably with the protein level of the ration. There were differences between the amino acids of the feed as to their apparent rate of fermentation in the rumen. Ser, , lys, and leu apparently proved to be most resistent whereas arg, asp, met,  and phe were apparently fermented for the most part or completely. The passage rate of amino acids at the duodenum related to intake were, from I ... III 126, 102 and 87%. With all rations there were always net losses of some amino acids (his, arg) and net gains (lys, thr, ser, gly, val, met and ) at the duodenum in comparison intake. 15N from urea was incorporated into all the amino acids determined in the duodenal protein, most intensively into glu, ser, asp, , gly, his and leu. The apparent digestibility of the total amino acids in the intestines was independent of the crude protein level of the rations and reached 69 ... 71% with a deviation range of 59 ... 81% for the individual amino acids.(ABSTRACT TRUNCATED AT 250 WORDS).</p><h2><a href=https://www.ncbi.nlm.nih.gov/pubmed/27064538>Hyperactive Editing Domain Variants Switch the Stereospecificity of Tyrosyl-tRNA Synthetase.</a></h2><p>d-Amino acids are excluded at three different steps during protein synthesis: the aminoacylation of tRNA, binding of aminoacyl-tRNAs to EF-Tu, and selection of the aminoacyl-tRNA by the ribosome. We previously altered the enantioselectivity of tyrosyl-tRNA synthetase (TyrRS) by inserting the editing domain from phenylalanyl-tRNA synthetase (FRSed) between Gly 161 and  162 in tyrosyl-tRNA synthetase (the editing domain hydrolyzes l--tRNA but not d--tRNA). In this paper, we test the hypothesis that the enantioselectivity of this TyrRS-FRSed chimera can be shifted further toward the formation of d--tRNA by introducing activating mutations into the editing site. Yokoyama and colleagues previously identified six alanine substitutions in phenylalanyl-tRNA synthetase that increase its editing activity.1 We have introduced these alanine substitutions into TyrRS-FRSed in various combinations, generating 14 different variants. To analyze their editing activity, we developed a continuous, spectrophotometric, steady-state post-transfer editing assay in which l--tRNA is generated in situ, resulting in the release of one molecule of AMP during each editing cycle. Post-transfer editing is monitored by coupling the release of AMP to the reduction of NAD(+) (via the actions of AMP deaminase and IMP dehydrogenase), resulting in an increase in absorbance at 340 nm. In general, TyrRS-FRSed variants containing two activating mutations are the most active, with additional alanine substitutions decreasing the activity of the editing domain. Linear free  relationships indicate that high kcat values are correlated with high binding affinities for l--tRNA. Lastly, competition assays indicate that at least one TyrRS-FRSed variant (F145A/S211A) preferentially aminoacylates tRNA with d-tyrosine, demonstrating that the enantioselectivity of tyrosyl-tRNA synthetase can be inverted using hyperactive editing domains.</p><h2><a href=https://www.ncbi.nlm.nih.gov/pubmed/24478310>Claudin-3 and claudin-5 protein folding and assembly into the tight junction are controlled by non-conserved residues in the transmembrane 3 (TM3) and extracellular loop 2 (ECL2) segments.</a></h2><p>The mechanism of tight junction (TJ) assembly and the structure of claudins (Cldn) that form the TJ strands are unclear. This limits the molecular understanding of paracellular barriers and strategies for drug delivery across tissue barriers. Cldn3 and Cldn5 are both common in the blood-brain barrier but form TJ strands with different ultrastructures. To identify the molecular determinants of folding and assembly of these classic claudins, Cldn3/Cldn5 chimeric mutants were generated and analyzed by cellular reconstitution of TJ strands, live cell confocal imaging, and freeze-fracture electron microscopy. A comprehensive screening was performed on the basis of the rescue of mutants deficient for strand formation. Cldn3/Cldn5 residues in transmembrane segment 3, TM3 (Ala-127/Cys-128, Ser-136/Cys-137, Ser-138/Phe-139), and the transition of TM3 to extracellular loop 2, ECL2 (Thr-141/-142) and ECL2 (Asn-148/Asp-149, Leu-150/Thr-151, Arg-157/-158), were identified to be involved in claudin folding and/or assembly. Blue native PAGE and FRET assays revealed 1% n-dodecyl β-d-maltoside-resistant cis-dimerization for Cldn5 but not for Cldn3. This homophilic interaction was found to be stabilized by residues in TM3. The resulting subtype-specific cis-dimer is suggested to be a subunit of polymeric TJ strands and contributes to the specific ultrastructure of the TJ detected by freeze-fracture electron microscopy. In particular, the Cldn5-like exoplasmic face-associated and particle-type strands were found to be related to cis-dimerization. These results provide new insight into the mechanisms of paracellular barrier formation by demonstrating that defined non-conserved residues in TM3 and ECL2 of classic claudins contribute to the formation of TJ strands with differing ultrastructures.</p><h2><a href=https://www.ncbi.nlm.nih.gov/pubmed/31265780>[Analysis of amino acid composition of skim milk and buttermilk for the production of dairy drink when introducing whey protein hydrolysate].</a></h2><p>The imbalance of macronutrient composition of people's diets determines the relevance of the functional foods development - sources of protein and essential amino acids.  of the work was to analyze the amino acid composition and determine the functional significance of the product based on skim milk and buttermilk byadding whey protein hydrolysate. . The control options were samples of skimmed milk, buttermilk and their mixtures in a 1:1 ratio. From 1 to 3% of the whey protein hydrolysate with a degree of hydrolysis of about 60% of peptide bonds were added to the same milk base samplesin the experimental options. The amino acid composition was analyzed by high performance liquid chromatography using a cation exchange analyzer and subsequent post-column derivatization with ninhydrin. . The content of essential amino acids per 100 g of whey protein hydrolysate (WPH) is 50.6-168.0% from the adequate level of their daily intake, while there are only 3.0-22.3 and 2.6-19.9% in skimmed milk and buttermilk, respectively. Despite the practically equal protein content in skimmed milk and buttermilk 3.36±0.02 and 3.09±0.01%, respectively, the absolute content of all essential amino acids in buttermilk is lower than in skimmed milk. This can be explained by differences in the qualitative composition of proteins in these types of dairy raw materials. . The amino acid composition of the milk base, consisting of buttermilk and skimmed milk mixture in a 1:1 ratio, can be presented as «functional» product in relation to Val, , Leu, Lis, Trp and , due to the addition of 3% WPH, because the product portion of 200 g provides 15% of adequate daily intake of these essential amino acids. There is the reason to include additional information: low-fat, low-calorie, contains only natural sugars and a source of protein for people whose protein requirement does not exceed 99 grams per day in the labeling of the developed product, in addition to its nutritional and  value.Copyright© GEOTAR-Media Publishing Group.</p><h2><a href=https://www.ncbi.nlm.nih.gov/pubmed/28201818>Collective repacking reveals that the structures of protein cores are uniquely specified by steric repulsive interactions.</a></h2><p>Protein core repacking is a standard test of protein modeling software. A recent study of six different modeling software packages showed that they are more successful at predicting side chain conformations of core compared to surface residues. All the modeling software tested have multicomponent  functions, typically including contributions from solvation, electrostatics, hydrogen bonding and Lennard-Jones interactions in addition to statistical terms based on observed protein structures. We investigated to what extent a simplified  function that includes only stereochemical constraints and repulsive hard-sphere interactions can correctly repack protein cores. For single residue and collective repacking, the hard-sphere model accurately recapitulates the observed side chain conformations for , Leu, Phe, Thr, Trp,  and Val. This result shows that there are no alternative, sterically allowed side chain conformations of core residues. Analysis of the same set of protein cores using the Rosetta software suite revealed that the hard-sphere model and Rosetta perform equally well on , Leu, Phe, Thr and Val; the hard-sphere model performs better on Trp and  and Rosetta performs better on Ser. We conclude that the high prediction accuracy in protein cores obtained by protein modeling software and our simplified hard-sphere approach reflects the high density of protein cores and dominance of steric repulsion.© The Author 2017. Published by Oxford University Press. All rights reserved. For Permissions, please e-mail: journals.permissions@oup.com.</p><h2><a href=https://www.ncbi.nlm.nih.gov/pubmed/19745556>Structural and mutagenic approach to create human serum albumin-based oxygen carrier and photosensitizer.</a></h2><p>Human serum albumin (HSA) is a versatile protein found at high concentration in blood plasma and binds a range of insoluble endogenous and exogenous compounds. We have shown that complexation of functional molecules into HSA creates unique proteins never seen in nature. Complexing an iron-protoporphyrin IX into a genetically engineered heme pocket of recombinant HSA (rHSA) generates an artificial hemoprotein, which binds O(2) reversibly in much the same way as hemoglobin. A pair of site-specific mutations, (i) introduction of a proximal histidine at the -142 position and (ii) substitution of -161 with Phe or Leu, allows the heme to bind O(2). Additional modification on the distal side of the heme pocket provides rHSA(triple mutant)-heme complexes with a variety of O(2) binding affinity. Complexing a carboxy-C(60)-fullerene (CF) into HSA generates a protein photosensitizer for photodynamic cancer therapy.  transfer occurs from a photoexcited triplet-state of HSA-CF (HSA-(3)CF(*)) to O(2), forming singlet oxygen ((1)O(2)). This protein does not show dark cytotoxicity, but induceds cell death under visible light irradiation.</p><h2><a href=https://www.ncbi.nlm.nih.gov/pubmed/8316244>[Superspiralized alpha-spiral. 2. Conformational analysis of a model fragment of the dual-chain parallel superspiralized alpha-spiral].</a></h2><p>To estimate the role of nonbonded interactions in the folding of the hydrophobic core of a two-stranded parallel alpha-helical coiled-coil, a conformational analysis was carried out for the model fragment: [formula: see text] with a 2-fold symmetry axis running between the parallel helices, where X denotes nonpolar residues Ala, Leu, Val, , Met, Phe, , Trp. For every residue X, the  of interaction between two helices and the complete conformational , were mapped as functions of (R, nu 0) where R is radius of coiled coil and nu 0 is the angle that determines orientation of two helices relatively to each other. It was shown that when Phe, ,  and Val were located in the d position of repeating heptapeptide, there appeared certain steric restrictions. The obtained results allow one to explain certain peculiarities in the distribution of nonpolar residues in coiled-coil structure in fibrous proteins, i.e., a high percentage of Leu in the hydrophobic core of these structures; preference of a position for Val and  compared to d position, and a rare occurrence of Phe in d position.</p><h2><a href=https://www.ncbi.nlm.nih.gov/pubmed/8718869>Mechanistic and structural contributions of critical surface and internal residues to cytochrome c electron transfer reactivity.</a></h2><p>The influence of mutations in two conserved regions of yeast iso-1-cytochrome c believed to be critical to the mechanism of cytochrome c electron transfer reactions has been investigated. The variants Asn52Ala, Tyr67Phe, Ile75Met, and Thr78Gly involve perturbation of critical hydrogen-bonding interactions with an internal water molecule (Wat166) and have been studied in terms of their electrochemical properties and the kinetics with which they are reduced by Fe(EDTA)2- and oxidized by Co(phen)3(3+). In parallel studies, the Co(phen)3(3+) oxidation kinetics of , Leu, , Ala, Ser, and Gly variants of the phylogenetically conserved residue Phe82 have been studied and correlated with previous electrochemical and kinetic results. To assist mechanistic interpretation of these results, the three-dimensional structures of the Asn52Ala and Ile75Met ferrocytochrome c variants have been determined. The reduction potentials of the variants modified in the region of Wat166 were at least 33 mV (pH 6, 25 degrees C, and mu = 0.1 M) lower than that of the wild-type protein. Electron transfer reactivity of this family of variants in both the oxidation and reduction reactions was increased as much as 10-fold over that of the wild-type cytochrome. On the other hand, the reactivity of the position-82 variants in both oxidation and reduction depended on the structural characteristics of the oxidation-reduction reagent with which they reacted, and this reactivity was related to the nature of the residue at this position. These findings have been interpreted as demonstrating that the principal influence of modification at position-82 arises from changes in the nature of reactant-protein interaction at the surface of the protein and in maintaining the high reduction potential of the cytochrome while the principal influence of internal modifications near Wat166 results from alteration of the reorganization  for the oxidation state-linked conformational change defined by crystallographic analysis of the wild-type protein.</p><h2><a href=https://www.ncbi.nlm.nih.gov/pubmed/9558312>Molecular basis of resistance to HIV-1 protease inhibition: a plausible hypothesis.</a></h2><p>The binding thermodynamics of the HIV-1 protease inhibitor acetyl pepstatin and the substrate Val-Ser-Gln-Asn--Pro--Val-Gln, corresponding to one of the cleavage sites in the gag, gag-pol polyproteins, have been measured by direct microcalorimetric analysis. The results indicate that the binding of the peptide substrate or peptide inhibitor is entropically driven; i.e., it is characterized by an unfavorable enthalpy and a favorable entropy change, in agreement with a structure-based thermodynamic analysis based upon an empirical parameterization of the energetics. Dissection of the binding enthalpy indicates that the intrinsic interactions are favorable and that the unfavorable enthalpy originates from the  cost of rearranging the flap region in the protease molecule. In addition, the binding is coupled to a negative heat capacity change. The dominant binding force is the increase in solvent entropy that accompanies the burial of a significant hydrophobic surface. Comparison of the binding energetics obtained for the substrate with that obtained for synthetic nonpeptide inhibitors indicates that the major difference is in the magnitude of the conformational entropy change. In solution, the peptide substrate has a higher flexibility than the synthetic inhibitors and therefore suffers a higher conformational entropy loss upon binding. This higher entropy loss accounts for the lower binding affinity of the substrate. On the other hand, due to its higher flexibility, the peptide substrate is more amenable to adapt to backbone rearrangements or subtle conformational changes induced by mutations in the protease. The synthetic inhibitors are less flexible, and their capacity to adapt is more restricted. The expected result is a more pronounced effect of mutations on the binding affinity of the synthetic inhibitors. On the basis of the thermodynamic differences in the mode of binding of substrate and synthetic inhibitors, it appears that a key factor to understanding resistance is given by the relative balance of the different forces that contribute to the binding free  and, in particular, the balance between conformational and solvation entropy.</p><h2><a href=https://www.ncbi.nlm.nih.gov/pubmed/24226385>Interaction of angiotensin peptides and zinc metal ions probed by electrospray ionization mass spectrometry.</a></h2><p>Electrospray ionization-tandem mass spectrometry experiments were used to provide evidence regarding the sites of interactions between zinc metal ions and angiotensin peptides. The electrospray ionization mass spectra of histidine-containing human angiotensin II (Asp-Arg-Val---His-Pro-Phe) and angiotensin I (Asp-Arg-Val---His-Pro-Phe-His-Leu) in the presence of zinc show abundant multiply charged ions for the zinc-attached peptide [M + aZn(2+) +(c - 2a)H(+)](c+), where a = 1, 2 and c is charge. From collisionally activated dissociation experiments, with both low  (triple quadrupole mass spectrometry) and high  collisions (linked scan at constant B/E with a double focusing instrument) of the [M + Zn](2+) and [M + Zn + H](3+) ions for angiotensin II, a [b 6 + Zn](2+) species is produced as the most abundant product ion, suggesting that the zinc interaction site is in the vicinity of the His(6) residue. Additionally, tandem mass spectra from the zinc-attached ions for angiotensin I show abundant [b 6 + Zn](2+) and [b 9 + Zn](2+) products, providing evidence that both His(6) and His(9) are involved in zinc coordination.</p><h2><a href=https://www.ncbi.nlm.nih.gov/pubmed/7670373>Interactions between hydrophobic side chains within alpha-helices.</a></h2><p>The thermodynamic basis of helix stability in peptides and proteins is a topic of considerable interest. Accordingly, we have computed the interactions between side chains of all hydrophobic residue pairs and selected triples in a model helix, using Boltzmann-weighted exhaustive modeling. Specifically, all possible pairs from the set Ala, Cys, His, , Leu, Met, Phe, Trp, , and Val were modeled at spacings of (i, i + 2), (i, i + 3), and (i, i + 4) in the central turn of a model poly-alanyl alpha-helix. Significant interactions--both stabilizing and destabilizing-- were found to occur at spacings of (i, i + 3) and (i, i + 4), particularly in side chains with rings (i.e., Phe, , Trp, and His). In addition, modeling of leucine triples in a helix showed that the free  can exceed the sum of pairwise interactions in certain cases. Our calculated interaction values both rationalize recent experimental data and provide previously unavailable estimates of the constituent energies and entropies of interaction.</p><h2><a href=https://www.ncbi.nlm.nih.gov/pubmed/25568365>The efficacy of a new 6-phytase obtained from Buttiauxella spp. expressed in Trichoderma reesei on digestibility of amino acids, , and nutrients in pigs fed a diet based on corn, soybean meal, wheat middlings, and corn distillers' dried grains with solubles.</a></h2><p>Sixteen cannulated pigs were used to evaluate the effect of a new 6-phytase derived from Buttiauxella spp. and expressed in Trichoderma reesei on apparent ileal digestibility (AID) of AA and apparent total tract digestibility (ATTD) of DM, N, Ca, P, Na, Mg, K, Cl, and . Pigs were fed 4 diets for 2 periods in a crossover design. Within each period, there were 4 blocks of 4 pigs per block with each diet represented within each block. The average initial BW in periods 1 and 2 were 22 and 30 kg, respectively. Each period lasted 9 d with fecal collection on d 5 and 6 and a 12-h ileal digesta collection on d 7, 8, and 9. Pigs received a daily feed allowance of approximately 4.5% of their BW. The experimental diets were based on corn, soybean meal, wheat middlings, and corn distillers dried grain with solubles. Phytase was added at 0; 500; 1,000; or 2,000 phytase units/kg of diet to a basal diet that contained 205, 15, 5.4, and 10 g of CP, Lys, total P (1.6 g of nonphytate P), and Ca/kg diet, respectively. The addition of phytase improved (P < 0.05) AID of DM, N, Ca, and P. Increasing phytase supplementation linearly and quadratically increased (P < 0.05) AID of P and Ca, respectively, with AID of Ca showing a tendency for a linear increase (P = 0.053). Phytase supplementation of the basal diet improved (P < 0.05) AID of P from 46 to 62%. Phytase supplementation increased (P < 0.05) ATTD of DM, N, Ca, P, Mg, K, and . Contrasts showed that phytase supplementation of the basal diet increased (P < 0.05) AID for 8 indispensable AA (Arg, His, , Leu, Lys, Phe, Thr, and Val), 6 dispensable AA (Ala, Asp, Cys, Glu, Ser, and ), as well as for total AA. Furthermore, phytase supplementation to the basal diet showed a tendency (P < 0.10) to increase ileal digestibility of Gly. Ileal digestibility of Met, Trp, and Pro were not affected by phytase supplementation. Increasing the level of phytase supplementation resulted in linear increases (P < 0.05) in AID of 6 indispensable AA (Arg, , Leu, Lys, Phe, and Val) and 1 dispensable AA (Asp) with 4 AA (His, Cys, Glu, and ) showing a tendency for linear increase (P < 0.10) in AID of AA. The results from this study showed that in addition to increasing P and Ca utilization, the new Buttiauxella 6-phytase expressed in Trichoderma reesei enhanced ileal digestibility of N and several AA in growing pigs in a dose-dependent manner.</p><h2><a href=https://www.ncbi.nlm.nih.gov/pubmed/30785183>Morelloflavone as a novel inhibitor of mitotic kinesin Eg5.</a></h2><p>Among 40 plant-derived biflavonoids with inhibitory potential against Eg5, morelloflavone from Garcinia dulcis leaves was selected for further testing based on in silico analysis of binding modes, molecular interactions, binding energies and functional groups that interact with Eg5. Computational models predicted that morelloflavone binds the putative allosteric pocket of Eg5, within the cavity surrounded by amino acid residues of -136, Glu-116, Glu-118, Trp-127, Gly-117, Ala-133, Glu-215, Leu-214 and -211. Binding  was -8.4 kcal/mol, with a single hydrogen bond formed between morelloflavone and -211. The binding configuration was comparable to that of a reference inhibitor, S-trityl-L-cysteine. Subsequent biochemical analysis in vitro confirmed that morelloflavone inhibited both the basal and microtubule-activated ATPase activity of Eg5 in a manner that does not compete with ATP binding. Morelloflavone also suppressed Eg5 gliding along microtubules. These results suggest that morelloflavone binds the allosteric binding site in Eg5 and thereby inhibits ATPase activity and motor function of Eg5.© The Author(s) 2019. Published by Oxford University Press on behalf of the Japanese Biochemical Society. All rights reserved.</p><h2><a href=https://www.ncbi.nlm.nih.gov/pubmed/22511205>Metabolic signatures of insulin resistance in 7,098 young adults.</a></h2><p>Metabolite associations with insulin resistance were studied in 7,098 young Finns (age 31 ± 3 years; 52% women) to elucidate underlying metabolic pathways. Insulin resistance was assessed by the homeostasis model (HOMA-IR) and circulating metabolites quantified by high-throughput nuclear magnetic resonance spectroscopy in two population-based cohorts. Associations were analyzed using regression models adjusted for age, waist, and standard lipids. Branched-chain and aromatic amino acids, gluconeogenesis intermediates, ketone bodies, and fatty acid composition and saturation were associated with HOMA-IR (P < 0.0005 for 20 metabolite measures). Leu, , Val, and  displayed sex- and obesity-dependent interactions, with associations being significant for women only if they were abdominally obese. Origins of fasting metabolite levels were studied with dietary and physical activity data. Here, protein  intake was associated with Val, Phe, , and Gln but not insulin resistance index. We further tested if 12 genetic variants regulating the metabolites also contributed to insulin resistance. The genetic determinants of metabolite levels were not associated with HOMA-IR, with the exception of a variant in GCKR associated with 12 metabolites, including amino acids (P < 0.0005). Nonetheless, metabolic signatures extending beyond obesity and lipid abnormalities reflected the degree of insulin resistance evidenced in young, normoglycemic adults with sex-specific fingerprints.</p><h2><a href=https://www.ncbi.nlm.nih.gov/pubmed/25568370>Nutrient digestibility of solvent-extracted Brassica napus and Brassica juncea canola meals and their air-classified fractions fed to ileal-cannulated grower pigs.</a></h2><p> and nutrient digestibility of solvent-extracted canola meal (CM) is limited in pigs by its relatively high fiber content. The seed hull, which greatly contributes to the fiber content of CM, is denser than the oil-free cotyledon. By utilizing streams of air, air classification partially separates these seed components on the basis of their different sizes and densities to produce a low-fiber, light-particle fraction and a high-fiber, heavy-particle fraction. Compared with parent CM, ADF and NDF were reduced by 31.9% and 29.5% in the light-particle fraction and were enriched by 16.5% and 9.0% in the heavy-particle fraction (DM basis), respectively. Particle size was 638, 18.9, and 76.1 µm for the parent CM and light- and heavy-particle fractions, respectively. To determine the nutrient digestibility of CM and their air-classified fractions, Brassica napus and B. juncea CM and their 2 air-classified fractions were evaluated in a 2 × 3 factorial arrangement together with a basal diet and an N-free diet. The experiment was conducted as an 8 × 8 Latin square in which diets contained 40% B. napus or B. juncea CM or their air-classified fractions and 60% basal diet. Digesta data from pigs fed the N-free diet served to subtract basal endogenous AA losses. Eight ileal-cannulated barrows (32 kg initial BW) were fed the 8 diets at 2.7 times maintenance DE for eight 11-d periods. At the end of each period, feces were collected for 48 h, and ileal digesta were collected for two 12-h periods. The DE and calculated NE values and the apparent total tract digestibility (ATTD) of GE were 6.3%, 10.0%, and 7.8% greater (P < 0.001) for B. juncea CM than for B. napus CM; 6.1%, 10.8%, and 5.3% greater (P < 0.001) for the light-particle fraction than for parent CM; and 5.4%, 7.2%, and 3.8% lower (P < 0.001) for the heavy-particle fraction than for parent CM, respectively. The standardized ileal digestibilities (SID) of His, , Val, Asp, and  were greater (P < 0.05) for B. juncea CM than for B. napus CM. The SID of CP and AA were greater (P < 0.01) in the light-particle fraction than in the heavy-particle fraction. The SID of Trp, Glu, Pro, and  were greater (P < 0.05) in the light-particle fraction than in parent CM. In conclusion, B. juncea CM had greater  and AA digestibility than B. napus CM because of reduced fiber content. Air classification of CM increased its  and AA digestibility in the light-particle fraction for pigs because of the reduced dietary fiber content and decreased particle size.</p><h2><a href=https://www.ncbi.nlm.nih.gov/pubmed/8125961>Use of synthetic peptide libraries and phosphopeptide-selective mass spectrometry to probe protein kinase substrate specificity.</a></h2><p>To search for peptides which serve as substrates for protein kinases, an approach based on peptide libraries has been developed. These peptide libraries are chemically synthesized by a modified "divide-couple-recombine" strategy. After reaction with the kinase of interest, the most highly phosphorylated substrate (selected from the library) is identified using on-line liquid chromatography-electrospray mass spectrometry (LC-ESMS). Negative ion LC-ESMS with stepped collision  is used to identify phosphorylated peptides in the enzyme reactions. As predicted, the cAMP-dependent protein kinase is shown to preferentially phosphorylate Kemptide (Leu-Arg-Arg-Ala-Ser-Leu-Gly) in a library consisting of 19 variants of Kemptide substituted at position 2. Additional experiments have been carried out on the nonreceptor tyrosine kinase v-Abl using a peptide library based on the v-Src autophosphorylation site (Arg-Arg-Leu--Glu-Asp-Ala-Glu--Ala-Ala-Arg-Gly). These results indicate that  is the optimal residue at the position N-terminal to tyrosine. Individual peptides containing the Glu-Asp-Ala-- motif have Vmax/Km values 6-fold higher than the peptide based on the autophosphorylation site itself, confirming the results of the library experiments. This motif has been identified in several tyrosine kinases at a position in the sequence not previously reported to serve as a phosphorylation or autophosphorylation site.</p><h2><a href=https://www.ncbi.nlm.nih.gov/pubmed/7700867>Molecular dynamics simulations of HIV-1 protease with peptide substrate.</a></h2><p>Molecular dynamics simulations of human immunodeficiency virus (HIV)-1 protease with a model substrate were used to test if there is a stable  minimum for a proton that is equidistant from the four delta oxygen atoms of the two catalytic aspartic acids. The crystal structure of HIV-1 protease with a peptidic inhibitor was modified to model the peptide substrate Ser-Gln-Asn--Pro--Val-Gln for the starting geometry. A proton was positioned between the two closet oxygen atoms of the two catalytic aspartic acids, and close to the carbonyl oxygen of the scissile bond in the substrate. All crystallographic water molecules were included. Two molecular dynamics simulations were run: 30 ps with united-atom potentials and 40 ps using the more accurate all-atom potentials. The molecular dynamics used a new algorithm that increased the speed and allowed the elimination of a cut-off for non-bonded interactions and the inclusion of an 8 A shell of water molecules in the calculations. The overall structure of the protease dimer, including the catalytic aspartic acids, was stable during the course of the molecular dynamics simulations. The substrate and a water molecule, that is an important component of the binding site, were stable during the simulation using all-atom potentials, but more mobile when united-atom potentials were used. A Poincare map representation showed that the positions of the proton and its coordinating oxygen atoms were stable for 93% of both simulations, although many of the buried and poorly accessible water molecules exchanged with solvent. The proton has a stable minimum  position and maintains coordination with all four delta oxygen atoms of the two catalytic aspartic acids and the carbonyl oxygen of the scissile bond of the substrate. Therefore, a loosely bound hydrogen ion at this position will not be rapidly exchanged with solvent, and will rebond to either a catalytic aspartic acid or possibly the substrate. The implications for the reaction mechanism are discussed.</p><h2><a href=https://www.ncbi.nlm.nih.gov/pubmed/11420432>Effect of the N2 residue on the stability of the alpha-helix for all 20 amino acids.</a></h2><p>N2 is the second position in the alpha-helix. All 20 amino acids were placed in the N2 position of a synthetic helical peptide (CH(3)CO-[AXAAAAKAAAAKAAGY]-NH(2)) and the helix content was measured by circular dichroism spectroscopy at 273K. The dependence of peptide helicity on N2 residue identity has been used to determine a free- scale by analysis with a modified Lifson-Roig helix-coil theory that includes a parameter for the N2  (n2). The rank order of DeltaDeltaG((relative to Ala)) is Glu(-), Asp(-) > Ala > Glu(0), Leu, Val, Gln, Thr, , Ser, Met, Asp(0), His(0), Arg, Cys, Lys, Phe > Asn, > Gly, His(+), Pro, . The results correlate very well with N2 propensities in proteins, moderately well with N1 and helix interior preferences, and not at all with N-cap preferences. The strongest energetic effects result from interactions with the helix dipole, which favors negative charges at the helix N terminus. Hydrogen bonds to side chains at N2, such as Gln, Ser, and Thr, are weak, despite occurring frequently in protein crystal structures, in contrast to the N-cap position. This is because N-cap hydrogen bonds are close to linear, whereas N2 hydrogen bonds have poor geometry. These results can be used to modify protein stability rationally, help design helices, and improve prediction of helix location and stability.</p><h2><a href=https://www.ncbi.nlm.nih.gov/pubmed/2527512>Conformation of the metastasis-inhibiting laminin pentapeptide.</a></h2><p>The binding of cancer cells to the basement membrane glycoprotein laminin appears to be a critical step in the metastatic process. This binding can be inhibited competitively by a specific pentapeptide sequence (--Gly-Ser-Arg) of the laminin B1 chain, and this peptide can prevent metastasis formation in vivo. However, other similar pentapeptide sequences (e.g., --Gly-Ser-Glu) have been found to be much less active in metastasis inhibition, raising the possibility that such amino acid substitutions produce structural changes responsible for altering binding to the laminin receptor. In this study, conformational  analysis has been used to determine the three-dimensional structures of these peptides. The results indicate that the substitution of Glu for the terminal Arg produces a significant conformational change in the peptide backbone at the middle Gly residue. These results have important implications for the design of drugs that may be useful in preventing metastasis formation and tumor spread.</p><h2><a href=https://www.ncbi.nlm.nih.gov/pubmed/25049229>Mapping the putative G protein-coupled receptor (GPCR) docking site on GPCR kinase 2: insights from intact cell phosphorylation and recruitment assays.</a></h2><p>G protein-coupled receptor kinases (GRKs) phosphorylate agonist-occupied receptors initiating the processes of desensitization and β-arrestin-dependent signaling. Interaction of GRKs with activated receptors serves to stimulate their kinase activity. The extreme N-terminal helix (αN), the kinase small lobe, and the active site tether (AST) of the AGC kinase domain have previously been implicated in mediating the allosteric activation. Expanded mutagenesis of the αN and AST allowed us to further assess the role of these two regions in kinase activation and receptor phosphorylation in vitro and in intact cells. We also developed a bioluminescence resonance  transfer-based assay to monitor the recruitment of GRK2 to activated α(2A)-adrenergic receptors (α(2A)ARs) in living cells. The bioluminescence resonance  transfer signal exhibited a biphasic response to norepinephrine concentration, suggesting that GRK2 is recruited to Gβγ and α(2A)AR with EC50 values of 15 nM and 8 μM, respectively. We show that mutations in αN (L4A, V7E, L8E, V11A, S12A, Y13A, and M17A) and AST (G475I, V477D, and I485A) regions impair or potentiate receptor phosphorylation and/or recruitment. We suggest that a surface of GRK2, including Leu(4), Val(7), Leu(8), Val(11), and Ser(12), directly interacts with receptors, whereas residues such as Asp(10), (13), Ala(16), Met(17), Gly(475), Val(477), and (485) are more important for kinase domain closure and activation. Taken together with data on GRK1 and GRK6, our data suggest that all three GRK subfamilies make conserved interactions with G protein-coupled receptors, but there may be unique interactions that influence selectivity.© 2014 by The American Society for Biochemistry and Molecular Biology, Inc.</p><h2><a href=https://www.ncbi.nlm.nih.gov/pubmed/31250878>Postprandial insulin and nutrient concentrations in lipopolysaccharide-challenged growing pigs reared in thermoneutral and high ambient temperatures1.</a></h2><p>The aim of this study was to evaluate the associated effects of ambient temperature and  caused by repeated administration of Escherichia coli lipopolysaccharide (LPS) on insulin, energy, and AA metabolism. Twenty-eight pigs were assigned to one of the two thermal conditions: thermoneutral (24 °C) or high ambient temperature (30 °C). The experimental period lasted 17 d, which was divided into a 7-d period without LPS (days -7 to -1), and a subsequent 10-d LPS period (days 1 to 10) in which pigs were administered 5 repeated injections of LPS at 2-d intervals. Postprandial profiles of plasma insulin and nutrients were evaluated through serial blood samples taken on days -4 (P0), 4 (P1), and 8 (P2). Before the LPS-challenge (P0), postprandial concentrations of glucose, lactate, Gln, , Leu, Phe, , and Val were greater in pigs kept at 24 °C than at 30 °C (P < 0.05). In contrast, Arg, Asp, Gly, His, and Met postprandial concentrations at P0 were lower at 24 °C than at 30 °C (P < 0.05). At both 24 and 30 °C conditions, pigs had greater postprandial concentrations of insulin (P < 0.01) and lower concentrations of NEFA (P < 0.01) and α-amino nitrogen (P < 0.05) at P1 and P2 than at P0. Compared with P0, postprandial concentrations of glucose were greater (P < 0.05) at P1 in pigs kept at 24 °C, and at P1 and P2 in pigs kept at 30 °C. At both ambient temperatures, pigs had lower (P < 0.05) postprandial concentrations of Ala, Gly, His, , Leu, Pro, Ser, Thr, Trp, and Val at P1 and P2 than at P0. Arginine postprandial concentration at P1 was lower than at P0 in pigs kept at 24 °C (P < 0.05), whereas no difference was observed in pigs at 30 °C. Relative to P0, Gln and  concentrations were lower at P1 and P2 in pigs kept at 24 °C (P < 0.01), whereas lower Gln concentration was observed only at P2 (P < 0.01) and lower  only at P1 (P < 0.01) in pigs kept at 30 °C. Our study shows a hyperglycemic and hyperinsulinemic state in LPS-challenged pigs and a greater magnitude of this response in pigs kept at 30 °C. Furthermore, LPS caused important changes in BCAA, His, Thr, and Trp profiles, suggesting the role these AA in supporting the inflammatory response. Finally, our results suggest that LPS-induced effects on postprandial profiles of specific AA (Arg, Gln, Phe, and ) may be modulated by ambient temperature.© The Author(s) 2019. Published by Oxford University Press on behalf of the American Society of Animal Science. All rights reserved. For permissions, please e-mail: journals.permissions@oup.com.</p><h2><a href=https://www.ncbi.nlm.nih.gov/pubmed/2599773>Correlation of beta-bend conformations of tetrapeptides with their activities in CD4-receptor binding assays.</a></h2><p>Conformational analysis, based on ECEPP (Empirical Conformational  Program for Peptides) using the chain build-up procedure, was applied to determine the low- conformations for a series of tetrapeptides. The tetrapeptides are components of larger peptides which have been found to bind to the CD4 receptor of monocytes. Several previous studies have implicated the tetrapeptide units investigated here as being critical to the biological activities of the full peptides. Five such tetrapeptides were studied: Ser-Ser-Asn- (from ribonuclease A), Thr-Thr-Asn- (from peptide T, known to block human immunodeficiency virus from attaching to CD4+ T cells), Thr--Asn- (from polio virus coat protein, which is less active than the other peptides in binding to CD4 receptors), Ser-Ser-Ala- (from the gp 120 coat protein of human immunodeficiency virus, a variant of the peptide T sequence, active in blocking viral attachment to CD4+ cells), and the tetrapeptide from an active synthetic pentapeptide, Asn-Thr-Lys- (from Asn-Thr-Lys--Thr). Using a 7 kcal/mol cutoff, the low- conformations for each peptide were computed. Approximately 20,000 conformations were computed for each tetrapeptide. Residue probability profiles were determined for each tetrapeptide. All tetrapeptides except for the polio sequence showed flexibility in the sense that many low- conformations were possible. In previous studies, it was postulated that the critical tetrapeptide units would adopt conformations similar to the one observed in a segment of ribonuclease A, residues 22-25, a beta-bend, which is part of an octapeptide segment (residues 19-26) that is homologous to the sequence of peptide T.(ABSTRACT TRUNCATED AT 250 WORDS).</p><h2><a href=https://www.ncbi.nlm.nih.gov/pubmed/15944418>Effects of mutation at methionine-42 of Escherichia coli dihydrofolate reductase on stability and function: implication of hydrophobic interactions.</a></h2><p>Methionine-42, distal to the active site of Escherichia coli dihydrofolate reductase, was substituted by site-directed mutagenesis with 14 amino acids (Ala, Cys, Glu, Gln, Gly, His, , Leu, Pro, Ser, Thr, Trp, , and Val) to elucidate its role in the stability and function of this enzyme. Far-ultraviolet circular dichroism spectra of these mutants showed a distinctive negative peak at around 230 nm beside 220 nm, depending on the hydrophobicity of the amino acids introduced. The fluorescence intensity also increased in an order similar to that of the amino acids. These spectroscopic data suggest that the mutations do not affect the secondary structure, but strongly perturb the exciton coupling between Trp47 and Trp74. The free  of urea unfolding, deltaG(o)u, increased with increases in the side-chain hydrophobicity in the range 2.96-6.40 kcal x mol(-1), which includes the value for the wild-type enzyme (6.08 kcal x mol(-1)). The steady-state kinetic parameters, Km and kcat, also increased with increases in the side-chain hydrophobicity, with the M42W mutant showing the largest increases in Km (35-fold) and kcat (4.3-fold) compared with the wild-type enzyme. These results demonstrate that site 42 distal to the active site plays an important role in the stability and function of this enzyme, and that the main effect of the mutations is to modify of hydrophobic interactions with the residues surrounding this position.</p><h2><a href=https://www.ncbi.nlm.nih.gov/pubmed/28798365>Mass & secondary structure propensity of amino acids explain their mutability and evolutionary replacements.</a></h2><p>Why is an amino acid replacement in a protein accepted during evolution? The answer given by bioinformatics relies on the frequency of change of each amino acid by another one and the propensity of each to remain unchanged. We propose that these replacement rules are recoverable from the secondary structural trends of amino acids. A distance measure between high-resolution Ramachandran distributions reveals that structurally similar residues coincide with those found in substitution matrices such as BLOSUM: Asn ↔ Asp, P™he ↔ , Lys ↔ Arg, Gln ↔ Glu,  ↔ Val, Met → Leu; with Ala, Cys, His, Gly, Ser, Pro, and Thr, as structurally idiosyncratic residues. We also found a high average correlation ([Formula: see text] = 0.85) between thirty amino acid mutability scales and the mutational inertia (I  ), which measures the energetic cost weighted by the number of observations at the most probable amino acid conformation. These results indicate that amino acid substitutions follow two optimally-efficient principles: (a) amino acids interchangeability privileges their secondary structural similarity, and (b) the amino acid mutability depends directly on its biosynthetic  cost, and inversely with its frequency. These two principles are the underlying rules governing the observed amino acid substitutions.</p><h2><a href=https://www.ncbi.nlm.nih.gov/pubmed/10415170>Thermococcus waiotapuensis sp. nov., an extremely thermophilic archaeon isolated from a freshwater hot spring.</a></h2><p>An extremely thermophilic, sulfur-dependent archaeon, strain WT1, was isolated from a freshwater hot spring in the Lake Taupo area of North Island, New Zealand. The cells are flagellated, strictly anaerobic cocci that grow optimally at 85 degrees C and 5.4 g NaCl l(-1). The strain grows heterotrophically on complex proteinaceous substrates or on appropriate salts plus amino acid mixtures and is also able to utilize maltose, starch, and pyruvate. Elemental sulfur could be replaced by cystine or thioglycollate. The range of temperatures allowing growth is from 60 to 90 degrees C; the pH supporting growth ranges from 5 to 8 (optimum, pH 7). Strain WT1 grew in a defined medium containing amino acids as the sole carbon and  sources. The required amino acids were: Arg, His, , Leu, Phe, Ser, Thr, Trp, , and Val. Strain WT1 showed sensitivity to rifampicin. DNA G+C content was 50.4 mol%. Phylogenetic analysis of the sequence encoding the 16S rRNA gene indicated that this isolate is a member of the Thermococcales. DNA/DNA hybridization studies revealed no similarity to several species of Thermococcus and Pyrococcus, with the exception of Thermococcus zilligii. Based on the reported results, we propose strain WT1 as a new species to be named Thermococcus waiotapuensis sp. nov.</p><h2><a href=https://www.ncbi.nlm.nih.gov/pubmed/2674124>Structural studies of mutants of T4 lysozyme that alter hydrophobic stabilization.</a></h2><p>Multiple replacements at amino acid position 3 of bacteriophage T4 lysozyme have shown that the conformational stability of the protein is directly governed by the hydrophobicity of the residue substituted (Matsumura, M., Becktel, W. J., and Matthews, B. W. (1988) Nature 334, 406-410). Of the 13 mutant lysozymes made by site-directed mutagenesis, two variants, one with valine (I3V) and the other with tyrosine (I3Y), were crystallized and their structures solved. In this report we describe the crystal structures of these variants at 1.7 A resolution. While the structure of the I3V mutant is essentially the same as that of wild-type lysozyme, the I3Y mutant has substantial changes in its structure. The most significant of these are that the side chain of the tyrosine is not accommodated within the interior of the protein and the amino-terminal polypeptide (residues 1-9) moves 0.6-1.1 A relative to the wild-type structure. Using coordinates based on the wild-type and available mutant structures, solvent accessible surface area of residue 3 as well as the adjacent 9 residues in the folded form were calculated. The free  of stabilization based on the transfer of these residues from a fully extended form to the interior to the folded protein was found to correlate well with the protein stability determined by thermodynamic analysis. The enhanced thermostability of the variant -3----Leu, relative to wild-type lysozyme, can also be rationalized by surface-area calculations based on a model-built structure. Noncrystallization of most lysozyme variants at position 3 appears to be due to disruption of intermolecular contacts in the crystal. The -3----Val variant is closely isomorphous with wild-type and maintains the same crystal contacts. In the -3---- variant, however, a new set of contacts is made in which direct protein-protein hydrogen bonds are replaced by protein-water-protein hydrogen bonds as well as a novel hydrogen bond involving the phenolic hydroxyl of the substituted tyrosine.</p></html>