<html><h1><a href=https://www.ncbi.nlm.nih.gov/pubmed/29549099>Metabolism of Oxo-Bile  and Characterization of Recombinant 12α-Hydroxysteroid Dehydrogenases from Bile  7α-Dehydroxylating Human Gut Bacteria.</a></h1><p>Bile  are important cholesterol-derived nutrient signaling hormones, synthesized in the liver, that act as detergents to solubilize dietary lipids. Bile  7α-dehydroxylating gut bacteria generate the toxic bile  deoxycholic  and lithocholic  from host bile . The ability of these bacteria to remove the 7-hydroxyl group is partially dependent on 7α-hydroxysteroid dehydrogenase (HSDH) activity, which reduces 7-oxo-bile  generated by other gut bacteria. 3α-HSDH has an important enzymatic activity in the bile  7α-dehydroxylation pathway. 12α-HSDH activity has been reported for the low-activity bile  7α-dehydroxylating bacterium ; however, this activity has not been reported for high-activity bile  7α-dehydroxylating bacteria, such as , , and  Here, we demonstrate that these strains express bile  12α-HSDH. The recombinant enzymes were characterized from each species and shown to preferentially reduce 12-oxolithocholic  to deoxycholic , with low activity against 12-oxochenodeoxycholic  and reduced activity when bile  were conjugated to taurine or glycine. Phylogenetic analysis suggests that 12α-HSDH is widespread among ,  in the  family, and human gut  12α-HSDH activity has been established in the medically important bile  7α-dehydroxylating bacteria , , and  Experiments with recombinant 12α-HSDHs from these strains are consistent with culture-based experiments that show a robust preference for 12-oxolithocholic  over 12-oxochenodeoxycholic . Phylogenetic analysis identified novel members of the gut microbiome encoding 12α-HSDH. Future reengineering of 12α-HSDH enzymes to preferentially oxidize cholic  may provide a means to industrially produce the therapeutic bile  ursodeoxycholic . In addition, a cholic -specific 12α-HSDH expressed in the gut may be useful for the reduction in deoxycholic  concentration, a bile  implicated in cancers of the gastrointestinal (GI) tract.Copyright © 2018 American Society for Microbiology.</p><h1><a href=https://www.ncbi.nlm.nih.gov/pubmed/31594827>Distinct Gut Microbiota Induced by Different Fat-to-Sugar-Ratio High-Energy Diets Share Similar Pro- Genetic and Metabolite Profiles in Prediabetic Mice.</a></h1><p>Gut microbiota play important roles in host metabolism, especially in diabetes. However, why different diets lead to similar diabetic states despite being associated with different microbiota is not clear. Mice were fed two high-energy diets (HED) with the same energy density but different fat-to-sugar ratios to determine the associations between the microbiota and early-stage metabolic syndrome. The two diets resulted in different microbiota but similar diabetic states. Interestingly, the microbial gene profiles were not significantly different, and many common metabolites were identified, including l-aspartic , cholestan-3-ol (5β, 3α), and campesterol, which have been associated with lipogenesis and inflammation. Our study suggests that different metabolic-syndrome-inducing diets may result in different microbiota but similar microbiomes and metabolomes. This suggests that the metagenome and metabolome are crucial for the prognosis and pathogenesis of  and metabolic syndrome. Various types of diet can lead to type 2 diabetes. The gut microbiota in type 2 diabetic patients are also different. So, two questions arise: whether there are any commonalities between gut microbiota induced by different pro-obese diets and whether these commonalities lead to disease. Here we found that high-energy diets with two different fat-to-sugar ratios can both cause  and prediabetes but enrich different gut microbiota. Still, these different gut microbiota have similar genetic and metabolite compositions. The microbial metabolites in common between the diets modulate lipid accumulation and macrophage inflammation  and  This work suggests that studies that only use 16S rRNA amplicon sequencing to determine how the microbes respond to diet and associate with diabetic state are missing vital information.Copyright © 2019 Shan et al.</p><h1><a href=https://www.ncbi.nlm.nih.gov/pubmed/29330189>Targeted Synthesis and Characterization of a Gene Cluster Encoding NAD(P)H-Dependent 3α-, 3β-, and 12α-Hydroxysteroid Dehydrogenases from Eggerthella CAG:298, a Gut Metagenomic Sequence.</a></h1><p>Gut metagenomic sequences provide a rich source of microbial genes, the majority of which are annotated by homology or unknown. Genes and gene pathways that encode enzymes catalyzing biotransformation of host bile  are important to identify in gut metagenomic sequences due to the importance of bile  in gut microbiome structure and host physiology. Hydroxysteroid dehydrogenases (HSDHs) are pyridine nucleotide-dependent enzymes with stereospecificity and regiospecificity for bile  and steroid hydroxyl groups. HSDHs have been identified in several protein families, including medium-chain and short-chain dehydrogenase/reductase families as well as the aldo-keto reductase family. These protein families are large and contain diverse functionalities, making prediction of HSDH-encoding genes difficult and necessitating biochemical characterization. We located a gene cluster in  sp. CAG:298 predicted to encode three HSDHs (CDD59473, CDD59474, and CDD59475) and synthesized the genes for heterologous expression in  We then screened bile  substrates against the purified recombinant enzymes. CDD59475 is a novel 12α-HSDH, and we determined that CDD59474 (3α-HSDH) and CDD59473 (3β-HSDH) constitute novel enzymes in an iso-bile  pathway. Phylogenetic analysis of these HSDHs with other gut bacterial HSDHs and closest homologues in the database revealed predictable clustering of HSDHs by function and identified several likely HSDH sequences from bacteria isolated or sequenced from diverse mammalian and avian gut samples. Bacterial HSDHs have the potential to significantly alter the physicochemical properties of bile , with implications for increased/decreased toxicity for gut bacteria and the host. The generation of oxo-bile  is known to inhibit host enzymes involved in glucocorticoid metabolism and may alter signaling through nuclear receptors such as farnesoid X receptor and G-protein-coupled receptor TGR5. Biochemical or similar approaches are required to fill in many gaps in our ability to link a particular enzymatic function with a nucleic  or amino  sequence. In this regard, we have identified a novel 12α-HSDH and a novel set of genes encoding an iso-bile  pathway (3α-HSDH and 3β-HSDH) involved in epimerization and detoxification of harmful secondary bile .Copyright © 2018 American Society for Microbiology.</p><h1><a href=https://www.ncbi.nlm.nih.gov/pubmed/29907951>Analysis of human C24 bile  metabolome in serum and urine based on enzyme digestion of conjugated bile  and LC-MS determination of unconjugated bile .</a></h1><p>Host-gut microbiota metabolic interactions are closely associated with health and disease. A manifestation of such co-metabolism is the vast structural diversity of bile  (BAs) involving both oxidative stereochemistry and conjugation. Herein, we describe the development and validation of a LC-MS-based method for the analysis of human C24 BA metabolome in serum and urine. The method has high throughput covering the discrimination of oxidative stereochemistry of unconjugated species in a 15-min analytical cycle. The validated quantitative performance provided an indirect way to ascertain the conjugation patterns of BAs via enzyme-digestion protocols that incorporated the enzymes, sulfatase, β-glucuronidase, and choloylglycine hydrolase. Application of the method has led to the detection of at least 70 unconjugated BAs including 27 known species and 43 newly found species in the post-prandial serum and urine samples from 7 nonalcoholic steatohepatitis patients and 13 healthy volunteers. Newly identified unconjugated BAs included 3α, 12β-dihydroxy-5β-cholan-24-oic , 12α-hydroxy-3-oxo-5β-cholan-24-oic , and 3α, 7α, 12β-trihydroxy-5β-cholan-24-oic . High-definition negative fragment spectra of the other major unknown species were acquired to facilitate future identification endeavors. An extensive conjugation pattern is the major reason for the "invisibility" of the newly found BAs to other common analytical methods. Metabolomic analysis of the total unconjugated BA profile in combination with analysis of their conjugation patterns and urinary excretion tendencies have provided substantial insights into the interconnected roles of host and gut microbiota in maintaining BA homeostasis. It was proposed that the urinary total BA profile may serve as an ideal footprint for the functional status of the host-gut microbial BA co-metabolism. In summary, this work provided a powerful tool for human C24 BA metabolome analysis that bridges the gap between GC-MS techniques in the past age and LC-MS techniques currently prevailing in biomedical researches. Further applications of the present method in clinical, translational research, and other biomedical explorations will continue to boost the construction of a host-gut microbial co-metabolism network of BAs and thus facilitate the decryption of BA-mediated host-gut microbiota crosstalk in health and diseases. Graphical abstract ᅟ.</p><h1><a href=https://www.ncbi.nlm.nih.gov/pubmed/29617190>Bile  oxidation by Eggerthella lenta strains C592 and DSM 2243.</a></h1><p>Strains of Eggerthella lenta are capable of oxidation-reduction reactions capable of oxidizing and epimerizing bile  hydroxyl groups. Several genes encoding these enzymes, known as hydroxysteroid dehydrogenases (HSDH) have yet to be identified. It is also uncertain whether the products of E. lenta bile  metabolism are further metabolized by other members of the gut microbiota. We characterized a novel human fecal isolate identified as E. lenta strain C592. The complete genome of E. lenta strain C592 was sequenced and comparative genomics with the type strain (DSM 2243) revealed high conservation, but some notable differences. E. lenta strain C592 falls into group III, possessing 3α, 3β, 7α, and 12α-hydroxysteroid dehydrogenase (HSDH) activity, as determined by mass spectrometry of thin layer chromatography (TLC) separated metabolites of primary and secondary bile . Incubation of E. lenta oxo-bile  and iso-bile  metabolites with whole-cells of the high-activity bile  7α-dehydroxylating bacterium, Clostridium scindens VPI 12708, resulted in minimal conversion of oxo-derivatives to lithocholic  (LCA). Further, Iso-chenodeoxycholic  (iso-CDCA; 3β,7α-dihydroxy-5β-cholan-24-oic ) was not metabolized by C. scindens. We then located a gene encoding a novel 12α-HSDH in E. lenta DSM 2243, also encoded by strain C592, and the recombinant purified enzyme was characterized and substrate-specificity determined. Genomic analysis revealed genes encoding an Rnf complex (rnfABCDEG), an energy conserving hydrogenase (echABCDEF) complex, as well as what appears to be a complete Wood-Ljungdahl pathway. Our prediction that by changing the gas atmosphere from nitrogen to hydrogen, bile  oxidation would be inhibited, was confirmed. These results suggest that E. lenta is an important bile  metabolizing gut microbe and that the gas atmosphere may be an important and overlooked regulator of bile  metabolism in the gut.</p></html>