<html><h1><a href=https://www.ncbi.nlm.nih.gov/pubmed/28222161>The distinct features of microbial 'dysbiosis' of Crohn's disease do not occur to the same extent in their unaffected, genetically-linked kindred.</a></h1><p>Studying the gut microbiota in unaffected relatives of people with Crohn's disease (CD) may advance our understanding of the role of bacteria in disease aetiology.Faecal microbiota composition (16S rRNA gene sequencing), genetic functional capacity (shotgun metagenomics) and faecal short chain fatty  (SCFA) were compared in unaffected adult relatives of CD children (CDR, n = 17) and adult healthy controls, unrelated to CD patients (HUC, n = 14). The microbiota characteristics of 19 CD children were used as a benchmark of CD 'dysbiosis'.The CDR microbiota was less diverse (p = 0.044) than that of the HUC group. Local contribution of β-diversity analysis showed no difference in community structure between the CDR and HUC groups. Twenty one of 1,243 (1.8%) operational taxonomic units discriminated CDR from HUC. The metagenomic functional capacity (p = 0.207) and SCFA concentration or pattern were similar between CDR and HUC (p>0.05 for all SCFA). None of the KEGG metabolic pathways were different between these two groups. Both of these groups (HUC and CDR) had a higher microbiota α-diversity (CDR, p = 0.026 and HUC, p<0.001) with a community structure (β-diversity) distinct from that of children with CD.While some alterations were observed, a distinct microbial 'dysbiosis', characteristic of CD patients, was not observed in their unaffected, genetically linked kindred.</p><h1><a href=https://www.ncbi.nlm.nih.gov/pubmed/29844325>Combining amplicon sequencing and metabolomics in cirrhotic patients highlights distinctive microbiota features involved in bacterial translocation, systemic inflammation and hepatic encephalopathy.</a></h1><p>In liver cirrhosis (LC), impaired intestinal functions lead to dysbiosis and possible bacterial translocation (BT). Bacteria or their byproducts within the bloodstream can thus play a role in systemic inflammation and hepatic encephalopathy (HE). We combined 16S sequencing, NMR metabolomics and network analysis to describe the interrelationships of members of the microbiota in LC biopsies, faeces, peripheral/portal blood and faecal metabolites with clinical parameters. LC faeces and biopsies showed marked dysbiosis with a heightened proportion of Enterobacteriaceae. Our approach showed impaired faecal bacterial metabolism of short-chain fatty  (SCFAs) and carbon/methane sources in LC, along with an enhanced stress-related response. Sixteen species, mainly belonging to the Proteobacteria phylum, were shared between LC peripheral and portal blood and were functionally linked to iron metabolism. Faecal Enterobacteriaceae and trimethylamine were positively correlated with blood proinflammatory cytokines, while Ruminococcaceae and SCFAs played a protective role. Within the peripheral blood and faeces, certain species (Stenotrophomonas pavanii, Methylobacterium extorquens) and metabolites (methanol, threonine) were positively related to HE. Cirrhotic patients thus harbour a 'functional dysbiosis' in the faeces and peripheral/portal blood, with specific keystone species and metabolites related to clinical markers of systemic inflammation and HE.</p><h1><a href=https://www.ncbi.nlm.nih.gov/pubmed/27717873>Microbial signatures of oral dysbiosis, periodontitis and edentulism revealed by Gene Meter methodology.</a></h1><p>Conceptual models suggest that certain microorganisms (e.g., the "red" complex) are indicative of a specific disease state (e.g., periodontitis); however, recent studies have questioned the validity of these models. Here, the abundances of 500+ microbial species were determined in 16 patients with clinical signs of one of the following oral conditions: periodontitis, established caries, edentulism, and oral health. Our goal was to determine if the abundances of certain microorganisms reflect dysbiosis or a specific clinical condition that could be used as a 'signature' for dental research. Microbial abundances were determined by the analysis of 138,718 calibrated probes using Gene Meter methodology. Each 16S rRNA gene was targeted by an average of 194 unique probes (n=25nt). The calibration involved diluting pooled gene target samples, hybridizing each dilution to a DNA microarray, and fitting the probe intensities to adsorption models. The fit of the model to the experimental data was used to assess individual and aggregate probe behavior; good fits (R>0.90) were retained for back-calculating microbial abundances from patient samples. The abundance of a gene was determined from the median of all calibrated individual probes or from the calibrated abundance of all aggregated probes. With the exception of genes with low abundances (<2 arbitrary units), the abundances determined by the different calibrations were highly correlated (r~1.0). Seventeen genera were classified as 'signatures of dysbiosis' because they had significantly higher abundances in patients with periodontitis and edentulism when contrasted with health. Similarly, 13 genera were classified as 'signatures of periodontitis', and 14 genera were classified as 'signatures of edentulism'. The signatures could be used, individually or in combination, to assess the clinical status of a patient (e.g., evaluating treatments such as antibiotic therapies). Comparisons of the same patient samples revealed high false negatives (45%) for next-generation-sequencing results and low false positives (7%) for Gene Meter results.Copyright Â© 2016 Elsevier B.V. All rights reserved.</p></html>