<html><h2><a href=https://www.ncbi.nlm.nih.gov/pubmed/16249103>Deamidation of peptides in aerobic nitric oxide solution by a nitrosative pathway.</a></h2><p>Hydrolytic deamidation of asparagine () and glutamine (Gln) residues to aspartate (Asp) and glutamate (), respectively, can have significant biological consequences. We hypothesize that a wholly different mechanism of deamidation might occur in the presence of aerobic nitric oxide (NO). Accordingly, we examined the deamidating ability of aerobic NO toward three model peptides, 2,4-dinitrophenyl (DNP)-Pro-Gln-Gly, -Trp-Asp--Gln, and Ser----Pro-Ile-Val, incubating them with the NO-generating compound, Et(2)N[N(O)NO]Na (DEA/NO, 30-48 mM), in aerobic, pH 7.4, buffer at 37 degrees C. DNP-Pro--Gly was detected after 2 h, while -Trp-Asp-Asp-Gln, -Trp-Asp--, and Ser--Asp--Pro-Ile-Val were detected within 10 min, accumulating in apparent yields of up to approximately 10%. In the latter case, tyrosine nitration was also observed, producing the expected nitrotyrosine residue. DEA/NO solutions preincubated to exhaust the NO before the peptides were added did not induce detectable deamidation. The data demonstrate that aerobic NO exposures can lead to nitrosative deamidation of peptides, a pathway that differs from the established hydrolytic deamidation mechanism in several key respects: the by-products of the former are molecular nitrogen and an acid (HONO) while that of the latter is a base (NH(3)); the nitrosative path can in principle proceed in the absence of water molecules;  is much more easily deamidated than Gln in the hydrolytic pathway, while the two amino acid residues were deamidated to a similar extent by exposure to NO in the presence of .</p><p>Keyword: oxygen</p><h2><a href=https://www.ncbi.nlm.nih.gov/pubmed/23923535>Crystal structure of a novel phospholipase A2 from crude venom of Indian cobra sub-species Naja naja sagittifera at 1.48 angstoms resolution.</a></h2><p>Secretory phospholipase A2s (PLA2s), the structurally-homologous enzymes share a common qualitative catalytic site, but differ greatly in their pharmacological properties and toxicities. There has been a recognizable pattern of mutations in the primary sequence of PLA2s that alter their catalytic properties significantly. In the present study, the amino acid sequence and the three-dimensional structure of a new isoform of PLA2 from crude venom of Indian cobra sub-species Naja naja sagittifera (N.n.s.) has been determined by X-ray crystallography. The crystal structure has revealed several novel features of PLA2 folding and function. It contains 913 protein atoms and one each of Ca2+, phosphate and acetate ions with 142 solvent water molecules. A Ca2+ ion is present in the calcium-binding loop and forms a seven-fold coordination with a distorted pentagonal bipyramidal geometry. One of the coordination linkages is with the acetate ion, instead of the conserved water molecule. The presence of  at position 31 has a stabilizing effect on the loop  25-Cys 29 by interacting with carbonyl  atoms of  25, Gly 26 and Cys 29. In turn, it lends stability to the Ca(2+)-binding loop as well. Another unique feature of the PLA2 structure is the formation of an intrastrand hydrogen bond, involving Ogamma of Thr 73 and Oepsilon2 of  71, thus helping the beta-wing to act as a sharp arrow for insertion into other molecules. Yet another important feature of this PLA2 pertains to the conformation of its C-terminal segment, which is stabilized by a unique hydrogen bond through carbonyl  of  116 and Ndelta2 of  120. This structural feature may be useful in the molecular recognition of the PLA2 through C-terminal segment.</p><p>Keyword: oxygen</p><h2><a href=https://www.ncbi.nlm.nih.gov/pubmed/18178652>Crystallographic study of hydration of an internal cavity in engineered proteins with buried polar or ionizable groups.</a></h2><p>Although internal water molecules are essential for the structure and function of many proteins, the structural and physical factors that govern internal hydration are poorly understood. We have examined the molecular determinants of internal hydration systematically, by solving the crystal structures of variants of staphylococcal nuclease with Gln-66, -66, and -66 at cryo (100 K) and room (298 K) temperatures, and comparing them with existing cryo and room temperature structures of variants with -66, Asp-66, -66, -92 or -92 obtained under conditions of pH where the internal ionizable groups are in the neutral state. At cryogenic temperatures the polar moieties of all these internal side chains are hydrated except in the cases of -66 and -92. At room temperature the internal water molecules were observed only in variants with -66 and -66; water molecules in the other variants are probably present but they are disordered and therefore undetectable crystallographically. Each internal water molecule establishes between 3 and 5 hydrogen bonds with the protein or with other internal water molecules. The strength of interactions between internal polar side chains and water molecules seems to decrease from carboxylic acids to amides to amines. Low temperature, low cavity volume, and the presence of  atoms in the cavity increase the positional stability of internal water molecules. This set of structures and the physical insight they contribute into internal hydration will be useful for the development and benchmarking of computational methods for artificial hydration of pockets, cavities, and active sites in proteins.</p><p>Keyword: oxygen</p><h2><a href=https://www.ncbi.nlm.nih.gov/pubmed/12617685>First-second shell interactions in metal binding sites in proteins: a PDB survey and DFT/CDM calculations.</a></h2><p>The role of the second shell in the process of metal binding and selectivity in metalloproteins has been elucidated by combining Protein Data Bank (PDB) surveys of Mg, Mn, Ca, and Zn binding sites with density functional theory/continuum dielectric methods (DFT/CDM). Peptide backbone groups were found to be the most common second-shell ligand in Mg, Mn, Ca, and Zn binding sites, followed (in decreasing order) by Asp/, /Arg, /Gln, and Ser/Thr side chains. Aromatic - or nitrogen-containing side chains (, His, and Trp) and sulfur-containing side chains (Cys and Met) are seldom found in the second coordination layer. The backbone and /Gln side chain are ubiquitous in the metal second coordination layer as their carbonyl  and amide hydrogen can act as a hydrogen-bond acceptor and donor, respectively, and can therefore partner practically every first-shell ligand. The second most common outer-shell ligand, Asp/, predominantly hydrogen bonds to a metal-bound water or Zn-bound histidine and polarizes the H-O or H-N bond. In certain cases, a second-shell Asp/ could affect the protonation state of the metal ligand. It could also energetically stabilize a positively charged metal complex more than a neutral ligand such as the backbone and /Gln side chain. As for the first shell, the second shell is predicted to contribute to the metal selectivity of the binding site by discriminating between metal cations of different ionic radii and coordination geometries. The first-shell-second-shell interaction energies decay rapidly with increasing solvent exposure of the metal binding site. They are less favorable but are of the same order of magnitude as compared to the respective metal-first-shell interaction energies. Altogether, the results indicate that the structure and properties of the second shell are dictated by those of the first layer. The outer shell is apparently designed to stabilize/protect the inner-shell and complement/enhance its properties.</p><p>Keyword: oxygen</p><h2><a href=https://www.ncbi.nlm.nih.gov/pubmed/7583766>Interaction of 2 amino acid substitutions within the same beta chain of human hemoglobin: the examples of Hb Corbeil and Hb Villeparisis.</a></h2><p>Only 16 hemoglobin (Hb) variants carrying 2 point mutations within the same polypeptide chain have been identified up to now. Two of those are reported in this paper. In Hb Villeparisis, the beta 77 (EF1) His--> mutation is identical to that of Hb Fukuyama while the other mutation, beta 80 (EF4) -->Ser, has not yet been described. These 2 abnormalities are located almost within interacting distance. They lead to decreased oxygen affinity and, according to molecular simulations, to a remodeling of the surface of the molecule. Hb Corbeil associates 2 substitutions, that of Hb E [beta 26 (B8) -->], and of Hb Sherwood Forest [beta 104 (G6) Arg-->Thr], which are far from one another. The resulting properties consist only in the additive effects of both abnormalities. Hb Corbeil is, thus, a thalassemic Hb, like Hb E, and a molecule having altered oxygen binding properties, characterized by decreased heterotropic effects, similar to those of Hb Sherwood Forest. The simultaneous presence of more than 1 structural abnormality leads to a large spectrum of possible consequences on the function of the molecule, from local effects to a perturbation of intermolecular interactions. The genetic mechanism resulting in Hb with more than 1 point mutation is discussed: in some cases it is very likely that the second mutation arose on a chromosome already carrying a point mutation, in contrast a single mutational event involving a micro gene conversion has been proposed in other cases.</p><p>Keyword: oxygen</p><h2><a href=https://www.ncbi.nlm.nih.gov/pubmed/11279165>Crystal structure of activated CheY. Comparison with other activated receiver domains.</a></h2><p>The crystal structure of BeF(3)(-)-activated CheY, with manganese in the magnesium binding site, was determined at 2.4-A resolution. BeF(3)(-) bonds to Asp(57), the normal site of phosphorylation, forming a hydrogen bond and salt bridge with Thr(87) and (109), respectively. The six coordination sites for manganese are satisfied by a fluorine of BeF(3)(-), the side chain oxygens of Asp(13) and Asp(57), the carbonyl  of (59), and two water molecules. All of the active site interactions seen for BeF(3)(-)-CheY are also observed in P-Spo0A(r). Thus, BeF(3)(-) activates CheY as well as other receiver domains by mimicking both the tetrahedral geometry and electrostatic potential of a phosphoryl group. The aromatic ring of (106) is found buried within a hydrophobic pocket formed by beta-strand beta4 and helix H4. The tyrosine side chain is stabilized in this conformation by a hydrogen bond between the hydroxyl group and the backbone carbonyl  of (89). This hydrogen bond appears to stabilize the active conformation of the beta4/H4 loop. Comparison of the backbone coordinates for the active and inactive states of CheY reveals that only modest changes occur upon activation, except in the loops, with the largest changes occurring in the beta4/H4 loop. This region is known to be conformationally flexible in inactive CheY and is part of the surface used by activated CheY for binding its target, FliM. The pattern of activation-induced backbone coordinate changes is similar to that seen in FixJ(r). A common feature in the active sites of BeF(3)(-)-CheY, P-Spo0A(r), P-FixJ(r), and phosphono-CheY is a salt bridge between (109) Nzeta and the phosphate or its equivalent, beryllofluoride. This suggests that, in addition to the concerted movements of Thr(87) and (106) (Thr- coupling), formation of the (109)-PO(3)(-) salt bridge is directly involved in the activation of receiver domains generally.</p><p>Keyword: oxygen</p><h2><a href=https://www.ncbi.nlm.nih.gov/pubmed/11500966>Statistical analysis of atomic contacts at RNA-protein interfaces.</a></h2><p>Forty-five crystals of complexes between proteins and RNA molecules from the Protein Data Bank have been statistically surveyed for the number of contacts between RNA components (phosphate, ribose and the four bases) and amino acid side chains. Three groups of complexes were defined: the tRNA synthetases; the ribosomal complexes; and a third group containing a variety of complexes. The types of atomic contacts were a priori classified into ionic, neutral H-bond, C-H...O H-bond, or van der Waals interaction. All the contacts were organized into a relational database which allows for statistical analysis. The main conclusions are the following: (i) in all three groups of complexes, the most preferred amino acids (Arg, , Ser, ) and the less preferred ones (Ala, Ile, Leu, Val) are the same; Trp and Cys are rarely observed (respectively 15 and 5 amino acids in the ensemble of interfaces); (ii) of the total number of amino acids located at the interfaces 22% are hydrophobic, 40% charged (positive 32%, negative 8%), 30% polar and 8% are Gly; (iii) in ribosomal complexes, phosphate is preferred over ribose, which is preferred over the bases, but there is no significant preference in the other two groups; (iv) there is no significant prevalence of a base type at protein-RNA interfaces, but specifically Arg and  display a preference for phosphate over ribose and bases; Pro and  prefer bases over ribose and phosphate; Met, Phe and  prefer ribose over phosphate and bases. Further, Ile, Pro, Ser prefer A over the others; Leu prefers C; Asp and Gly prefer G; and  prefers U. Considering the contact types, the following conclusions could be drawn: (i) 23% of the contacts are via potential H-bonds (including CH...O H-bonds and ionic interactions), 72% belong to van der Waals interactions and 5% are considered as short contacts; (ii) of all potential H-bonds, 54% are standard, 33% are of the C-H...O type and 13% are ionic; (iii) the Watson-Crick sites of G, O6(G) and principally N2(G) and the hydroxyl group O2' is more often involved in H-bonds than expected; the protein main chain is involved in 32% and the side chains in 68% of the H-bonds; considering the neutral and ionic H-bonds, the following couples are more frequent than expected-base A-Ser, base G-Asp/, base U-. The RNA CH groups interact preferentially with  atoms (62% on the main chain and 19% on the side chains); (iv) the bases are involved in 38% of all H-bonds and more than 26% of the H-bonds have the H donor group on the RNA; (v) the atom O2' is involved in 21% of all H-bonds, a number greater than expected; (vi) amino acids less frequently in direct contact with RNA components interact frequently via their main chain atoms through water molecules with RNA atoms; in contrast, those frequently observed in direct contact, except Ser, use instead their side chain atoms for water bridging interactions.Copyright 2001 John Wiley & Sons, Ltd.</p><p>Keyword: oxygen</p></html>