<html><h1><a href=https://www.ncbi.nlm.nih.gov/pubmed/11171120>Site-directed mutations in the mitochondrially encoded subunits I and III of yeast cytochrome oxidase.</a></h1><p>Since yeast is amenable to mitochondrial transformation, designed mutations can be introduced in the mitochondrially encoded subunits of the respiratory complexes. In the present work, six mutations have been introduced by the biolistic method into yeast (Saccharomyces cerevisiae) cytochrome oxidase subunits I and III. The effects of these mutations on respiratory growth competence, cytochrome oxidase activity and optical properties were then characterized. Firstly, the conserved glutamate Glu-243 in the D-channel of subunit I was replaced by an asparagine or an aspartate residue. The effects of the mutations showed that Glu-243, which is essential for proton movement in bacterial oxidases, is also required for the activity of the eukaryotic enzyme. Secondly, four mutations associated with human disease were introduced in yeast, allowing detailed analysis of their deleterious effects on cytochrome oxidase function: Met-273-->, Ile-280--> and Gly-317-->Ser, affecting residues located in or near the K-channel in subunit I, and a short in-frame deletion comprising residues -102 to -106 in subunit III (DeltaF102-F106). The subunit III mutation was highly deleterious and abolished enzyme assembly. The change Gly-317-->Ser had no effect on respiratory function. However, mutations Met-273--> and Ile-280--> were mildly deleterious, decreased cytochrome oxidase activity and slightly perturbed the properties of the binuclear centre.</p><h1><a href=https://www.ncbi.nlm.nih.gov/pubmed/17123703>Comparison of codon usage and tRNAs in mitochondrial genomes of Candida species.</a></h1><p>To gain insight into the nature of the mitochondrial genomes (mtDNA) of different Candida species, the synonymous codon usage bias of mitochondrial protein coding genes and the tRNAs in C. albicans, C. parapsilosis, C. stellata, C. glabrata and the closely related yeast Saccharomyces cerevisiae were analyzed. Common features of the mtDNA in Candida species are a strong A+T pressure on protein coding genes, and insufficient mitochondrial tRNA species are encoded to perform protein synthesis. The wobble site of the anticodon is always U for the NNR (NNA and NNG) codon families, which are dominated by A-ending codons, and always G for the NNY (NNC and NNU) codon families, which is dominated by U-ending codons, and always U for the NNN (NNA, NNU, NNC and NNG) codon families, which are dominated by A-ending codons and U-ending codons. Patterns of synonymous codon usage of Candida species can be classified into three groups: (1) optimal codon-anticodon usage, Glu, Lys, Leu (translated by anti-codon UAA), Gln, Arg (translated by anti-codon UCU) and Trp are containing NNR codons. NNA, whose corresponding tRNA is encoded in the mtDNA, is used preferentially. (2) Non-optimal codon-anticodon usage, Cys, Asp, , His, Asn, Ser (translated by anti-codon GCU) and Tyr are containing NNY codons. The NNU codon, whose corresponding tRNA is not encoded in the mtDNA, is used preferentially. (3) Combined codon-anticodon usage, Ala, Gly, Leu (translated by anti-codon UAG), Pro, Ser (translated by anti-codon UGA),  and Val are containing NNN codons. NNA (tRNA encoded in the mtDNA) and NNU (tRNA not encoded in the mtDNA) are used preferentially. In conclusion, we propose that in Candida species, codons containing A or U at third position are used preferentially, regardless of whether corresponding tRNAs are encoded in the mtDNA. These results might be useful in understanding the common features of the mtDNA in Candida species and patterns of synonymous codon usage.</p><h1><a href=https://www.ncbi.nlm.nih.gov/pubmed/19191958>Acrebol, a novel toxic peptaibol produced by an Acremonium exuviarum indoor isolate.</a></h1><p>To identify a toxin and its producer isolated from woody material in a building where the occupants experienced serious ill health symptoms.Hyphal extracts of an indoor fungus, identified as the cycloheximide-tolerant species Acremonium exuviarum, inhibited motility of boar spermatozoa (EC(50) 5 +/- 2 microg of crude solids ml(-1)) and caused cytolysis of murine neuroblastoma cells (MNA) and feline fetal lung cells (FL). The responsible substances were purified and identified as two structurally similar, heat-stable, novel, toxic peptaibols, 1726 Da and 1740 Da, respectively, with amino acid sequences of Acetyl--Iva/Val-Gln-Aib-Ile--Leu-Aib-Pro-Aib-Gln-Pro-Aib-(X-X-X)-SerOH and Acetyl--Iva/Val-Gln-Aib-Ile--Leu-Val-Pro-Aib-Gln-Pro-Aib-(X-X-X)-SerOH. Purified acrebol inhibited motility of boar sperm, depleted ATP half-content in 1 day (EC(50) of 0.1 microg ml(-1), 60 nmol l(-1)) depolarised the  after 2 days, but did not affect the cellular content in NADH. This indicates mitochondrial toxicity. Plate-grown biomass of A. exuviarum BMB4 contained 0.1-1% (w/w) of acrebol, depending on the culture medium.Acrebol paralysed the energy generation of mammalian cells suggesting that  were its target of action.Acremonium exuviarum, as an indoor fungus, is potentially hazardous to health because of the toxic peptaibols that it produces.</p><h1><a href=https://www.ncbi.nlm.nih.gov/pubmed/6114957>Inactivation of the bovine mitochondrial F1-ATPase with dicyclohexyl[14C]carbodiimide leads to the modification of a specific glutamic acid residue in the beta subunit.</a></h1><p>When the bovine mitochondrial F1-ATPase is inactivated with dicyclohexyl[14C]carbodiimide and then gel-filtered, from 2 to 3 g atoms of 14C are incorporated/mol of enzyme. Prior inactivation of the enzyme by the modification of an essential tyrosine residue with 4-chloro-7-nitrobenzofurazan, a reaction that can be reversed by thiols, does not affect the irreversible inactivation of the ATPase by dicyclohexyl[14C]carbodiimide. During the large scale modification of the F1-ATPase by dicyclohexyl[14C]carbodiimide which led to 70% inactivation, 1.9 g atoms of 14C were incorporated/mol of enzyme. Isolation of the alpha, beta, and gamma subunits from this large scale inactivation revealed that the gram atoms of 14C bound per mol of each of the subunits was: alpha, 0.04; beta, 0.56; and gamma, 0.04. The majority of the radioactivity in a cyanogen bromide digest of the 14C-labeled beta subunit was isolated in a fragment that has the following amino acid sequence: Glu-Leu-Ile-Asn-Asn-Val-Ala-Lys-Ala-His-Gly-Gly-Tyr-Ser-Val--Ala-Gly-Val-Gly -Glu-Arg--Arg-Glu-Gly-Asn-Asp-Leu-Tyr-Glu*-His-Met; where Glu* represents the N gamma-glutamyl derivative of dicyclohexyl[14C]urea.</p><h1><a href=https://www.ncbi.nlm.nih.gov/pubmed/15158725>Effects of mutagenesis of residue 221 on the properties of bacterial and mitochondrial elongation factor EF-Tu.</a></h1><p>During protein biosynthesis, elongation factor Tu (EF-Tu) delivers aminoacyl-tRNA (aa-tRNA) to the A-site of ribosomes. This factor is highly conserved throughout evolution. However, several key residues differ between bacterial and mammalian mitochondrial EF-Tu (EF-Tu(mt)). One such residue is Ser221 (Escherichia coli numbering). This residue is conserved as a Ser or  in the bacterial factors but is present as Pro269 in EF-Tu(mt). Pro269 reorients the loop containing this residue and shifts the adjoining beta-strand in EF-Tu(mt) compared to that of E. coli EF-Tu potentially altering the binding pocket for the acceptor stem of the aa-tRNA. Pro269 was mutated to a serine residue (P269S) in EF-Tu(mt). For comparison, the complementary mutation was created at Ser221 in E. coli EF-Tu (S221P). The E. coli EF-Tu S221P variant is poorly expressed in E. coli and the majority of the molecules fail to fold into an active conformation. In contrast, EF-Tu(mt) P269S is expressed to a high level in E. coli. When corrected for the percentage of active molecules, both variants function as effectively as their respective wild-type factors in ternary complex formation using E. coli -tRNA() and Cys-tRNA(Cys). They are also active in A-site binding and in vitro translation assays with E. coli -tRNA(). In addition, both variants are as active as their respective wild-type factors in ternary complex formation, A-site binding and in vitro translation assays using mitochondrial -tRNA().</p><h1><a href=https://www.ncbi.nlm.nih.gov/pubmed/12514185>Proline oxidase induces apoptosis in tumor cells, and its expression is frequently absent or reduced in renal carcinomas.</a></h1><p>Proline oxidase is a p53-induced gene that can mediate apoptosis in lung carcinoma cells. Here, we provide evidence implicating a role for proline oxidase in renal carcinoma. We observed absent or reduced expression of proline oxidase in 8 of 12 primary renal cell carcinomas, with respect to their normal tissue counterparts. Two renal cell carcinomas, which displayed little or no expression of proline oxidase, expressed p53s that were less capable of inducing proline oxidase than p53 isolated from normal renal tissue. One of those tumor-derived p53s contained a double transition mutation at amino acid residues 125 (Ala to ) and 193 (Arg to His), and the other exhibited a single transition mutation at amino acid 149 (Ser to ). Forced up-regulation of proline oxidase induced the formation of reactive oxygen species and mediated apoptosis in the 786-0 renal cell carcinoma cell line. A proline oxidase antisense vector repressed p53-induced up-regulation of proline oxidase, release of cytochrome c from , and apoptosis in 786-0 renal carcinoma cells. Taken together, these findings support a role for proline oxidase as a downstream effector in p53-mediated apoptosis. We hypothesize that its altered expression can contribute to the development of renal carcinomas. The presence of proline oxidase in , a primary organelle that regulates apoptosis, places this molecule in a subcellular localization that can directly influence the apoptotic pathway and thus tumorigenesis.</p><h1><a href=https://www.ncbi.nlm.nih.gov/pubmed/10733001>Modulation of glutamine uptake and phosphate-activated glutaminase activity in rat brain  by amino acids and their synthetic analogues.</a></h1><p>Uptake of L-[14C]Gln and phosphate-activated glutaminase (PAG) activity were measured in nonsynaptic  isolated from rat cerebral hemispheres, in the presence of protein and nonprotein amino acids and their synthetic structural analogues and derivatives. The uptake was inhibited by > 50% in the presence of a 10-fold excess of His, homocysteine (Hcy), Trp, Leu, Tyr, Ile, , Ala, , Met, Ser, by > 20% in the presence of a 10-fold excess of Val, Arg, Glu, and was not affected by a 10-fold excess of Orn, alpha-ketoglutarate, Tau and Pro. Uptake of L-[14C] Leu differed from Gln uptake by its resistance to Arg, Glu, and a relatively high sensitivity to the reference inhibitor of the plasma membrane transport of large neutral amino acids (L-system)--BCH (2-aminobicyclo[2.2.1]heptane-2-carboxylic acid), and a number of natural L-system substrates. A newly synthesized alanine analogue, 2'-cyano-(biphenyl) alanine, referred to as MRC01, was the only compound tested that inhibited Gln uptake more strongly than Leu uptake. The strongest Gln uptake inhibitors: MRC01, His, Hcy and Leu, inhibited PAG activity by > 50% when added at the inhibitor/Gln concentration ratio of 1:2. PAG activity was not affected by Tau, Lys or Pro, compounds which did affect Gln uptake. The results suggest that a number of natural amino acids function as common endogenous modulators of cerebral mitochondrial Gln uptake and its degradation. MRC01, because of its inhibitory potency towards both mitochondrial Gln uptake and PAG activity, may become a convenient tool in studying the role of Gln transport in its mitochondrial metabolism in intact CNS cell and tissues.</p><h1><a href=https://www.ncbi.nlm.nih.gov/pubmed/2164842>The small molecular mass ubiquinone-binding protein (QPc-9.5 kDa) in mitochondrial ubiquinol-cytochrome c reductase: isolation, ubiquinone-binding domain, and immunoinhibition.</a></h1><p>The small molecular mass ubiquinone-binding protein (QPc-9.5 kDa) was purified to homogeneity from 3-azido-2-methyl-5-methoxy-6-(3,7-dimethyl[3H]octyl)-1,4-benzoquinol+ ++- labeled bovine heart mitochondrial ubiquinol-cytochrome c reductase. The N-terminal amino acid sequence of the isolated protein is Gly-Arg-Gln--Gly-His-Leu--Arg-Val-Arg-His-, which is identical with that of a Mr = 9500 protein in the reductase [Borchart et al. (1986) FEBS Lett. 200, 81-86]. A ubiquinone-binding peptide was prepared from [3H]azidoubiquinol-labeled QPc-9.5 kDa protein by trypsin digestion followed by HPLC separation. The partial N-terminal amino acid sequence of this peptide, Val-Ala-Pro-Pro--Val-Ala--Tyr-Leu-, corresponds to amino acid residues 48-57 in the reported Mr = 9500 protein. According to the proposed structural model for the Mr = 9500 protein, the azido-Q-labeled peptide is located in the membrane on the matrix side. These results confirm our previous assessment that the Mr = 13,400 subunit is not the small molecular weight Q-binding protein. Purified antibodies against QPc-9.5 kDa have a high titer with isolated QPc-9.5 kDa protein and complexes that contain it. Although antibodies against QPc-9.5 kDa do not inhibit intact succinate- and ubiquinol-cytochrome c reductases, a decrease of 85% and 20% in restoration of succinate- and ubiquinol-cytochrome c reductases, respectively, is observed when delipidated succinate- or ubiquinol-cytochrome reductases are incubated with antibodies prior to reconstitution with ubiquinone and phospholipid, indicating that epitopes at the catalytic site of QPc-9.5 kDa are buried in the phospholipid environment.(ABSTRACT TRUNCATED AT 250 WORDS).</p><h1><a href=https://www.ncbi.nlm.nih.gov/pubmed/489581>Ornithine transcarbamylase of rat liver. Kinetic, physical, and chemical properties.</a></h1><p>Ornithine transcarbamylase of rat liver has been purified to homogeneity. The purified enzyme of specific activity 870 to 920 focuses as a single protein at pH 7.2. At pH 7.7, the Km for carbamyl phosphate is 0.026 mM, and the Km for ornithine is 0.04 mM. The inhibition constants of a number of amino acids that act as competitive inhibitors of the enzyme are reported. The native enzyme of Mr = 112,000 is composed of three subunits of Mr = 39,600 +/- 1,000. Chemical evidence indicates that the subunits are identical in amino acid composition and amino acid sequence. The amino acid sequence of the NH2-terminal region of ornithine transcarbamylase is Ser-Gln-Val-Gln-Leu-Lys-Gly-Ser-Asp-Leu-Leu--Leu-Lys-Asn-()-X--X-Glu-Ile-Gln-Tyr-Met-.</p><h1><a href=https://www.ncbi.nlm.nih.gov/pubmed/6149548>Amino acid sequence of bovine heart coupling factor 6.</a></h1><p>The amino acid sequence of bovine heart mitochondrial coupling factor 6 (F6) has been determined by automated Edman degradation of the whole protein and derived peptides. Preparations based on heat precipitation and ethanol extraction showed allotypic variation at three positions while material further purified by HPLC yielded only one sequence that also differed by a - replacement at residue 62. The mature protein contains 76 amino acids with a calculated molecular weight of 9006 and a pI of approximately equal to 5, in good agreement with experimentally measured values. The charged amino acids are mainly clustered at the termini and in one section in the middle; these three polar segments are separated by two segments relatively rich in nonpolar residues. Chou-Fasman analysis suggests three stretches of alpha-helix coinciding (or within) the high-charge-density sequences with a single beta-turn at the first polar-nonpolar junction. Comparison of the F6 sequence with those of other proteins did not reveal any homologous structures.</p><h1><a href=https://www.ncbi.nlm.nih.gov/pubmed/26082885>Substrate specificity of mitochondrial intermediate peptidase analysed by a support-bound peptide library.</a></h1><p>The substrate specificity of recombinant human mitochondrial intermediate peptidase (hMIP) using a synthetic support-bound FRET peptide library is presented. The collected fluorescent beads, which contained the hydrolysed peptides generated by hMIP, were sequenced by Edman degradation. The results showed that this peptidase presents a remarkable preference for polar uncharged residues at P1 and P1' substrate positions: Ser = Gln >  at P1 and Ser >  at P1'. Non-polar residues were frequent at the substrate P3, P2, P2' and P3' positions. Analysis of the predicted MIP processing sites in imported mitochondrial matrix proteins shows these cleavages indeed occur between polar uncharged residues. Previous analysis of these processing sites indicated the importance of positions far from the MIP cleavage site, namely the presence of a hydrophobic residue ( or Leu) at P8 and a polar uncharged residue (Ser or ) at P5. To evaluate this, additional kinetic analyses were carried out, using fluorogenic substrates synthesized based on the processing sites attributed to MIP. The results described here underscore the importance of the P1 and P1' substrate positions for the hydrolytic activity of hMIP. The information presented in this work will help in the design of new substrate-based inhibitors for this peptidase.</p><h1><a href=https://www.ncbi.nlm.nih.gov/pubmed/2246257>Mitochondrial phosphate transport. N-ethylmaleimide insensitivity correlates with absence of beef heart-like Cys42 from the Saccharomyces cerevisiae phosphate transport protein.</a></h1><p>The mitochondrial phosphate transport protein (PTP) has been purified in a reconstitutively active form from Saccharomyces cerevisiae and Candida parapsilosis. ADP/ATP carriers that copurify have been identified. The PTP from S. cerevisiae migrates as a single band (35 kDa) in sodium dodecyl sulfate gels with the same mobility as the N-ethylmaleimide-alkylated beef heart PTP. It does not cross-react with anti-sera against beef heart PTP. The CNBr peptide maps of the yeast and beef proteins are very different. The rate of unidirectional phosphate uptake into reconstituted proteoliposomes is stimulated about 2.5-fold to a Vmax of 170 mumol of phosphate min-1 (mg PTP)-1 (22 degrees C) by increasing the pHi of the proteoliposomes from 6.8 (same as ) to 8.0. The Km for Pi of this reconstituted activity is 2.2 mM. The transport is sensitive to mersalyl (50% inhibition at 60 microM) and insensitive to N-ethylmaleimide. We have purified peptides matching the highly conserved motif Pro-X-(Asp/glu)-X-X-(Lys/Arg)-X-(Arg/lys) (X is an unspecified amino acid) of the triplicate gene structure sequence of the beef heart PTP. The N-ethylmaleimide-reactive Cys42 of the beef heart protein, located between the two basic amino acids of this motif (Lys41-Cys42-Arg43), is replaced with a  in the yeast protein. This substitution most likely is responsible for the lack of N-ethylmaleimide sensitivity of the yeast protein and mersalyl thus reacts with another cysteine to inhibit the transport. Finally it is concluded that Cys42 has no essential role in the catalysis of inorganic phosphate transport by the mitochondrial phosphate transport protein.</p><h1><a href=https://www.ncbi.nlm.nih.gov/pubmed/24117191>Complete mitochondrial genome of the desert toad-headed agama, Phrynocephalus przewalskii (Reptilia, Squamata, Agamidae), a novel gene organization in vertebrate mtDNA.</a></h1><p>The mitogenome of the desert toad-headed agama, Phrynocephalus przewalskii, was amplified using polymerase chain reaction (PCR), long-and-accurate PCR and directly sequenced by primer walking. The complete mitogenome was 16,892 bp in size and contained 13 protein-coding, 23 tRNA, and 2 rRNA genes, and 1 control region. The mitogenome of the P. przewalskii was similar to those of other Phrynocephalus sand lizards in gene arrangement and composition, except that tRNA- and tRNA-Pro were exchanged and tRNA- had two copies. The control region comprised three parts, one between tRNA- and tRNA-, a second between the two tRNA- copies, and a third between tRNA-Pro and 12S RNA. The overall nucleotide composition of the H-strand was 36.3% A, 26.7% T, 12.5% G, 24.6% C. The complete mitogenome of P. przewalskii will contribute to understanding the evolution of the genus Phrynocephalus and the family Agamidae.</p><h1><a href=https://www.ncbi.nlm.nih.gov/pubmed/30867195>NMR- and MD simulation-based structural characterization of the membrane-associating FATC domain of ataxia telangiectasia mutated.</a></h1><p>The Ser/ protein kinase ataxia telangiectasia mutated (ATM) plays an important role in the DNA damage response, signaling in response to redox signals, the control of metabolic processes, and mitochondrial homeostasis. ATM localizes to the nucleus and at the plasma membrane, , peroxisomes, and other cytoplasmic vesicular structures. It has been shown that the C-terminal FATC domain of human ATM (hATMfatc) can interact with a range of membrane mimetics and may thereby act as a membrane-anchoring unit. Here, NMR structural and N relaxation data, NMR data using spin-labeled micelles, and MD simulations of micelle-associated hATMfatc revealed that it binds the micelle by a dynamic assembly of three helices with many residues of hATMfatc located in the headgroup region. We observed that none of the three helices penetrates the micelle deeply or makes significant tertiary contacts to the other helices. NMR-monitored interaction experiments with hATMfatc variants in which two conserved aromatic residues ( and Trp) were either individually or both replaced by alanine disclosed that the double substitution does not abrogate the interaction with micelles and bicelles at the high concentrations at which these aggregates are typically used, but impairs interactions with small unilamellar vesicles, usually used at much lower lipid concentrations and considered a better mimetic for natural membranes. We conclude that the observed dynamic structure of micelle-associated hATMfatc may enable it to interact with differently composed membranes or membrane-associated interaction partners and thereby regulate ATM's kinase activity. Moreover, the FATC domain of ATM may function as a membrane-anchoring unit for other biomolecules.© 2019 Abd Rahim et al.</p><h1><a href=https://www.ncbi.nlm.nih.gov/pubmed/27188525>Structure-function characterization of the human mitochondrial thiamin pyrophosphate transporter (hMTPPT; SLC25A19): Important roles for Ile(33), Ser(34), Asp(37), His(137) and Lys(291).</a></h1><p>Thiamin plays a critical role in cellular energy metabolism. Mammalian cells obtain the vitamin from their surroundings, converted it to thiamin pyrophosphate (TPP) in the cytoplasm, followed by uptake of TPP by  via a carrier-mediated process that involves the MTPPT (product of the SLC25A19 gene). Previous studies have characterized different physiological/biological aspects of the human MTPPT (hMTPPT), but less is known about structural features that are important for its function. Here, we used a protein-docking model ("Phyre2" and "DockingServer") to predict residues that may be important for function (substrate recognition) of the hMTPPT; we also examined the role of conserved positively-charged residues predicted ("PRALINE") to be in the trans-membrane domains (TMDs) in uptake of the negatively-charged TPP. Among the six residues predicted by the docking model (i.e., (29), Arg(30), Ile(33), Ser(34), Asp(37) and (298)), only Ile(33), Ser(34) and Asp(37) were found to be critical for function. While no change in translational efficiency/protein stability of the Ser(34) mutant was observed, both the Ile(33) and Asp(37) mutants showed a decrease in this parameter(s); there was also a decrease in the expression of the latter two mutants in . A need for a polar residue at position 34 of the hMTPPT was evident. Our findings with the positively-charged residues (i.e., His(82), His(137), Lys(231) and Lys(291)) predicted in the TMD showed that His(137) and Lys(291) are important for function (via a role in proper delivery of the protein to ). These investigations provide important information about the structure-function relationship of the hMTPPT.Copyright © 2016 Elsevier B.V. All rights reserved.</p><h1><a href=https://www.ncbi.nlm.nih.gov/pubmed/18440669>The decapeptide CMS001 enhances swimming endurance in mice.</a></h1><p>Now peptides achieve distinct advantages over protein in biological application because of its quick and easy absorption, low power, and high activity. Some bioactive peptides had been developed to be used in the management of exercise-related disorders. In this study, we investigated whether the decapeptide CMS001 (Pro---Lys--Tyr--Pro-His-) isolated from pig spleen had anti-fatigue effects. Male Balb/c mice were administered CMS001 (20 microg/(kgd)(-1) or 5 microg/(kgd)(-1) for 30 d, intraperitoneal injections) and tested in an exhaustive swim time task. In order to examine the mechanisms of CMS001 anti-fatigue effects, we analyzed liver glycogen stores, blood urea nitrogen (BUN) levels, lactic acid levels, ultrastructural integrity, and levels of both a free radical metabolite and an anti-oxidant enzyme. CMS001 treatment prolonged exhaustive swim time, increased liver glycogen levels, reduced BUN levels, and decreased accumulation of lactic acid in the blood, relative to mice injected with only saline. Examination of the ultrastructure of  and sarcoplasmic reticulum in skeletal and cardiac muscle of CMS001-treated and control mice revealed that CMS001 can reduce the damage to cardiac and skeletal muscle caused by an exhaustive swim challenge, such that the structure of most tissue specimens were normal in the peptide-treated group. Furthermore the free radical analysis after acute exercise indicated that CMS001 treatment decreased malondialdehyde (MDA) and increased superoxide dismutase (SOD) levels. The present findings indicate that the spleen-derived peptide CMS001 has anti-fatigue effects in mice, and further suggest that the mechanism may involve reduction of tissue damaging free radicals in muscle tissues.</p><h1><a href=https://www.ncbi.nlm.nih.gov/pubmed/8144526>Primary structural constraints of P-loop of mitochondrial F1-ATPase from yeast.</a></h1><p>Nucleotide binding proteins, including ras, elongation factor Tu, adenylate kinase, and the mitochondrial F1-ATPase have a glycine-rich motif known as the P-loop or the Walker A sequence (Walker, J. E., Saraste, M., Runswick, M. J., and Gay, N. J. (1982) EMBO J. 1, 945-951). The primary structural constraints have been determined in the P-loop located in the beta-subunit of the mitochondrial ATPase from yeast. The primary structural constraints were determined for 9 residues that form the P-loop, 190Gly-Gly-Ala-Gly-Val-Gly-Lys--Val198. Each residue was tested individually for possible functional replacements while keeping the primary structure of the remainder of the molecule constant. This analysis indicates with greater than 95% confidence that Gly190,Gly195, and Lys196 are invariant and Thr197 can only be replaced with Ser. The most alterable residue is Gly191, where 10 replacements, even , form a functional enzyme. The remaining positions allow some amino acid replacements while restricting others. The primary structural constraints of the P-loop of the mitochondrial F1 suggests that the three-dimensional structure of the P-loop is similar to that of ras.</p><h1><a href=https://www.ncbi.nlm.nih.gov/pubmed/4055729>The amino acid sequences of the flavin-peptides of dimethylglycine dehydrogenase and sarcosine dehydrogenase from rat liver .</a></h1><p>The flavoenzymes dimethylglycine dehydrogenase (EC 1.5.99.2) and sarcosine dehydrogenase (EC 1.5.99.1) contain covalently bound FAD linked via the 8 alpha-position of the isoalloxazine ring to the imidazole N(3) of a histidine residue (Cook, R. J., Misono, K. S., and Wagner, C. (1984) J. Biol. Chem. 259, 12475-12480). The flavin-peptides from tryptic digests of these two enzymes have been isolated and sequenced. Automated sequence analysis showed that the flavin-peptide from dimethylglycine dehydrogenase contained 25 amino acid residues in the following sequence: Ser-Glu-Leu--Ala-Gly-Ser- -Trp-His(flavin)-Ala-Ala-Gly-Leu---Tyr--His-Pro-Gly-Ile-A sn-Leu-Lys. The sequence determined for the flavin-peptide from sarcosine dehydrogenase contained 14 amino acid residues Leu--Ser-Gly---Trp-His(flavin)--Ala-Gly-Leu-Gly-Arg.</p><h1><a href=https://www.ncbi.nlm.nih.gov/pubmed/3812085>Mechanism of synthesis and localization of mitochondrial and cytosolic fumarases in rat liver.</a></h1><p>Fumarases in the mitochondrial and cytosolic fractions of rat liver were separately purified and crystallized. These two fumarases were not distinguishable in physicochemical, catalytic, or immunochemical properties. The sequences of seven amino acids in the C-terminal portions of the two fumarases were shown using carboxypeptidase P to be identical, i.e.-Val-Asp-Glu--Ala-Leu-Lys-. The amino acid sequence of the N-terminal portion of the mitochondrial fumarase was determined by the Edman method as Ala-Gln-Gln-Asn--Glu-Ile-Pro-Asp-, but that of the cytosolic fumarase could not be determined by the Edman method, since the N-terminal amino acid was blocked. The N-terminal amino acid of the cytosolic fumarase was identified as N-acetyl-alanine by analysis of the acidic amino acid produced by digestion of the enzyme protein with pronase E, carboxypeptidase A and B. Then the sequence of five amino acids in the N-terminal portion was determined by analyzing the acidic peptide obtained by limited proteolysis of the enzyme protein with carboxypeptidase A as Ac-Ala-Ser-Gln-Asn-Ser-. Peptide mapping of the tryptic peptides obtained from the mitochondrial and cytosolic fumarases showed no difference in the amino acid sequences of the two except in their N-terminal portions. The turnover rates of the mitochondrial and cytosolic fumarases were determined by injecting L-[U-14C]leucine into rat and following the decay of specific radioactivity incorporated into immunoprecipitates from the partially purified enzyme. The half-life of the cytosolic fumarase was estimated as 4.8 days from the decay curve of its specific radioactivity. The decay curve of the specific radioactivity of the mitochondrial fumarase, obtained after a single injection of L-[U-14]leucine, was quite unusual: its specific radioactivity remained constant for about 7 days after pulse labeling, and then decreased exponentially with a half-life of 9.7 days. Similar amounts of cytosolic and mitochondrial fumarase were found in the livers of the rat, mouse, rabbit, dog, chicken, snake, frog, and carp, respectively. Similar subcellular distributions of the enzyme were also found in the kidney, heart, and skeletal muscle of rats, and in hepatoma cells (AH-109A). However, in rat brain no fumarase activity was detected in the cytosolic fraction. Two putative precursor polypeptides of rat liver fumarase were synthesized when rat liver RNA was translated in vitro in a rabbit reticulocyte lysate system.(ABSTRACT TRUNCATED AT 400 WORDS).</p><h1><a href=https://www.ncbi.nlm.nih.gov/pubmed/18558627>Phosphoproteome analysis of isoflurane-protected heart : phosphorylation of adenine nucleotide translocator-1 on Tyr194 regulates mitochondrial function.</a></h1><p>Reversible phosphorylation of mitochondrial proteins is essential in the regulation of respiratory function, energy metabolism, and mitochondrion-mediated cell death. We hypothesized that mitochondrial protein phosphorylation plays a critical role in cardioprotection during pre and postconditioning, two of the most efficient anti-ischaemic therapies.Using phosphoproteomic approaches, we investigated the profiles of phosphorylated proteins in Wistar rat heart  protected by pharmacological pre and postconditioning elicited by isoflurane. Sixty-one spots were detected by two-dimensional blue-native gel electrophoresis-coupled Western blotting using a phospho-Ser//Tyr-specific antibody, and 45 of these spots were identified by matrix-assisted laser desorption/ionization-time of flight mass spectrometry. Eleven protein spots related to oxidative phosphorylation, energy metabolism, chaperone, and carrier functions exhibited significant changes in their phosphorylation state when protected  were compared with unprotected. Using a phosphopeptide enrichment protocol followed by liquid chromatography-MS/MS, 26 potential phosphorylation sites were identified in 19 proteins. Among these, a novel phosphorylation site was detected in adenine nucleotide translocator-1 (ANT1) at residue Tyr(194). Changes in ANT phosphorylation between protected and unprotected  were confirmed by immunoprecipitation. The biological significance of ANT phosphorylation at Tyr(194) was further tested with site-directed mutagenesis in yeast. Substitution of Tyr(194) with , mimicking the non-phosphorylated state, resulted in the inhibition of yeast growth on non-fermentable carbon sources, implying a critical role of phosphorylation at this residue in regulating ANT function and cellular respiration.Our analysis emphasizes the regulatory functions of the phosphoproteome in heart  and reveals a novel, potential link between bioenergetics and cardioprotection.</p><h1><a href=https://www.ncbi.nlm.nih.gov/pubmed/25640585>Design and synthesis of new substrates of HtrA2 protease.</a></h1><p>HtrA2 belongs to the HtrA (high temperature requirement A) family of ATP-independent serine proteases. The primary function of HtrA2 includes maintaining the  homeostasis, cell death (by apoptosis, necrosis, or anoikis), and contribution to the cell signaling. Several recent reports have shown involvement of HtrA2 in development of cancer and neurodegenerative disorders. Here, we describe the profiling of HtrA2 protease substrate specificity via the combinatorial chemistry approach that led to the selection of novel intramolecularly quenched substrates. For all synthesized compounds, the highest HtrA2-mediated hydrolysis efficiency and selectivity among tested HtrA family members was observed for ABZ-Ile-Met--Abu-Tyr-Met--Tyr(3-NO2)-NH2, which displayed a specificity constant kcat/KM value of 14,535M(-1)s(-1).Copyright © 2015 Elsevier Inc. All rights reserved.</p><h1><a href=https://www.ncbi.nlm.nih.gov/pubmed/4066697>Sequence of the N-terminal formic acid fragment and location of the N-ethylmaleimide-binding site of the phosphate transport protein from beef heart .</a></h1><p>The N-terminal formic acid fragment (FA1) of the N-[3H]ethylmaleimide-labeled and carboxymethylated bovine mitochondrial phosphate transport protein (PTPN*CM) has been purified and completely sequenced: NH2-Ala-Val-Glu-Glu-Gln-Tyr-Ser-Cys-Asp-Tyr10-Gly-Ser-Gly-Arg-- -Ile-Leu-Cys- Gly20-Leu-Gly-Gly-Ile-Ile-Ser-Cys-Gly--Thr30-His- -Ala-Leu-Val-Pro-Leu-Asp- -Leu-Val40-Lys-Cys(N-[3H]ethylmaleimide)-Arg-Met-Gln-Val-Asp- COOH. By thermolysin digestion of FA1 and high-performance liquid chromatography isolation of the radioactive subfragment Leu39-Arg43, the sole N-ethylmaleimide-binding residue has been identified as Cys42. FA1 contains a high mole percentage of cysteine (8.5%) and shows silver staining anomaly. Its sequence reveals significant homology in the triplicated gene regions (Pro27,132,229) of the mitochondrial ADP/ATP carrier from beef heart and Neurospora crassa. The hydropathic profile suggests that FA1 contains a transmembrane segment (Phe15-Val40) with only one basic (His31) and one acidic (Asp38) residue. The presence of the phosphate transport protein gene among nuclear genes is suggested from a lack of significant homology between the reverse-translated FA1 (mitochondrial codons) and the bovine mitochondrial genome. The inhibitory action of N-ethylmaleimide on the phosphate transport mechanism is discussed.</p><h1><a href=https://www.ncbi.nlm.nih.gov/pubmed/2524209>Photolabeling of the phosphate binding site of mitochondrial F1-ATPase by [32P]azidonitrophenyl phosphate. Identification of the photolabeled amino acid residues.</a></h1><p>[32P]Azidonitrophenyl phosphate [( 32P]ANPP) is a photoactivatable analogue of Pi. It competes efficiently with Pi for binding to the F1 sector of beef heart mitochondrial ATPase and photolabels the Pi binding site located in the beta subunit of F1 [Lauquin, G. J. M., Pougeois, R., & Vignais, P. V. (1980) Biochemistry 19, 4620-4626]. By cleavage of the photolabeled beta subunit of F1 with cyanogen bromide, trypsin, and chymotrypsin, bound [32P]ANPP was localized in a fragment spanning  299- 326. By Edman degradation of the radiolabeled tryptic peptide spanning Ile 296-Arg 337, [32P]ANPP was found to be attached covalently by its photoreactive group to Ile 304, Gln 308, and Tyr 311. These results are discussed in terms of a model in which the phosphate group of [32P]ANPP interacts with a glycine-rich sequence of the beta subunit, spanning Gly 156-Lys 162, which is spatially close to the photolabeled Ile 304-Tyr 311 segment of the same subunit.</p><h1><a href=https://www.ncbi.nlm.nih.gov/pubmed/2266129>Delta-bag cell peptide from the egg-laying hormone precursor of Aplysia. Processing, primary structure, and biological activity.</a></h1><p>The bag cells of the marine mollusk Aplysia express a gene encoding a 271-residue egg-laying hormone (ELH) precursor that is processed into at least nine peptide products. Four of the peptides have been identified in bag cell releasates and are known to act as nonsynaptic neurotransmitters in the abdominal ganglion. The isolation, primary structure, and proposed biological activity of a fifth peptide product (delta-bag cell peptide (delta-BCP)) from the ELH precursor are described. delta-BCP was established to be a 39-residue peptide: NH2-Asp-Gln-Asp-Glu-Gly-Asn--Arg-Arg--Pro--Asn-Ala-Val-Ser-Met- Ser-Ala-Asp- Glu-Asn-Ser-Pro--Asp-Leu-Ser-Asn-Glu-Asp-Gly-Ala-Val-Tyr-Gln-Arg- Asp-Leu-COOH. This sequence corresponds to residues 81-119 of the ELH prohormone and shares sequence identity with atrial gland peptides A and B. Significantly, synthetic delta-BCP stimulated Ca2+ uptake into  of secretory cells in the albumin gland in vitro, suggesting that the peptide regulates the cellular release of perivitelline fluid by the gland. Similar results were obtained with purified peptide A and a shorter version of delta-BCP (delta-BCP-(14-33)). These results indicate that delta-BCP belongs to a family of structurally related peptides with similar pharmacological activities that center at a conserved region of sequence corresponding to delta-BCP-(14-33).</p><h1><a href=https://www.ncbi.nlm.nih.gov/pubmed/2374706>Transfer RNAs of potato (Solanum tuberosum)  have different genetic origins.</a></h1><p>Total transfer RNAs were extracted from highly purified potato . From quantitative measurements, the in vivo tRNA concentration in  was estimated to be in the range of 60 microM. Total potato mitochondrial tRNAs were fractionated by two-dimensional polyacrylamide gel electrophoresis. Thirty one individual tRNAs, which could read all sense codons, were identified by aminoacylation, sequencing or hybridization to specific oligonucleotides. The tRNA population that we have characterized comprises 15 typically mitochondrial, 5 'chloroplast-like' and 11 nuclear-encoded species. One tRNA(Ala), 2 tRNAs(Arg), 1 tRNA(Ile), 5 tRNAs(Leu) and 2 tRNAs() were shown to be coded for by nuclear DNA. A second, mitochondrial-encoded, tRNA(Ile) was also found. Five 'chloroplast-like' tRNAs, tRNA(Trp), tRNA(Asn), tRNA(His), tRNA(Ser)(GGA) and tRNA(Met)m, presumably transcribed from promiscuous chloroplast DNA sequences inserted in the mitochondrial genome, were identified, but, in contrast to wheat (1), potato  do not seem to contain 'chloroplast-like' tRNA(Cys) and tRNA(). The two identified tRNAs(Val), as well as the tRNA(Gly), were found to be coded for by the mitochondrial genome, which again contrasts with the situation in wheat, where the mitochondrial genome apparently contains no tRNA(Val) or tRNA(Gly) gene (2).</p><h1><a href=https://www.ncbi.nlm.nih.gov/pubmed/10773071>Repair of tRNAs in metazoan .</a></h1><p>The integrity of 3'-ends of tRNAs is essential for aminoacylation and consequently for protein synthesis. The CCA-termini are generated and, if truncated by exonucleolytic activity, restored by tRNA nucleotidyltransferase. However, further truncations at the 3'-end can occur by exonuclease activity or during processing of overlapping tRNA primary transcripts in metazoan . In the latter case, the upstream tRNA is released in a 3'-truncated form (lacking up to six bases) and subsequently completed. In human , tRNA(Tyr)(missing the discriminator nucleotide A(73)) is completed by a discriminator adding activity followed by CCA addition. Since in vivo a high percentage of further 3'-terminally degraded human tRNA(Tyr)transcripts could be observed, it was tested in an in vitro system whether this repair mechanism for tRNA 3'-ends acts also on these further degraded tRNA versions. Additionally, 3'-truncated versions of two non-overlapping mitochondrial tRNAs (tRNA()and tRNA()) were examined. The results show that these transcripts can be repaired during incubation. A similar base incorporating activity was observed in mouse , indicating that a repair mechanism for the 3'-end of several tRNAs exists in  of humans and possibly other metazoans which goes beyond the CCA addition.</p><h1><a href=https://www.ncbi.nlm.nih.gov/pubmed/7737993>Reversal by GroES of the GroEL preference from hydrophobic amino acids toward hydrophilic amino acids.</a></h1><p>The chaperones GroEL/hsp60 are present in all prokaryotes and in  and chloroplasts of eukaryotic cells. They are involved in protein folding, protein targeting to membranes, protein renaturation, and control of protein-protein interactions. They interact with many polypeptides in an ATP-dependent manner and possess a peptide-dependent ATPase activity. GroEL/hsp60 cooperates with GroES/hsp10, and the productive folding of proteins by GroEL generally requires GroES, which appears to regulate the binding and release of substrate proteins by GroEL. In a recent study, we have shown that GroEL interacts preferentially with the side chains of hydrophobic amino acids (Ile, , Val, Leu, and Trp) and more weakly with several polar or charged amino acids, including the strongest alpha-helix and beta-sheet formers (Glu, Gln, His, , and Tyr). In this study, we show that GroES reduces the specificity of GroEL for hydrophobic amino acids and increases its specificity for hydrophilic ones. This shift by GroES of the GroEL specificity from hydrophobic amino acids toward hydrophilic ones might be of importance for its function in protein folding.</p><h1><a href=https://www.ncbi.nlm.nih.gov/pubmed/8011636>Prediction and site-specific mutagenesis of residues in transmembrane alpha-helices of proton-pumping nicotinamide nucleotide transhydrogenases from Escherichia coli and bovine heart .</a></h1><p>Nicotinamide nucleotide transhydrogenase from bovine heart consists of a single polypeptide of 109 kD. The complete gene for this transhydrogenase was constructed, and the protein primary structure was determined from the cDNA. As compared to the previously published sequences of partially overlapping clones, three residues differed: Ala591 (previously ), Val777 (previously Glu), and Ala782 (previously Arg). The Escherichia coli transhydrogenase consists of an alpha subunit of 52 kD and a beta subunit of 48 kD. Alignment of the protein primary structure of the bovine trashydrogenase with that of the transhydrogenase from E. coli showed an identity of 52%, indicating similarly folded structures. Prediction of transmembrane-spanning alpha-helices, obtained by applying several prediction algorithms to the primary structures of the revised bovine heart and E. coli transhydrogenases, yielded a model containing 10 transmembrane alpha-helices in both transhydrogenases. In E. coli transhydrogenase, four predicted alpha-helices were located in the alpha subunit and six alpha-helices were located in the beta subunit. Various conserved amino acid residues of the E. coli transhydrogenase located in or close to predicted transmembrane alpha-helixes were replaced by site-specific mutagenesis. Conserved negatively charged residues in predicted transmembrane alpha-helices possibly participating in proton translocation were identified as beta Glu82 (Asp655 in the bovine enzyme) and beta Asp213 (asp787 in the bovine enzyme) located close to the predicted alpha-helices 7 and 9 of the beta subunit. beta Glu82 was replaced by Lys or Gln and beta Asp213 by Asn or His. However, the catalytic as well as the proton pumping activity was retained. In contrast, mutagenesis of the conserved beta His91 residue (His664 in the bovine enzyme) to Ser, , and Cys gave an essentially inactive enzyme. Mutation of alpha His450 (corresponding to His481 in the bovine enzyme) to  greatly lowered catalytic activity without abolishing proton pumping. Since no other conserved acidic or basic residues were predicted in transmembrane alpha-helices regardless of the prediction algorithm used, proton translocation by transhydrogenase was concluded to involve a basic rather than an acidic residue. The only conserved cysteine residue, beta Cys260 (Cys834 in the bovine enzyme), located in the predicted alpha-helix 10 of the E. coli transhydrogenase, previously suggested to function as a redox-active dithiol, proved not to be essential, suggesting that redox-active dithiols do not play a role in the mechanism of transhydrogenase.</p><h1><a href=https://www.ncbi.nlm.nih.gov/pubmed/23156681>Universal primers for amplification of the complete mitochondrial control region in marine fish species.</a></h1><p>Through multiple alignment analysis of mitochondrial tRNA- and tRNA- sequences from 161 fishes, new universal primers specially targeting the entire mitochondrial control region were designed. This new primer set successfully amplified the expected PCR products from various kinds of marine fish species, belonging to various families, and the amplified segments were confirmed to be the control region by sequencing. These primers provide a useful tool to study the control region diversity in economically important fish species, the possible mechanism of control region evolution, and the functions of the conserved motifs in the control region.</p></html>