<html><h2><a href=https://www.ncbi.nlm.nih.gov/pubmed/2036381>Correlations between kinetic and X-ray analyses of engineered enzymes: crystal structures of mutants Cys----Gly-35 and ----Phe-34 of tyrosyl-tRNA synthetase.</a></h2><p>The crystal structures of two mutant tyrosyl-tRNA synthetases (TyrTS) are reported to test predictions from kinetic data about structural perturbations and also to aid in the interpretation of apparent strengths of hydrogen bonds measured by protein engineering. The enzyme-tyrosine and enzyme-tyrosyl adenylate complexes of the mutant, TyrTS(Cys----Gly-35), have been determined at 2.5- and 2.7-A resolution, respectively. Residue Cys-35 is in the ribose binding site. Small rearrangements in structure are seen in the enzyme-tyrosine complex that are localized around the cavity created by the mutation. The side chain of Thr-51 moves to occupy the cavity, and -52 adopts two significantly populated conformations, one as in the native enzyme and a second unique to the mutant. On binding tyrosyl adenylate, -52 in the mutant crystal structure preferentially occupies the conformation observed in the native structure. The side chain at Thr-51 becomes disordered. The double-mutant test, which was designed to detect interactions between residues, had previously shown a discrepancy of some 0.4 kcal/mol on mutating Cys-35 and Thr-51 separately and together. A crystal structure of a second mutant, delta TyrTS(----Phe-34), complexed with tyrosine has been determined at 2.7-A resolution. -34 in wild-type enzyme makes a hydrogen bond with the phenolic  of the bound tyrosine substrate. The mutant crystal structure was solved to discover whether or not a water molecule binds to the substrate instead of the hydroxyl of -34 as the interpretation of apparent binding energies from site-directed mutagenesis experiments hinges crucially on whether there is access of water to the mutated region.</p><h2><a href=https://www.ncbi.nlm.nih.gov/pubmed/31134998>Peptide identification from a Porphyra dioica protein hydrolysate with antioxidant, angiotensin converting enzyme and dipeptidyl peptidase IV inhibitory activities.</a></h2><p>A Porphyra dioica protein extract was enzymatically hydrolysed and then fractionated using semi-preparative reverse-phase high performance chromatography. The hydrolysate and its fractions were tested for their  radical absorbance capacity (ORAC) along with their angiotensin converting enzyme (ACE) and dipeptidyl peptidase IV (DPP-IV) inhibitory activities. The most potent fraction was analysed by liquid chromatography mass spectrometry. Eight peptide sequences were selected for synthesis based on their structure-activity criteria for bioactivity. Asp---Lys-Arg showed the highest ORAC activity (4.27 ± 0.15 μmol Trolox equivalent per μM). Thr---Ala had the highest ACE inhibitory activity (IC50: 89.7 ± 7.10 μM). -Leu-Val-Ala was the only peptide showing DPP-IV inhibitory activity (IC50: 439 ± 44 μM). Apart from Asp---Lys-Arg and Thr---Ala, which displayed increased ORAC activity, the bioactivities of the peptides were either maintained or decreased following in vitro simulated gastrointestinal digestion. The results indicate that P. dioica-derived peptides may have potential applications as health enhancing ingredients.</p><h2><a href=https://www.ncbi.nlm.nih.gov/pubmed/10222199>Crystal structure of Trematomus newnesi haemoglobin re-opens the root effect question.</a></h2><p>As new structural data have become available, somewhat contrasting explanations of the Root effect in fish haemoglobins (Hb) have been provided. Hb 1 of the Antarctic fish Trematomus newnesi has a nearly pH-independent  affinity, in spite of 95 % sequence identity with Hb 1 of Trematomus (previously named Pagothenia) bernacchii that has a strong Root effect. Here, the 2.2 A R-state structure of Trematomus newnesi Hb 1 is presented. The structure is similar to that of Root effect fish Hbs from Spot and T. bernacchii, suggesting that the differences in the pH dependence cannot be related to the modulation of the R-state. In comparison to T. bernacchii Hb 1, the role of the three mutations Thr41 (C6)alpha-->, Ala97 (G3)alpha-->Ser and His41 (C7)beta--> at the alpha1beta2-interface is discussed.Copyright 1998 Academic Press.</p><h2><a href=https://www.ncbi.nlm.nih.gov/pubmed/1883830>Human immunodeficiency virus-1 protease. 1. Initial velocity studies and kinetic characterization of reaction intermediates by 18O isotope exchange.</a></h2><p>The peptidolytic reaction of HIV-1 protease has been investigated by using four oligopeptide substrates, Ac-Ser-Gln-Asn--Pro-Val-Val-NH2, Ac-Arg-Ala-Ser-Gln-Asn--Pro-Val-Val-NH2, Ac-Ser-Gln-Ser--Pro-Val-Val-NH2, and Ac-Arg-Lys--Leu-Phe-Leu-Asp-Gly-NH2, that resemble two cleavage sites found within the naturally occurring polyprotein substrates Pr55gag and Pr160gag-pol. The values for the kinetic parameters V/KEt and V/Et were 0.16-7.5 mM-1 s-1 and 0.24-29 s-1, respectively, at pH 6.0, 0.2 M NaCl, and 37 degrees C. By use of a variety of inorganic salts, it was concluded that the peptidolytic reaction is nonspecifically activated by increasing ionic strength. V/K increased in an apparently parabolic fashion with increasing ionic strength, while V was either increased or decreased slightly. From product inhibition studies, the kinetic mechanism of the protease is either random or ordered uni-bi, depending on the substrate studied. The reverse reaction or a partial reverse reaction (as measured by isotope exchange of the carboxylic product into substrate) was negligible for most of the oligopeptide substrates, but the enzyme catalyzed the formation of Ac-Ser-Gln-Asn--Phe-Leu-Asp-Gly-NH2 from the products Ac-Ser-Gln-Asn- and Phe-Leu-Asp-Gly-NH2. The protease-catalyzed exchange of an atom of 18O from H2 18O into the re-formed substrates occurred at a rate which was 0.01-0.12 times that of the forward peptidolytic reaction. The results of these studies are in accord with the formation of a kinetically competent enzyme-bound amide hydrate intermediate, the collapse of which is the rate-limiting chemical step in the reaction pathway.</p><h2><a href=https://www.ncbi.nlm.nih.gov/pubmed/29352967>Product specificity of fungal 8R- and 9S-dioxygenases of the peroxidase-cyclooxygenase superfamily with amino acid derivatized polyenoic fatty acids.</a></h2><p>Pathogenic fungi express fatty acid dioxygenases (DOX) fused to cytochromes P450 with diol or allene oxide synthase activities. The orientation of the fatty acids in the active sites of DOX was investigated with amino acid conjugates of 18:3n-3 and 18:2n-6. 9S-DOX-allene oxide synthase (AOS) oxidized the Gly, , and Trp derivatives at C-9, which suggests that these conjugates enter the substrate recognition site with the omega end in analogy with fatty acids bound to cyclooxygenases and coral 8R-lipoxygenase (8R-LOX). In contrast, 7,8-diol synthases (7,8-LDS), 5,8-LDS, and 8R-DOX-AOS oxidized the Gly conjugates in most case only to small amounts of metabolites, but with retention of hydrogen abstraction at C-8 and relatively minor hydrogen abstraction at C-11. The  and Trp conjugates were not oxidized at C-8, and often insignificantly at C-9/C-13. The 8-DOX domains of these enzymes likely position the carboxyl group of substrates at the end of the active site in analogy with plant α-DOX and 9-LOX.  radicals of the 9S-DOX and 8R-DOX domains catalyze antarafacial hydrogen abstraction and oxygen insertion in 18:3n-3. This occurs by abstraction of the proR and proS hydrogens at C-11 and C-8, respectively, in agreement with different "head to tail" orientation in the active site.Copyright © 2018 Elsevier Inc. All rights reserved.</p><h2><a href=https://www.ncbi.nlm.nih.gov/pubmed/18658150>SNAP-25 substrate peptide (residues 180-183) binds to but bypasses cleavage by catalytically active Clostridium botulinum neurotoxin E.</a></h2><p>Clostridium botulinum neurotoxins are the most potent toxins to humans. The recognition and cleavage of SNAREs are prime evente in exhibiting their toxicity. We report here the crystal structure of the catalytically active full-length botulinum serotype E catalytic domain (BoNT E) in complex with SNAP-25 (a SNARE protein) substrate peptide Arg(180)-(181)-Met(182)-Glu(183) (P1-P3'). It is remarkable that the peptide spanning the scissile bond binds to but bypasses cleavage by the enzyme and inhibits the catalysis fairly with K(i) approximately 69 microm. The inhibitory peptide occupies the active site of BoNT E and shows well defined electron density. The catalytic zinc and the conserved key residue (350) of the enzyme facilitate the docking of Arg(180) (P1) by interacting with its carbonyl  that displaces the nucleophilic water. The general base Glu(212) side chain interacts with the main chain amino group of P1 and P1'. Conserved Arg(347) of BoNT E stabilizes the proper docking of the (181) (P1') main chain, whereas the hydrophobic pockets stabilize the side chains of (181) (P1') and Met(182) (P2'), and the 250 loop stabilizes Glu(183) (P3'). Structural and functional analysis revealed an important role for the P1' residue and S1' pocket in driving substrate recognition and docking at the active site. This study is the first of its kind and rationalizes the substrate cleavage strategy of BoNT E. Also, our complex structure opens up an excellent opportunity of structure-based drug design for this fast acting and extremely toxic high priority BoNT E.</p><h2><a href=https://www.ncbi.nlm.nih.gov/pubmed/16249103>Deamidation of peptides in aerobic nitric oxide solution by a nitrosative pathway.</a></h2><p>Hydrolytic deamidation of asparagine (Asn) and glutamine (Gln) residues to aspartate (Asp) and glutamate (Glu), respectively, can have significant biological consequences. We hypothesize that a wholly different mechanism of deamidation might occur in the presence of aerobic nitric oxide (NO). Accordingly, we examined the deamidating ability of aerobic NO toward three model peptides, 2,4-dinitrophenyl (DNP)-Pro-Gln-Gly, Lys-Trp-Asp-Asn-Gln, and Ser-Glu-Asn--Pro--Val, incubating them with the NO-generating compound, Et(2)N[N(O)NO]Na (DEA/NO, 30-48 mM), in aerobic, pH 7.4, buffer at 37 degrees C. DNP-Pro-Glu-Gly was detected after 2 h, while Lys-Trp-Asp-Asp-Gln, Lys-Trp-Asp-Asn-Glu, and Ser-Glu-Asp--Pro--Val were detected within 10 min, accumulating in apparent yields of up to approximately 10%. In the latter case, tyrosine nitration was also observed, producing the expected nitrotyrosine residue. DEA/NO solutions preincubated to exhaust the NO before the peptides were added did not induce detectable deamidation. The data demonstrate that aerobic NO exposures can lead to nitrosative deamidation of peptides, a pathway that differs from the established hydrolytic deamidation mechanism in several key respects: the by-products of the former are molecular nitrogen and an acid (HONO) while that of the latter is a base (NH(3)); the nitrosative path can in principle proceed in the absence of water molecules; Asn is much more easily deamidated than Gln in the hydrolytic pathway, while the two amino acid residues were deamidated to a similar extent by exposure to NO in the presence of .</p><h2><a href=https://www.ncbi.nlm.nih.gov/pubmed/29078103>Ethanol binding sites on proteins.</a></h2><p>This study is on the analysis of ethanol binding sites on 3D structures of nonredundant proteins from the Protein Data Bank. The only one amino acid residue that is significantly overrepresented around ethanol molecules is . There are usually two or more  residues in the same ethanol binding site, while residues of Thr, Asp and Gln are underrepresented around them. Residues of Ala and Pro are significantly underrepresented in ethanol binding surfaces. Several residues (Phe, Val, Pro, Ala, Arg, His, Ser, Asp) bind ethanol significantly more frequent if they are not included in beta strands. Residues of Ala,  and Arg preferably bind ethanol when they are included in an alpha helix. Ethanol molecules often make hydrogen bonds with oxygen and nitrogen atoms from the main chain of a protein. Because of this reason, the binding of ethanol may be associated with the decrease of the length of alpha helices and the disappearance of 3/10 helices. Obtained data should be useful for studies on new targets of the direct action of ethanol on enzymes, receptors, and transcription factors.Copyright © 2017 Elsevier Inc. All rights reserved.</p><h2><a href=https://www.ncbi.nlm.nih.gov/pubmed/8075386>Conformation and sweet tastes of L-aspartyl dipeptide methyl esters.</a></h2><p>In order to investigate the conformational preferences to elicit tastes, conformational free energy calculations using an empirical potential (ECEPP/2) and the hydration shell model were carried out on the L-aspartyl dipeptide methyl esters, L-(+)HAsp(-)-L-Xaa-OMe, in the hydrated state, where Xaa includes sweet (Phe, , Met, and Gly), bitter (Ala, Trp, Val, Leu, and ), and tasteless (Ser, Thr, and Abu) residues. The refined preferred conformation of the Phe dipeptide (aspartame) with side chain chi 1/2 conformation g- is g-Fg- in the hydrated state, which is consistent with the structure deduced from 1H-nmr experiments. Irrespective of the Xaa and taste, all the dipeptides have the same conformation for the Asp residue, which is attributable to the hydrogen bond between protonated amino hydrogen and carboxylate  and the favored hydration of the carboxylate group. This implies that the L-aspartyl residue is a necessary factor for the dipeptides to be sweet not a sufficient factor. The computed conformational preferences for sweet, bitter, and tasteless dipeptides in the hydrated state indicate to us that the conformation about the N--C alpha bond of the Xaa residue, i.e., the orientation of the hydrophobic moiety with respect to the AH/B functionalities in the aspartyl moiety, seems to be crucial to elicit the tastes. In addition, the hydrophobicity and the size of the Xaa residue are found to play a major role in determining the tastes. These well accord with the related works reported previously.</p><h2><a href=https://www.ncbi.nlm.nih.gov/pubmed/7896794>Residues important for radical stability in ribonucleotide reductase from Escherichia coli.</a></h2><p>The R2 protein of ribonucleotide reductase contains at the side chain of tyrosine 122 a stable free radical, which is essential for enzyme catalysis. The tyrosyl radical is buried in the protein matrix close to a dinuclear iron center and a cluster of three hydrophobic residues (Phe-208, Phe-212, and -234) conserved throughout the R2 family. A key step in the generation of the tyrosyl radical is the activation of molecular  at the iron center. It has been suggested that the hydrophobic cluster provides an inert binding pocket for molecular  bound to the iron center and that it may play a role in directing the oxidative power of a highly reactive intermediate toward tyrosine 122. We have tested these hypotheses by constructing the following mutant R2 proteins:F208Y, F212Y, F212W, and I234N. The resulting mutant proteins all have the ability to form a tyrosine radical, which indicates that binding of molecular  can occur. In the case of F208Y, the yield of tyrosyl radical is substantially lower than in the wild-type case. A competing reaction resulting in hydroxylation of -208 implies that the phenylalanine at position 208 may influence the choice of target for electron abstraction. The most prominent result is that all mutant proteins show impaired radical half-life; in three of the four mutants, the half-lives are several orders of magnitude shorter than that of the wild-type radical. This suggests that the major role of the hydrophobic pocket is to stabilize the tyrosyl radical. This hypothesis is corroborated by comparative studies of the environment of other naturally occurring tyrosyl radicals.</p><h2><a href=https://www.ncbi.nlm.nih.gov/pubmed/18687350>Opposing roles for caspase and calpain death proteases in L-glutamate-induced oxidative neurotoxicity.</a></h2><p>Oxidative glutamate toxicity in HT22 murine hippocampal cells is a model for neuronal death by oxidative stress. We have investigated the role of proteases in HT22 cell oxidative glutamate toxicity. L-glutamate-induced toxicity was characterized by cell and nuclear shrinkage and chromatin condensation, yet occurred in the absence of either DNA fragmentation or mitochondrial cytochrome c release. Pretreatment with the selective caspase inhibitors either benzyloxycarbonyl-Val-Ala-Asp-fluoromethylketone (pan-caspase), N-acetyl-Leu-Glu-His-Asp-aldehyde (caspase 9) or N-acetyl--Glu-Thr-Asp-aldehyde (caspase 8), significantly increased L-glutamate-induced cell death with a corresponding increase in observed nuclear shrinkage and chromatin condensation. This enhancement of glutamate toxicity correlated with an increase in L-glutamate-dependent production of reactive  species (ROS) as a result of caspase inhibition. Pretreating the cells with N-acetyl-L-cysteine prevented ROS production, cell shrinkage and cell death from L-glutamate as well as that associated with the presence of the pan-caspase inhibitor. In contrast, the caspase-3/-7 inhibitor N-acetyl-Asp-Glu-Val-Asp aldehyde was without significant effect. However, pretreating the cells with the calpain inhibitor N-acetyl-Leu-Leu-Nle-CHO, but not the cathepsin B inhibitor CA-074, prevented cell death. The cytotoxic role of calpains was confirmed further by: 1) cytotoxic dependency on intracellular Ca(2+) increase, 2) increased cleavage of the calpain substrate Suc-Leu-Leu-Val--AMC and 3) immunoblot detection of the calpain-selective 145 kDa alpha-fodrin cleavage fragment. We conclude that oxidative L-glutamate toxicity in HT22 cells is mediated via calpain activation, whereas inhibition of caspases-8 and -9 may exacerbate L-glutamate-induced oxidative neuronal damage through increased oxidative stress.</p><h2><a href=https://www.ncbi.nlm.nih.gov/pubmed/11720974>Influence of a  interface on protein dynamics in a fungal lipase.</a></h2><p>Lipases catalyze lipolytic reactions and for optimal activity they require a  interface. To study the effect of a  aggregate on the behavior of the enzyme at the interfacial plane and how the aggregate influences an attached substrate or product molecule in time and space, we have performed molecular dynamics simulations. The simulations were performed over 1 to 2 ns using explicit SPC water. The interaction energies between protein and  are mainly due to van der Waals contributions reflecting the hydrophobic nature of the  molecules. Estimations of the protonation state of titratable residues indicated that the negative charge on the fatty acid is stabilized by interactions with the titratable residues -28, His-143, and His-257. In the presence of a  patch, the active site lid opens wider than observed in the corresponding simulations in an aqueous environment. In that lid conformation, the hydrophobic residues -85, -89, and Leu-92 are embedded in the  patch. The behavior of the substrate or product molecule is sensitive to the environment. Entering and leaving of substrate molecules could be observed in presence of the  patch, whereas the product forms strong hydrogen bonds with Ser-82, Ser-144, and Trp-88, suggesting that the formation of hydrogen bonds may be an important contribution to the mechanism by which product inhibition might take place.</p><h2><a href=https://www.ncbi.nlm.nih.gov/pubmed/10193575>Hydrogen peroxide-induced apoptosis in HL-60 cells requires caspase-3 activation.</a></h2><p>Apoptosis has been associated with oxidative stress in biological systems. Caspases have been considered to play a pivotal role in the execution phase of apoptosis. However, which caspases function as executioners in reactive  species (ROS)-induced apoptosis is not known. The present study was performed to identify the major caspases acting in ROS-induced apoptosis. Treatment of HL-60 cells with 50 microM hydrogen peroxide (H2O2) for 4 h induced the morphological changes such as condensed and/or fragmented nuclei, increase in caspase-3 subfamily protease activities, reduction of the procaspase-3 and a DNA fragmentation. To determine the role of caspases in H2O2-induced apoptosis, caspase inhibitors, acetyl--Val-Ala-Asp-chloromethyl ketone (Ac-YVAD-cmk), acetyl-Asp-Glu-Val-Asp-aldehyde (Ac-DEVD-CHO) and acetyl-Val-Glu--Asp-aldehyde (Ac-VEID-CHO), selective for caspase-1 subfamily, caspase-3 subfamily and caspase-6, respectively, were loaded into the cells using an osmotic lysis of pinosomes method. Of these caspase inhibitors, only Ac-DEVD-CHO completely blocked morphological changes, caspase-3 subfamily protease activation and DNA ladder formation in H2O2-treated HL-60 cells. This inhibitory effect was dose-dependent. These results suggest that caspase-3, but not caspase-1 is required for commitment to ROS-triggered apoptosis.</p><h2><a href=https://www.ncbi.nlm.nih.gov/pubmed/17718594>Modeling study of Rusticyanin-Cytochrome C(4) complex: an insight to possible H-bond mediated recognition and electron--transfer process.</a></h2><p>Rusticyanin (RCy) mediated transfer of electron to Cytochrome C(4) (Cytc(4)) from the extracellular Fe(+2) ion is primarily involved in the Thiobacillus ferrooxidans induced bio-leaching of pyrite ore and also in the metabolism of this acidophilic bacteria. The modeling studies have revealed the two possible mode of RCy-Cytc(4) complexation involving nearly the same stabilization energy approximately -15 x 10(3) kJ/mol, one through N-terminal Asp 15 and another -C terminal Glu 121 of Cytc(4) with the Cu-bonded His 143 of RCy. The Asp 15:His 143 associated complex (DH) of Cytc(4)-RCy was stabilized by the intermolecular H-bonds of the carboxyl  atoms O(delta1) and O(delta2) of Asp 15 with the Nepsilon-atom of His 143 and O(b) atoms of Ala 8 and Asp 5 (of Cytc(4)) with the Thr 146 and Phe 51 (of RCy). But the other Glu 121:His 143 associated complex (EH) of Cytc(4)-RCy was stabilized by the H-bonding interaction of the  atoms O(epsilon1) and O(epsilon2) of Glu 121 with the Nepsilon and Ogamma atoms of His 143 and Thr 146 of RCy. The six water molecules were present in the binding region of the two proteins in the energy minimized autosolvated DH and EH-complexes. The MD studies also revealed the presence of six interacting water molecules at the binding region between the two proteins in both the complexes. Several residues Gly 82 and 84, His 143 (RCy) were participated through the water mediated (W 389, W 430, W 413, W 431, W 373, and W 478) interaction with the Asp 15,  82, and 62,  63 (Cytc(4)) in DH complex, whereas in EH complex the Phe 51, Asn 80,  146 (RCy) residues were observed to interact with Asn 108, Met 120, Glu 121 (of Cytc(4)) through the water molecules W 507, W 445, W 401, W 446, and W 440. The direct water mediated (W 478) interaction of His 143 (RCy) to Asp 15 (of Cytc(4)) was observed only in the DH complex but not in EH. These direct and water mediated H-bonding between the two respective proteins and the binding free energy with higher interacting buried surface area of the DH complex compare to other EH complex have indicated an alternative possibility of the electron transfer route through the interaction of His 143 of RCy and the N-terminal Asp 15 of Cytc(4).</p><h2><a href=https://www.ncbi.nlm.nih.gov/pubmed/24425867>Oxicams bind in a novel mode to the cyclooxygenase active site via a two-water-mediated H-bonding Network.</a></h2><p>Oxicams are widely used nonsteroidal anti-inflammatory drugs (NSAIDs), but little is known about the molecular basis of the interaction with their target enzymes, the cyclooxygenases (COX). Isoxicam is a nonselective inhibitor of COX-1 and COX-2 whereas meloxicam displays some selectivity for COX-2. Here we report crystal complexes of COX-2 with isoxicam and meloxicam at 2.0 and 2.45 angstroms, respectively, and a crystal complex of COX-1 with meloxicam at 2.4 angstroms. These structures reveal that the oxicams bind to the active site of COX-2 using a binding pose not seen with other NSAIDs through two highly coordinated water molecules. The 4-hydroxyl group on the thiazine ring partners with Ser-530 via hydrogen bonding, and the heteroatom of the carboxamide ring of the oxicam scaffold interacts with -385 and Ser-530 through a highly coordinated water molecule. The nitrogen atom of the thiazine and the oxygen atom of the carboxamide bind to Arg-120 and -355 via another highly ordered water molecule. The rotation of Leu-531 in the structure opens a novel binding pocket, which is not utilized for the binding of other NSAIDs. In addition, a detailed study of meloxicam·COX-2 interactions revealed that mutation of Val-434 to  significantly reduces inhibition by meloxicam due to subtle changes around Phe-518, giving rise to the preferential inhibition of COX-2 over COX-1.</p><h2><a href=https://www.ncbi.nlm.nih.gov/pubmed/15453585>Antioxidant activity of peptides derived from egg white proteins by enzymatic hydrolysis.</a></h2><p>This work reports the antioxidant activity of peptides produced by enzymatic hydrolysis of crude egg white with pepsin. Four peptides included in the protein sequence of ovalbumin possessed radical scavenging activity higher than that of Trolox. The hydrolysate of egg white with pepsin for 3 h was previously found to exhibit a strong angiotensin I-converting enzyme (ACE) inhibitory activity in vitro. The combined antioxidant and ACE inhibition properties make it a very useful multifunctional preparation for the control of cardiovascular diseases, particularly hypertension. No correlation was found between antioxidant and ACE inhibitory activities. However, the peptide -Ala-Glu-Glu-Arg--Pro--Leu, which was a strong ACE inhibitor (50% inhibitory concentration, 4.7 microM) also exhibited a high radical scavenging activity (oxygen radical absorbance capacity-fluorescein value, 3.8 micromol of Trolox equivalent per micromol of peptide) and delayed the low-density lipoprotein  oxidation induced by Cu2+ at a concentration of approximately 0.16 mg/mg of low-density lipoprotein. Present results support that antioxidant peptides and amino acids not only act individually, but also cooperatively and synergistically.</p><h2><a href=https://www.ncbi.nlm.nih.gov/pubmed/22248593>Overexpression of a modified protein from amaranth seed in Escherichia coli and effect of environmental conditions on the protein expression.</a></h2><p>Amaranth seeds are considered as an excellent complementary source of food protein due to their balanced amino acid composition. Amarantin acidic subunit has the potential as a functional and nutraceutical protein, and it is structurally a good candidate for modification. The aim of this work was to improve its functionality, then the primary structure was modified into the third variable region of 11S globulins, by inserting antihypertensive peptides: four Val- in tandem and Arg--Pro-Pro in the C-terminal region. Modified protein was expressed in Escherichia coli Origami (DE3) and was purified. The culture conditions, including the culture media, temperature, agitation speed and air flow were tested in order to obtain an increased expression levels of the modified protein. A 2(3) factorial design was used for evaluate the effect of environmental conditions on modified protein production. The results indicated that the yield of modified protein could be increased by up 3-fold in bioreactor as compared with flask. In addition, the temperature, the agitation speed and the  were significant factors on the expression of the antihypertensive protein. The maximum production was 99 mg protein-L(-1). The hydrolyzed protein showed a high inhibitory activity of the angiotensin converting enzyme (IC50=0.047 mg mL(-1)).Copyright © 2012 Elsevier B.V. All rights reserved.</p><h2><a href=https://www.ncbi.nlm.nih.gov/pubmed/7547884>Crystal structure of a peptidyl pyridinium methyl ketone inhibitor with thrombin.</a></h2><p>The crystal structure of a complex between a bivalent peptidyl pyridinium methyl ketone inhibitor and human alpha-thrombin has been solved and refined at 2.0 A to an R factor of 0.18. The inhibitor, (D)cyclohexylalanine-Pro-Arg-(CH2N+C5H4CH2CO)-(Gly)4-Asp- -Glu-Pro--Pro-Glu-Glu-Ala-cyclo-hexylalanine-(D)Glu (coded P596), which forms a reversible covalent complex with thrombin, is highly potent with a Ki = 4.6 +/- 1.0 x 10(-14) M, lower than that of recombinant hirudin. The N-terminal, active-site-directed portion of the inhibitor is linked to the fibrinogen recognition exosite binding portion by a tetraglycine segment. The strong electron-withdrawing effect provided by the permanent positive charge on the pyridinium nitrogen makes the arginyl carbonyl carbon more susceptible to nucleophilic attack. In the crystal, a covalent P596-thrombin complex is observed. The electron density surrounding the active site portion and the pyridinium of the inhibitor is very well defined, clearly showing the existence of a covalent bond between the Ser195 O gamma and the now tetrahedral carbon of the inhibitor. The decreased binding ability of thrombin inhibitors containing N-terminal acetylation is discussed as is the effect of replacing the P3 (D)phenylalanine with (D)cyclohexylalanine. The electron density surrounding the remainder of the inhibitor is generally well defined, the exceptions being the C-terminal (D)Glu, the highly flexible tetraglycine linker, and some of the solvent-directed side chains.(ABSTRACT TRUNCATED AT 250 WORDS).</p><h2><a href=https://www.ncbi.nlm.nih.gov/pubmed/1518799>NMR docking of a substrate into the X-ray structure of staphylococcal nuclease.</a></h2><p>The conformation of the staphylococcal nuclease-bound metal-dTdA complex, previously determined by NMR methods [Weber, D.J., Mullen, G.P., Mildvan, A.S. (1991) Biochemistry 30:7425-7437] was docked into the X-ray structure of the enzyme-Ca(2+)-3',5'-pdTp complex [Loll, P.J., Lattman, E.E. (1989) Proteins: Struct., Funct., Genet. 5:183-201] by superimposing the metal ions, taking into account intermolecular nuclear Overhauser effects from assigned aromatic proton resonances of -85, -113, and -115 to proton resonances of the leaving dA moiety of dTdA, and energy minimization to relieve small overlaps. The proton resonances of the Phe, , and Trp residues of the enzyme in the ternary enzyme-La(3+)-dTdA complex were sequence specifically assigned by 2D phase-sensitive NOESY, with and without deuteration of the aromatic protons of the  residues, and by 2D heteronuclear multiple quantum correlation (HMQC) spectroscopy and 3D NOESY-HMQC spectroscopy with 15N labeling. While resonances of most Phe,  and Trp residues were unshifted by the substrate dTdA from those found in the enzyme-La(3+)-3',5'-pdTp complex and the enzyme-Ca(2+)-3',5'-pdTp complex, proton resonances of -85, -113, -115, and Phe-34 were shifted by 0.08 to 0.33 ppm and the 15N resonance of -113 was shifted by 2.1 ppm by the presence of substrate. The optimized position of enzyme-bound dTdA shows the 5'-dA leaving group to partially overlap the inhibitor, 3',5'-pdTp (in the X-ray structure). The 3'-TMP moiety of dTdA points toward the solvent in a channel defined by -18, Asp-19, Thr-22, Lys-45, and His-46. The phosphate of dTdA is coordinated by the metal, and an adjacent inner sphere water ligand is positioned to donate a hydrogen bond to the general base Glu-43 and to attack the phosphorus with inversion. Arg-35 and Arg-87 donate monodentate hydrogen bonds to different phosphate oxygens of dTdA, with Arg-87 positioned to protonate the leaving 5'- of dA, thus clarifying the mechanism of hydrolysis. Model building of an additional 5'-dGMP onto the 3'- of dA placed this third nucleotide onto a surface cleft near residues Glu-80, Asp-83, Lys-84, and -115 with its 3'-OH group accessible to the solvent, thus defining the size of the substrate binding site as accommodating a trinucleotide.</p><h2><a href=https://www.ncbi.nlm.nih.gov/pubmed/15565239>Structural scaffold of 18-crown-6 tetracarboxylic acid for optical resolution of chiral amino acid: X-ray crystal analyses and energy calculations of complexes of D- and L-isomers of tyrosine, isoleucine, methionine and phenylglycine.</a></h2><p>To clarify the structural scaffold of (+)-18-crown-6 tetracarboxylic acid ((+)-18C6H4) for the optical resolution of a chiral amino acid, the crystal structures of its equimolar complexes with L- and D-isomers of tyrosine (), isoleucine (), methionine (Met) and phenylglycine (PheG) were analysed by X-ray diffraction methods. (+)-18C6H4 took very similar conformations for all complexes. Although the chemical structure of (+)-18C6H4 is C2-symmetric, it took a similar asymmetric ring conformation of radius ca. 6.0 A. In all complexes, the amino group of chiral amino acids was located near the center of the ring and formed three hydrogen bonds and five electrostatic interactions with eight  atoms of the ether ring and carboxyl groups. Also, the Calpha atom of chiral amino acids participated in Calpha-H...O interaction with the  atom of (+)-18C6H4. In contrast, the carboxyl group of chiral amino acids did not directly interact with (+)-18C6H4. These results indicate that the structural scaffold of (+)-18C6H4 for the optical resolution of chiral amino acids is mainly based on the mode of interaction of (+)-18C6H4 with the amino and Calpha-H groups of chiral amino acids. The differences in interaction pattern and binding energy between the L- and D-isomers of each amino acid are discussed in relation to the chiral recognition of (+)-18C6H4.</p><h2><a href=https://www.ncbi.nlm.nih.gov/pubmed/12468553>Identification of residues critical for regulation of protein stability and the transactivation function of the hypoxia-inducible factor-1alpha by the von Hippel-Lindau tumor suppressor gene product.</a></h2><p>Under normoxic conditions the hypoxia-inducible factor-1alpha (HIF-1alpha) protein is targeted for degradation by the von Hippel-Lindau (pVHL) tumor suppressor protein acting as an E3 ubiquitin ligase. Binding of pVHL to HIF-1alpha is dependent on hydroxylation of specific proline residues by O(2)-dependent prolyl 4-hydroxylases. Upon exposure to hypoxia the hydroxylase activity is inhibited, resulting in stabilization of HIF-1alpha protein levels and activation of transcription of target genes. One of the two critical proline residues, Pro(563) in mouse HIF-1alpha, is located within a bifunctional domain, the N-terminal transactivation domain (N-TAD), which mediates both pVHL-dependent degradation at normoxia and transcriptional activation at hypoxia. Here we have identified two N-TAD residues, (564) and (565), which, in addition to Pro(563), were critical for pVHL-mediated degradation at normoxia. We have also identified D568A/D569A/D570A, F571A, and L573A as mutations of the N-TAD that abrogated binding to pVHL both in vitro and in vivo, and constitutively stabilized N-TAD against degradation. Moreover, the mutations Y564G, L556A/L558A, and F571A/L573A drastically reduced the transactivation function of either the isolated N-TAD or full-length HIF-1alpha in hypoxic cells. Interestingly, the P563A mutant exhibited a constitutively active and potent transactivation function that was enhanced by functional interaction with the transcriptional coactivator protein CREB-binding protein. In conclusion, we have identified by mutation analysis several residues that are critical for either one or both of the interdigitated and conditionally regulated degradation and transactivation functions of the N-TAD of HIF-1alpha.</p><h2><a href=https://www.ncbi.nlm.nih.gov/pubmed/22259083>Differential effects of delta and epsilon protein kinase C in modulation of postischemic cerebral blood flow.</a></h2><p></p><h2><a href=https://www.ncbi.nlm.nih.gov/pubmed/17705494>Genetic engineering of the heme pocket in human serum albumin: modulation of O2 binding of iron protoporphyrin IX by variation of distal amino acids.</a></h2><p>Complexing an iron protoporphyrin IX into a genetically engineered heme pocket of recombinant human serum albumin (rHSA) generates an artificial hemoprotein, which can bind O2 in much the same way as hemoglobin (Hb). We previously demonstrated a pair of mutations that are required to enable the prosthetic heme group to bind O2 reversibly: (i) -142-->His, which is axially coordinated to the central Fe2+ ion of the heme, and (ii) -161-->Phe or Leu, which makes the sixth coordinate position available for ligand interactions [I142H/Y161F (HF) or I142H/Y161L (HL)]. Here we describe additional new mutations designed to manipulate the architecture of the heme pocket in rHSA-heme complexes by specifically altering distal amino acids. We show that introduction of a third mutation on the distal side of the heme (at position Leu-185, Leu-182, or Arg-186) can modulate the O2 binding equilibrium. The coordination structures and ligand (O2 and CO) binding properties of nine rHSA(triple mutant)-heme complexes have been physicochemically and kinetically characterized. Several substitutions were severely detrimental to O2 binding: for example, Gln-185, His-185, and His-182 all generated a weak six-coordinate heme, while the rHSA(HF/R186H)-heme complex possessed a typical bis-histidyl hemochrome that was immediately autoxidized by O2. In marked contrast, HSA(HL/L185N)-heme showed very high O2 binding affinity (P1/2O2 1 Torr, 22 degrees C), which is 18-fold greater than that of the original double mutant rHSA(HL)-heme and very close to the affinities exhibited by myoglobin and the high-affinity form of Hb. Introduction of Asn at position 185 enhances O2 binding primarily by reducing the O2 dissociation rate constant. Replacement of polar Arg-186 with Leu or Phe increased the hydrophobicity of the distal environment, yielded a complex with reduced O2 binding affinity (P1/2O2 9-10 Torr, 22 degrees C), which nevertheless is almost the same as that of human red blood cells and therefore better tuned to a role in O2 transport.</p><h2><a href=https://www.ncbi.nlm.nih.gov/pubmed/10049310>Structural dynamics of ligand diffusion in the protein matrix: A study on a new myoglobin mutant Y(B10) Q(E7) R(E10).</a></h2><p>A triple mutant of sperm whale myoglobin (Mb) [Leu(B10) --> , His(E7) --> Gln, and Thr(E10) --> Arg, called Mb-YQR], investigated by stopped-flow, laser photolysis, crystallography, and molecular dynamics (MD) simulations, proved to be quite unusual. Rebinding of photodissociated NO, O2, and CO from within the protein (in a "geminate" mode) allows us to reach general conclusions about dynamics and cavities in proteins. The 3D structure of oxy Mb-YQR shows that bound O2 makes two H-bonds with (B10)29 and Gln(E7)64; on deoxygenation, these two residues move toward the space occupied by O2. The bimolecular rate constant for NO binding is the same as for wild-type, but those for CO and O2 binding are reduced 10-fold. While there is no geminate recombination with O2 and CO, geminate rebinding of NO displays an unusually large and very slow component, which is pretty much abolished in the presence of xenon. These results and MD simulations suggest that the ligand migrates in the protein matrix to a major "secondary site," located beneath (B10)29 and accessible via the motion of (G8)107; this site is different from the "primary site" identified by others who investigated the photolyzed state of wild-type Mb by crystallography. Our hypothesis may rationalize the O2 binding properties of Mb-YQR, and more generally to propose a mechanism of control of ligand binding and dissociation in hemeproteins based on the dynamics of side chains that may (or may not) allow access to and direct temporary sequestration of the dissociated ligand in a docking site within the protein. This interpretation suggests that very fast (picosecond) fluctuations of amino acid side chains may play a crucial role in controlling O2 delivery to tissue at a rate compatible with physiology.</p><h2><a href=https://www.ncbi.nlm.nih.gov/pubmed/16277537>O2 and CO binding properties of artificial hemoproteins formed by complexing iron protoporphyrin IX with human serum albumin mutants.</a></h2><p>The binding properties of O2 and CO to recombinant human serum albumin (rHSA) mutants with a prosthetic heme group have been physicochemically and kinetically characterized. Iron(III) protoporphyrin IX (hemin) is bound in subdomain IB of wild-type rHSA [rHSA(wt)] with weak axial coordination by -161. The reduced ferrous rHSA(wt)-heme under an Ar atmosphere exists in an unusual mixture of four- and five-coordinate complexes and is immediately autoxidized by O2. To confer O2 binding capability on this naturally occurring hemoprotein, a proximal histidine was introduced into position -142 or Leu-185 by site-directed mutagenesis. A single mutant (I142H) and three double mutants (I142H/Y161L, I142H/Y161F, and Y161L/L185H) were prepared. Both rHSA(I142H/Y161L)-heme and rHSA(I142H/Y161F)-heme formed ferrous five-N-coordinate high-spin complexes with axial ligation of His-142 under an Ar atmosphere. These artificial hemoproteins bind O2 at room temperature. Mutation at the other side of the porphyrin, Y161L/L185H, also allowed O2 binding to the heme. In contrast, the single mutant rHSA(I142H)-heme could not bind O2, suggesting that removal of Y161 is necessary to confer reversible O2 binding. Laser flash photolysis experiments showed that the kinetics of CO recombination with the rHSA(mutant)-heme were biphasic, whereas O2 rebinding exhibited monophasic kinetics. This could be due to the two different geometries of the axial imidazole coordination arising from the two orientations of the porphyrin plane in the heme pocket. The O2 binding affinities of the rHSA(mutant)-heme were significantly lower than those of hemoglobin and myoglobin, principally due to the high O2 dissociation rates. Changing Leu-161 to Phe-161 at the distal side increased the association rates of both O2 and CO, which resulted in enhanced binding affinity.</p><h2><a href=https://www.ncbi.nlm.nih.gov/pubmed/11500966>Statistical analysis of atomic contacts at RNA-protein interfaces.</a></h2><p>Forty-five crystals of complexes between proteins and RNA molecules from the Protein Data Bank have been statistically surveyed for the number of contacts between RNA components (phosphate, ribose and the four bases) and amino acid side chains. Three groups of complexes were defined: the tRNA synthetases; the ribosomal complexes; and a third group containing a variety of complexes. The types of atomic contacts were a priori classified into ionic, neutral H-bond, C-H...O H-bond, or van der Waals interaction. All the contacts were organized into a relational database which allows for statistical analysis. The main conclusions are the following: (i) in all three groups of complexes, the most preferred amino acids (Arg, Asn, Ser, Lys) and the less preferred ones (Ala, , Leu, Val) are the same; Trp and Cys are rarely observed (respectively 15 and 5 amino acids in the ensemble of interfaces); (ii) of the total number of amino acids located at the interfaces 22% are hydrophobic, 40% charged (positive 32%, negative 8%), 30% polar and 8% are Gly; (iii) in ribosomal complexes, phosphate is preferred over ribose, which is preferred over the bases, but there is no significant preference in the other two groups; (iv) there is no significant prevalence of a base type at protein-RNA interfaces, but specifically Arg and Lys display a preference for phosphate over ribose and bases; Pro and Asn prefer bases over ribose and phosphate; Met, Phe and  prefer ribose over phosphate and bases. Further, , Pro, Ser prefer A over the others; Leu prefers C; Asp and Gly prefer G; and Asn prefers U. Considering the contact types, the following conclusions could be drawn: (i) 23% of the contacts are via potential H-bonds (including CH...O H-bonds and ionic interactions), 72% belong to van der Waals interactions and 5% are considered as short contacts; (ii) of all potential H-bonds, 54% are standard, 33% are of the C-H...O type and 13% are ionic; (iii) the Watson-Crick sites of G, O6(G) and principally N2(G) and the hydroxyl group O2' is more often involved in H-bonds than expected; the protein main chain is involved in 32% and the side chains in 68% of the H-bonds; considering the neutral and ionic H-bonds, the following couples are more frequent than expected-base A-Ser, base G-Asp/Glu, base U-Asn. The RNA CH groups interact preferentially with  atoms (62% on the main chain and 19% on the side chains); (iv) the bases are involved in 38% of all H-bonds and more than 26% of the H-bonds have the H donor group on the RNA; (v) the atom O2' is involved in 21% of all H-bonds, a number greater than expected; (vi) amino acids less frequently in direct contact with RNA components interact frequently via their main chain atoms through water molecules with RNA atoms; in contrast, those frequently observed in direct contact, except Ser, use instead their side chain atoms for water bridging interactions.Copyright 2001 John Wiley & Sons, Ltd.</p><h2><a href=https://www.ncbi.nlm.nih.gov/pubmed/25354707>Permethrin and its metabolites affect Cu/Zn superoxide conformation: fluorescence and in silico evidences.</a></h2><p>The proclivity of permethrin and its metabolites to affect the structure and activity of Cu/Zn superoxide dismutase (SOD) has been investigated by using intrinsic fluorescence and 8-ANS fluorescence techniques. In silico molecular docking investigations were carried out in order to assess the means of interaction at a molecular level between SOD and the considered ligands. Results show that both, permethrin and its metabolites are able to induce conformational variation on SOD. Permethrin and 3-phenoxybenzyl alcohol metabolite induce a blue shift toward the hydrophobic amino acids Leu-101, -102, Leu-104, -110 and -111, with a significant peak increase. An opposite effect was shown by 3-phenoxy benzaldehyde and 3-phenoxybenzoic acid with a progressive reduction of tyrosine fluorescence emission, without any shift. Computational findings confirm that all the molecules considered have more than one allosteric binding site but none of them interact with SOD at its catalytic Cu/Zn cleft. Moreover, all the binding poses found are very close in binding energy thus demonstrating that there is not only a preferred interaction site but most of them are important due to their relative energy in equilibrium with a population strictly connected to the ligand concentration. In the obtained complexes, all the ligands are involved in many hydrogen bonds through their polar oxygen moieties but due to the presence of a common aromatic hydrophobic core, many hydrophobic interactions are due to the SOD nature rich in apolar amino acids. Furthermore, for each ligand it can be pointed out the presence of a highly populated docked structure with a specific interaction of permethrin and its metabolites with -108, responsible for changes in fluorescence emission.</p><h2><a href=https://www.ncbi.nlm.nih.gov/pubmed/17998746>Evaluation of titanium and titanium oxides as chemo-affinity sorbents for the selective enrichment of organic phosphates.</a></h2><p>Titanium (Ti) and TiO, Ti2O3, Ti3O5 and TiO(1.98) as well as TiO2 have been evaluated as chemo-affinity sorbents for the selective enrichment of organic phosphates. A column-switching high-performance liquid chromatography (HPLC) system, constructed with a precolumn (4.6 mm i.d. x 10 mm) packed with the titanium sorbents, an anion-exchange analytical column and a UV detector (215 nm) was used. When an aqueous 0.015% trifluoroacetic acid (TFA) was used as a sample-loading solution, O-phospho-L-tyrosine (P-), phenyl phosphate and phenylphosphonic acid were adsorbed onto all of the titanium sorbents with recoveries of 60.9 - 102.9%. Some acidic compounds other than phosphates, such as benzenedicarboxylic acid (BDA) isomers, were also adsorbed onto all of the titanium sorbents. To improve the selectivity to organic phosphates, various sample-loading solutions were examined using a Ti precolumn, two phosphorylated peptides [-Ser(p)-Val-Arg (PP1) and Gln--Ser(p)-Val-Arg (PP2)], P-, BDA isomers and diglutamic acid (Glu-Glu) as test compounds. Among the sample-loading solutions tested, such as TFA, HClO4, organic acids, boric acid and NaCl, the use of 100 mM NaCl in 10 mM boric acid was found to be effective. The recoveries of PP1, PP2 and P- were 73.0, 88.3 and 71.5%, respectively, whereas those of Glu-Glu and BDAs were suppressed to only below 10%.</p><h2><a href=https://www.ncbi.nlm.nih.gov/pubmed/12071346>Lysozyme modification by the fenton reaction and gamma radiation.</a></h2><p>A comparative study was performed on lysozyme modification after exposure to Fenton reagent (Fe(II)/H2 O2) or hydroxyl radicals produced by y radiation. The conditions were adjusted to obtain, with both systems, a 50% loss of activity of the modified ensemble. Gamma radiation modified almost all types of amino acid residues in the enzyme, with little specificity. The modification order was  > Met = Cys > Lys >  + Leu > Gly > Pro = Phe > Thr + Ala > Trp = Ser > Arg > Asp + Glu, with 42 mol of modified residues per initial mole of native enzyme. In contrast, when the enzyme was exposed to the Fenton reaction, only some types of amino acids were modified. Furthermore, a smaller number of residues (13.5) were damaged per initial mole of enzyme. The order of the modified residues was  > Cys > Trp > Met His >  + Leu > Val > Arg. These results demonstrate that the modifications elicited by these two free radical sources follow different mechanisms. An intramolecular free radical chain reaction is proposed to play a dominant role in the oxidative modification of the protein promoted by gamma radiation.</p><h2><a href=https://www.ncbi.nlm.nih.gov/pubmed/10510718>A study on growth characteristics and nutrient consumption of Lactobacillus plantarum in A-stat culture.</a></h2><p>Lactobacillus plantarum was grown in complex media containing glucose and yeast extract. The maximum growth yield based on yeast extract consumption was 0.5 g dwt g-1. Growth yield YATP 15-17 g dwt mol ATP-1 was almost constant in the glucose limited A-stat experiment whereas in the yeast extract limited culture it increased with dilution rate. The maximum specific growth rate observed, 0.5 h-1, was similar for both A-stat and batch cultures. Specific  consumption, QO2, reached the value of 1.8 mmol O2 h-1 g dwt-1. It was shown that Val, , Leu,  and Phe, were consumed mainly as free amino acids, while Asp, Pro, Lys and Arg were derived from peptides. Significantly more Asp, Ser, Glu, Val, , Leu and Phe were consumed than needed to build up cell protein whereas some Pro, Gly, Ala and Lys was synthesized. A network of metabolic reactions in L. plantarum was proposed on the basis of the experimental data.</p><h2><a href=https://www.ncbi.nlm.nih.gov/pubmed/9606940>Specific interactions of bovine and human beta-casomorphin-7 with Cu(II) ions.</a></h2><p>Complex formation between Cu(II) and human and bovine beta-casomorphin heptapeptides, -Pro-Phe-Val-Glu-Pro- and -Pro-Phe-Pro-Gly-Pro-, respectively, was investigated by pH potentiometry and spectroscopic (CD, EPR and electronic absorption) techniques. The results showed the critical impact of Pro residues on the complex equilibria formed. The presence of the Pro residue at the second position leads to formation of very stable dimeric species in which two metal ions co-ordinate to N-terminal ¿NH2, C=O¿ binding sites of one peptide molecule and the deprotonated phenolic  of the second ligand molecule. The presence of two additional hydrophobic residues on the C-terminal makes heptapeptide molecule much more effective ligand than its pentapeptide N-terminal fragment.</p><h2><a href=https://www.ncbi.nlm.nih.gov/pubmed/7479707>Phe-46(CD4) orients the distal histidine for hydrogen bonding to bound ligands in sperm whale myoglobin.</a></h2><p>The role of Phe-46(CD4) in modulating the functional properties of sperm whale myoglobin was investigated by replacing this residue with Leu, , Val, Ala, Trp, , and Glu. This highly conserved amino acid almost makes direct contact with the distal histidine and has been postulated to affect ligand binding. The overall association rate constants for CO, O2, and NO binding were little affected by decreasing the size of residue 46 step-wise from Phe to Leu to Val to Ala. In contrast, the rates of CO, O2, and NO dissociation increased 4-, 10-, and 25-fold, respectively, for the same series of mutants, causing large decreases in the affinity of myoglobin for all three diatomic gases. The rates of autooxidation at 37 degrees C, pH 7.0 increased dramatically from approximately 0.1-0.3 h-1 for wild-type, -46, and Trp-46 myoglobins to 1.5, 5.2, 4.9, and 5.0 h-1 for the Leu-46, -46, Val-46 and Ala-46 mutants, respectively. Rates of NO and O2 geminate recombination were measured using 35 ps and 9 ns laser excitation pulses. Decreasing the size of residue 46 causes significant decreases in the extent of both picosecond and nanosecond rebinding processes. High resolution structures of Leu-46 and Val-46 metmyoglobins, Val-46 CO-myoglobin, and Val-46 deoxymyoglobin were determined by X-ray crystallography. When Phe-46 is replaced by Val, the loss of internal packing volume is compensated by (1) contraction of the CD corner toward the core of the protein, (2) movement of the E-helix toward the mutation site, (3) greater exposure of the distal pocket to intruding solvent molecules, and (4) large disorder in the position of the side chain of the distal histidine (His-64). In wild-type myoglobin, the van der Waals contact between C zeta of Phe-46 and C beta of His-64 appears to restrict rotation of the imidazole side chain. Insertion of Val at position 46 relieves this steric restriction, allowing the imidazole side chain to rotate about the C alpha - C beta bond toward the surface of the globin and about the C beta - C gamma bond toward the space previously occupied by the native Phe-46 side chain. This movement disrupts hydrogen bonding with bound ligands, causing significant decreases in affinity, and opens the distal pocket to solvent water molecules, causing marked increases in the rate of autooxidation.(ABSTRACT TRUNCATED AT 400 WORDS).</p><h2><a href=https://www.ncbi.nlm.nih.gov/pubmed/10360941>Crystal structure of macrophage migration inhibitory factor complexed with (E)-2-fluoro-p-hydroxycinnamate at 1.8 A resolution: implications for enzymatic catalysis and inhibition.</a></h2><p>Macrophage migration inhibitory factor (MIF) exhibits dual activities. It acts as an immunoregulatory protein as well as a phenylpyruvate tautomerase. To understand better the relationship between these two activities and to elucidate the structural basis for the enzymatic activity, a crystal structure of a complex between murine MIF and (E)-2-fluoro-p-hydroxycinnamate, a competitive inhibitor of the tautomerase activity, has been determined to 1.8 A resolution. The structure is nearly superimposable on that of the free protein indicating that the presence of the inhibitor does not result in any major structural changes. The inhibitor also confirms the location of the active site in a hydrophobic cavity containing the amino-terminal proline. Within this cavity, the inhibitor interacts with residues from adjacent subunits. At the back of the cavity, the side-chain carbonyl  of Asn-97' interacts with the phenolic hydroxyl group of the inhibitor while at the mouth of the cavity the ammonium group of Lys-32 interacts with a carboxylate . The other carboxylate  of the inhibitor interacts with Pro-1. The hydroxyl group of -95' interacts weakly with the fluoro group on the inhibitor. The hydrophobic side chains of five active-site residues (Met-2, -64, Met-101, Val-106, and Phe-113) and the phenyl moiety of -95' are responsible for the binding of the phenyl group. Further insight into the enzymatic activity of MIF was obtained by carrying out kinetic studies using the enol isomers of phenylpyruvate and (p-hydroxyphenyl)pyruvate. The results demonstrate that MIF processes the enol isomers more efficiently than the keto isomers primarily because of a decrease in Km. On the basis of these results, a mechanism is proposed for the MIF-catalyzed tautomerization reaction.</p><h2><a href=https://www.ncbi.nlm.nih.gov/pubmed/24439548>Effect of reverse photoperiod on in vitro regeneration and piperine production in Piper nigrum L.</a></h2><p>In this study, a novel approach for in vitro regeneration of Piper nigrum L. has been applied in order to increase healthy biomass, phytochemicals and piperine production via reverse photoperiod (16hD/8hL). Leaf portions of the seed-derived plants were placed on an MS-medium fortified with different PGRs. Under 16hD/8hL, thidiazuron (TDZ; 4.0 mg L⁻¹) and BA (1.5 mg L⁻¹) was found to be the most effective (<90%) in callus induction. Two concentrations (1.5, 2.0 mg L⁻¹) of the IBA produced>80% shoots from callus cultures. Healthy shoots were transferred to rooting medium and higher percentage of rooting (<90%) was observed on IBA (1.5 mg L⁻¹). These in vitro tissues were subjected to amino acid analysis, spectrophotometry, and HPLC. ARG, SER, THR, and  were the most abundant components out of 17 amino acids. Higher amino acid production was observed under normal photoperiod (16hL/8hD) than under reverse photoperiod (16hD/8hL). The highest total phenolic content (TPC; 9.91 mg/g-DW) and flavonoid content (7.38 mg/g-DW) were observed in callus cultures incubated under 16hL/8hD than other tissues incubated under 16hD/8hL photoperiod. Higher DPPH and PoMo activities were observed in tissues incubated under 16hL/8hD photoperiod, while ABTS and Fe²⁺ chelating activities were found higher in tissues incubated under reverse photoperiod. Significant quantities of piperine content were observed in all tissues except callus cultures. These results suggest that reverse photoperiod is a promising approach for callus induction, phytochemicals and piperine production for commercial applications.Copyright © 2013 Académie des sciences. Published by Elsevier SAS. All rights reserved.</p><h2><a href=https://www.ncbi.nlm.nih.gov/pubmed/19745556>Structural and mutagenic approach to create human serum albumin-based  carrier and photosensitizer.</a></h2><p>Human serum albumin (HSA) is a versatile protein found at high concentration in blood plasma and binds a range of insoluble endogenous and exogenous compounds. We have shown that complexation of functional molecules into HSA creates unique proteins never seen in nature. Complexing an iron-protoporphyrin IX into a genetically engineered heme pocket of recombinant HSA (rHSA) generates an artificial hemoprotein, which binds O(2) reversibly in much the same way as hemoglobin. A pair of site-specific mutations, (i) introduction of a proximal histidine at the -142 position and (ii) substitution of -161 with Phe or Leu, allows the heme to bind O(2). Additional modification on the distal side of the heme pocket provides rHSA(triple mutant)-heme complexes with a variety of O(2) binding affinity. Complexing a carboxy-C(60)-fullerene (CF) into HSA generates a protein photosensitizer for photodynamic cancer therapy. Energy transfer occurs from a photoexcited triplet-state of HSA-CF (HSA-(3)CF(*)) to O(2), forming singlet  ((1)O(2)). This protein does not show dark cytotoxicity, but induceds cell death under visible light irradiation.</p><h2><a href=https://www.ncbi.nlm.nih.gov/pubmed/22084238>Intramembrane proton binding site linked to activation of bacterial pentameric ion channel.</a></h2><p>Prokaryotic orthologs of eukaryotic Cys-loop receptor channels recently emerged as structural and mechanistic surrogates to investigate this superfamily of intercellular signaling proteins. Here, we examine proton activation of the prokaryotic ortholog GLIC using patch clamp electrophysiology, mutagenesis, and molecular dynamics (MD) simulations. Whole-cell current recordings from human embryonic kidney (HEK) 293 cells expressing GLIC show half-maximal activation at pH 6, close to the pK(a) of histidine, implicating the three native His residues in proton sensing linked to activation. The mutation H235F abolishes proton activation, H277Y is without effect, and all nine mutations of His-127 prevent expression on the cell surface. In the GLIC crystal structure, His-235 on transmembrane (TM) α-helix 2, hydrogen bonds to the main chain carbonyl  of -259 on TM α-helix 3. MD simulations show that when His-235 is protonated, the hydrogen bond persists, and the channel remains in the open conformation, whereas when His-235 is deprotonated, the hydrogen bond dissociates, and the channel closes. Mutations of the proximal -263, which also links TM α-helices 2 and 3 via a hydrogen bond, alter proton sensitivity over a 1.5 pH unit range. MD simulations show that mutations of -263 alter the hydrogen bonding capacity of His-235. The overall findings show that His-235 in the TM region of GLIC is a novel proton binding site linked to channel activation.</p><h2><a href=https://www.ncbi.nlm.nih.gov/pubmed/30486436>Two Novel Multi-Functional Peptides from Meat and Visceral Mass of Marine Snail  and Their Activities In Vitro and In Vivo.</a></h2><p> () is a marine snail with high nutritional and commercial value; however, little is known about its active peptides. In this study, two multi-functional peptides, YSQLENEFDR (-Ser-Gln-Leu-Glu-Asn-Glu-Phe-Asp-Arg) and YIAEDAER (--Ala-Glu-Asp-Ala-Glu-Arg), were isolated and purified from meat and visceral mass extracts of  using a multi-bioassay-guided method and were characterized by using liquid chromatography-tandem mass spectrometry. Both peptides showed high antioxidant, angiotensin-converting enzyme (ACE)-inhibitory, and anti-diabetic activities, with half-maximal effective concentrations values less than 1 mM. Antioxidant and ACE-inhibitory activities were significantly higher for YSQLENEFDR than for YIAEDAER. In a zebrafish model, the two peptides exhibited strong scavenging ability for reactive  species and effectively protected skin cells against oxidative damage without toxicity. Molecular docking simulation further predicted the interactions of the two peptides and ACE. Stability analysis study indicated that the two synthetic peptides maintained their activities under thermal stress and simulated gastrointestinal digestion conditions. The low molecular weight, high proportion of hydrophobic and negatively-charged amino acids, and specific C-terminal and N-terminal amino acids may contribute to the observed bio-activities of these two peptides with potential application for the prevention of chronic noncommunicable diseases.</p><h2><a href=https://www.ncbi.nlm.nih.gov/pubmed/31435933>Evaluation of antioxidant peptides generated from Jiuzao (residue after Baijiu distillation) protein hydrolysates and their effect of enhancing healthy value of Chinese Baijiu.</a></h2><p>Jiuzao is the residue after Bajiu distillation which is usually used as forage for livestock. However, it is not fully utilized yet considering the content of protein remained. The present study aimed to isolate antioxidant peptides from Jiuzao protein hydrolysates, then add these peptides into Baijiu product to enhance the healthy value of Baiju. Meanwhile environmental pollution caused by massive Jiuzao can be mitigated indirectly.Four peptides Ala--(Leu) (AYI(L)) and Asp-Arg-Glu-(Leu) (DREI(L)) were identified from Jiuzao protein hydrolysates, the extraction contents of AYI + AYL and DREI + DREL were 896.10 and 110.51 mg kg Jiuzao, respectively. On the one hand, antioxidant activities of these peptides were investigated. For in vitro antioxidant assays, AYI, AYL and DREI exhibited strong capacities in  radical absorbance capacity (ORAC) assay. Furthermore, three levels of four peptides were assessed by 2,2'-azobis(2-methylpropanimidamidine) (AAPH)-induced HepG2 cells model. The results showed that these peptides exerted a degree of antioxidant activities in cells. Meanwhile, selected peptides concentrations according to cell assays remined at effective doses after in vitro digestion. On the other hand, the influence of these four peptides on the characteristic aroma compounds in Baijiu was studied. Most characteristic aroma compounds releases were increased with the addition of peptides.In the study, antioxidant activities of peptides were evaluated, the feasibility of utilizing Jiuzao protein hydrolysates to obtain beneficial peptides was also proved. Healthy effect of Baijiu or other food can be increased by adding these functional substances. The findings might contribute to food application and Baijiu industries. © 2019 Society of Chemical Industry.© 2019 Society of Chemical Industry.</p><h2><a href=https://www.ncbi.nlm.nih.gov/pubmed/21714509>Determinants of the heme-CO vibrational modes in the H-NOX family.</a></h2><p>The Heme Nitric oxide/ binding (H-NOX) family of proteins have important functions in gaseous ligand signaling in organisms from bacteria to humans, including nitric oxide (NO) sensing in mammals, and provide a model system for probing ligand selectivity in hemoproteins. A unique vibrational feature that is ubiquitous throughout the H-NOX family is the presence of a high C-O stretching frequency. To investigate the cause of this spectroscopic characteristic, the Fe-CO and C-O stretching frequencies were probed in the H-NOX domain from Thermoanaerobacter tengcongensis (Tt H-NOX) using resonance Raman (RR) spectroscopy. Four classes of heme pocket mutants were generated to assess the changes in stretching frequency: (i) the distal H-bonding network, (ii) the proximal histidine ligand, (iii) modulation of the heme conformation via -5 and Pro-115, and (iv) the conserved -Ser-Arg (YxSxR) motif. These mutations revealed important electrostatic interactions that dampen the back-donation of the Fe(II) d(π) electrons into the CO π* orbitals. The most significant change occurred upon disruption of the H-bonds between the strictly conserved YxSxR motif and the heme propionate groups, producing two dominant CO-bound heme conformations. One conformer was structurally similar to Tt H-NOX WT, whereas the other displayed a decrease in ν(C-O) of up to ∼70 cm(-1) relative to the WT protein, with minimal changes in ν(Fe-CO). Taken together, these results show that the electrostatic interactions in the Tt H-NOX binding pocket are primarily responsible for the high ν(C-O) by decreasing the Fe d(π) → CO π* back-donation and suggest that the dominant mechanism by which this family modulates the Fe(II)-CO bond likely involves the YxSxR motif.</p><h2><a href=https://www.ncbi.nlm.nih.gov/pubmed/26342079>Mapping Key Residues of ISD11 Critical for NFS1-ISD11 Subcomplex Stability: IMPLICATIONS IN THE DEVELOPMENT OF MITOCHONDRIAL DISORDER, COXPD19.</a></h2><p>Biogenesis of the iron-sulfur (Fe-S) cluster is an indispensable process in living cells. In mammalian mitochondria, the initial step of the Fe-S cluster assembly process is assisted by the NFS1-ISD11 complex, which delivers sulfur to scaffold protein ISCU during Fe-S cluster synthesis. Although ISD11 is an essential protein, its cellular role in Fe-S cluster biogenesis is still not defined. Our study maps the important ISD11 amino acid residues belonging to putative helix 1 (Phe-40), helix 3 (Leu-63, Arg-68, Gln-69, -72, -76), and C-terminal segment (Leu-81, Glu-84) are critical for in vivo Fe-S cluster biogenesis. Importantly, mutation of these conserved ISD11 residues into alanine leads to its compromised interaction with NFS1, resulting in reduced stability and enhanced aggregation of NFS1 in the mitochondria. Due to altered interaction with ISD11 mutants, the levels of NFS1 and Isu1 were significantly depleted, which affects Fe-S cluster biosynthesis, leading to reduced electron transport chain complex (ETC) activity and mitochondrial respiration. In humans, a clinically relevant ISD11 mutation (R68L) has been associated in the development of a mitochondrial genetic disorder, COXPD19. Our findings highlight that the ISD11 R68A/R68L mutation display reduced affinity to form a stable subcomplex with NFS1, and thereby fails to prevent NFS1 aggregation resulting in impairment of the Fe-S cluster biogenesis. The prime affected machinery is the ETC complex, which showed compromised redox properties, causing diminished mitochondrial respiration. Furthermore, the R68L ISD11 mutant displayed accumulation of mitochondrial iron and reactive oxygen species, leading to mitochondrial dysfunction, which correlates with the phenotype observed in COXPD19 patients.© 2015 by The American Society for Biochemistry and Molecular Biology, Inc.</p><h2><a href=https://www.ncbi.nlm.nih.gov/pubmed/7700867>Molecular dynamics simulations of HIV-1 protease with peptide substrate.</a></h2><p>Molecular dynamics simulations of human immunodeficiency virus (HIV)-1 protease with a model substrate were used to test if there is a stable energy minimum for a proton that is equidistant from the four delta  atoms of the two catalytic aspartic acids. The crystal structure of HIV-1 protease with a peptidic inhibitor was modified to model the peptide substrate Ser-Gln-Asn--Pro--Val-Gln for the starting geometry. A proton was positioned between the two closet  atoms of the two catalytic aspartic acids, and close to the carbonyl  of the scissile bond in the substrate. All crystallographic water molecules were included. Two molecular dynamics simulations were run: 30 ps with united-atom potentials and 40 ps using the more accurate all-atom potentials. The molecular dynamics used a new algorithm that increased the speed and allowed the elimination of a cut-off for non-bonded interactions and the inclusion of an 8 A shell of water molecules in the calculations. The overall structure of the protease dimer, including the catalytic aspartic acids, was stable during the course of the molecular dynamics simulations. The substrate and a water molecule, that is an important component of the binding site, were stable during the simulation using all-atom potentials, but more mobile when united-atom potentials were used. A Poincare map representation showed that the positions of the proton and its coordinating  atoms were stable for 93% of both simulations, although many of the buried and poorly accessible water molecules exchanged with solvent. The proton has a stable minimum energy position and maintains coordination with all four delta  atoms of the two catalytic aspartic acids and the carbonyl  of the scissile bond of the substrate. Therefore, a loosely bound hydrogen ion at this position will not be rapidly exchanged with solvent, and will rebond to either a catalytic aspartic acid or possibly the substrate. The implications for the reaction mechanism are discussed.</p><h2><a href=https://www.ncbi.nlm.nih.gov/pubmed/8373357>Use of 18O-labelled leucine and phenylalanine to measure protein turnover in muscle cell cultures and possible futile cycling during aminoacylation.</a></h2><p>Amino acids labelled with 18(O) on both carboxy  atoms have the potential for use as non-recyclable tracers to measure protein turnover. During protein synthesis one of the labelled  atoms is removed, and thus release of the mono-labelled amino acid could be used to determine proteolysis. Primary cultures of embryonic-chick skeletal-muscle cells were used to test the use of 18(O2)-labelled Leu to measure proteolysis. For 9-day cultures, prelabelled on days 2-8 with medium containing one-half the Leu as [18O2]Leu and one-half as [2H3]Leu, release of [18(O)]Leu was less than 50% that of [2H]Leu over 24 h, suggesting a loss of the 18O label by a mechanism other than protein synthesis. Medium containing [18(O2)]Leu, [2H3]Leu, [18O2]Phe and [13C]Phe was then incubated with 9-day cultures to compare the rate of loss of the 18(O)-label from Leu and Phe with the rate of uptake of the non-carboxy--labelled amino acids. Results for Leu demonstrated an 81% loss of the 18(O) label compared with a 33% decrease in [2H]Leu over 12 h. Loss of the 18(O) label was four times as great for Leu as for Phe. Loss of the 18(O) label was not decreased by addition of cycloheximide or by addition of a 3-fold excess of , Val and ; thus the loss of label was not due to protein synthesis alone or to misbinding to incorrect tRNAs. Infusion of the isotopes into pigs showed that the 18(O) label of Leu was not lost during transamination to alpha-ketoisocaproate (alpha-oxoisohexanoate). The most probable explanation is that the 18(O) label is lost as a result of the enzymic deacylation of tRNA, that this process is substantially faster for Leu than for Phe, and that this represents a potentially costly futile cycle for Leu.</p><h2><a href=https://www.ncbi.nlm.nih.gov/pubmed/20231286>Incorporation of tyrosine and glutamine residues into the soluble guanylate cyclase heme distal pocket alters NO and O2 binding.</a></h2><p>Nitric oxide (NO) is the physiologically relevant activator of the mammalian hemoprotein soluble guanylate cyclase (sGC). The heme cofactor of alpha1beta1 sGC has a high affinity for NO but has never been observed to form a complex with . Introduction of a key tyrosine residue in the sGC heme binding domain beta1(1-385) is sufficient to produce an -binding protein, but this mutation in the full-length enzyme did not alter  affinity. To evaluate ligand binding specificity in full-length sGC we mutated several conserved distal heme pocket residues (beta1 Val-5, Phe-74, -145, and -149) to introduce a hydrogen bond donor in proximity to the heme ligand. We found that the NO coordination state, NO dissociation, and enzyme activation were significantly affected by the presence of a tyrosine in the distal heme pocket; however, the stability of the reduced porphyrin and the proteins affinity for  were unaltered. Recently, an atypical sGC from Drosophila, Gyc-88E, was shown to form a stable complex with . Sequence analysis of this protein identified two residues in the predicted heme pocket (tyrosine and glutamine) that may function to stabilize  binding in the atypical cyclase. The introduction of these residues into the rat beta1 distal heme pocket (-145 -->  and -149 --> Gln) resulted in an sGC construct that oxidized via an intermediate with an absorbance maximum at 417 nm. This absorbance maximum is consistent with globin Fe(II)-O(2) complexes and is likely the first observation of a Fe(II)-O(2) complex in the full-length alpha1beta1 protein. Additionally, these data suggest that atypical sGCs stabilize O(2) binding by a hydrogen bonding network involving tyrosine and glutamine.</p><h2><a href=https://www.ncbi.nlm.nih.gov/pubmed/24655798>The conserved Phe GH5 of importance for hemoglobin intersubunit contact is mutated in gadoid fish.</a></h2><p>Functionality of the tetrameric hemoglobin molecule seems to be determined by a few amino acids located in key positions. Oxygen binding encompasses structural changes at the interfaces between the α1β2 and α2β1 dimers, but also subunit interactions are important for the oxygen binding affinity and stability. The latter packing contacts include the conserved Arg B12 interacting with Phe GH5, which is replaced by Leu and  in the αA and αD chains, respectively, of birds and reptiles.Searching all known hemoglobins from a variety of gnathostome species (jawed vertebrates) revealed the almost invariant Arg B12 coded by the AGG triplet positioned at an exon-intron boundary. Rare substitutions of Arg B12 in the gnathostome β globins were found in pig, tree shrew and scaled reptiles. Phe GH5 is also highly conserved in the β globins, except for the Leu replacement in the β1 globin of five marine gadoid species, gilthead seabream and the Comoran coelacanth, while Cys and  were found in burbot and yellow croaker, respectively. Atlantic cod β1 globin showed a Leu/Met polymorphism at position GH5 dominated by the Met variant in northwest-Atlantic populations that was rarely found in northeast-Atlantic cod. Site-specific analyses identified six consensus codons under positive selection, including 122β(GH5), indicating that the amino acid changes identified at this position may offer an adaptive advantage. In fact, computational mutation analysis showed that the replacement of Phe GH5 with Leu or Cys decreased the number of van der Waals contacts essentially in the deoxy form that probably causes a slight increase in the oxygen binding affinity.The almost invariant Arg B12 and the AGG codon seem to be important for the packing contacts and pre-mRNA processing, respectively, but the rare mutations identified might be beneficial. The Leu122β1(GH5)Met and Met55β1(D6)Val polymorphisms in Atlantic cod hemoglobin modify the intradimer contacts B12-GH5 and H2-D6, while amino acid replacements at these positions in avian hemoglobin seem to be evolutionary adaptive in air-breathing vertebrates. The results support the theory that adaptive changes in hemoglobin functions are caused by a few substitutions at key positions.</p><h2><a href=https://www.ncbi.nlm.nih.gov/pubmed/11744723>Controlling ligand binding in myoglobin by mutagenesis.</a></h2><p>A quadruple mutant of sperm whale myoglobin was constructed to mimic the structure found in Ascaris suum hemoglobin. The replacements include His(E7)-->Gln, Leu(B10)-->, Thr(E10)--> Arg, and (G8)-->Phe. Single, double, and triple mutants were characterized to dissect out the effects of the individual substitutions. The crystal structures of the deoxy and oxy forms of the quadruple mutant were determined and compared with that of native Ascaris hemoglobin. (B10) myoglobin displays low O(2) affinity, high dissociation rate constants, and heterogeneous kinetic behavior, suggesting unfavorable steric interactions between the B10 phenol side chain and His(E7). In contrast, all mutants containing the (B10)/Gln(E7) pair show high O(2) affinity, low dissociation rate constants, and simple, monophasic kinetic behavior. Replacement of (107) with Phe enhances nanosecond geminate recombination singly and in combination with the (B10)/Gln(E7)/Arg(E10) mutation by limiting access to the Xe4 site. These kinetic results and comparisons with native Ascaris hemoglobin demonstrate the importance of distal pocket cavities in governing the kinetics of ligand binding. The approximately 150-fold higher O(2) affinity of Ascaris hemoglobin compared with that for (B10)/Gln(E7)-containing myoglobin mutants appears to be the result of favorable proximal effects in the Ascaris protein, due to a staggered orientation of His(F8), the lack of a hydrogen bonding lattice between the F4, F7, and F8 residues, and the presence of a large polar Trp(G5) residue in the interior portion of the proximal heme pocket.</p><h2><a href=https://www.ncbi.nlm.nih.gov/pubmed/20456866>Multiple PKCε-dependent mechanisms mediating mechanical hyperalgesia.</a></h2><p>We have recently implicated mitochondrial mechanisms in models of neuropathic and inflammatory pain, in some of which a role of protein kinase Cepsilon (PKCepsilon) has also been implicated. Since mitochondria contain several proteins that are targets of PKCepsilon, we evaluated the role of mitochondrial mechanisms in mechanical hyperalgesia induced by proinflammatory cytokines that induce PKCepsilon-dependent nociceptor sensitization, and by a direct activator of PKCepsilon (psiepsilonRACK), in the rat. Prostaglandin E(2) (PGE(2))-induced hyperalgesia is short lived in naïve rats, while it is prolonged in psiepsilonRACK pre-treated rats, a phenomenon referred to as priming. Inhibitors of two closely related mitochondrial functions, electron transport (complexes I-V) and oxidative stress (reactive  species), attenuated mechanical hyperalgesia induced by intradermal injection of psiepsilonRACK. In marked contrast, in a PKCepsilon-dependent form of mechanical hyperalgesia induced by prostaglandin E(2) (PGE(2)), inhibitors of mitochondrial function failed to attenuate hyperalgesia. These studies support the suggestion that at least two downstream signaling pathways can mediate the hyperalgesia induced by activating PKCepsilon.Copyright  2010 International Association for the Study of Pain. Published by Elsevier B.V. All rights reserved.</p></html>