<html><h2><a href=https://www.ncbi.nlm.nih.gov/pubmed/8601843>Amino acid pair interchanges at spatially conserved locations.</a></h2><p>Here we study the pattern of amino acid interchanges at spatially, locally conserved regions in globally dissimilar and unrelated proteins. By using a method which completely separates the amino acid sequence from its respective structure, this work addresses the question of which properties of the amino acids are the most crucial for the stability of conserved structural motifs. The proteins are taken from a structurally non-redundant dataset. The spatially conserved substructural motifs are defined as consisting of a "large enough" number of Calpha atoms found to provide a geometric match between two proteins, regardless of the order of the Calpha atoms in the sequence, or of the sequence composition of the substructures. This approach can apply to proteins with little or no sequence similarity but with sufficient structural similarity, and is unique in its ability to handle local, non-topological matches between pairs of dissimilar proteins. The method uses a computer-version based algorithm, the Geometric Hashing. Since the Geometric Hashing ignores sequence information it lends itself to answer the question posed above. The interchanges at geometrically similar positions that have been obtained with our method demonstrate the expected behaviour. Yet, a closer inspection reveals some distant characteristics, as compared with interchanges based upon sequence-order based techniques, or from -contact-based considerations. First, a pronounced division of the amino acids into two classes is displayed: , , Arg, Gln, Asp, , Pro, Gly, Thr, Ser and His on the one hand, and Ile, Val, Leu, Phe, Met, , Trp, Cys and Ala on the other. These groups further cluster into subgroups: , , Arg, Gln; Asp ; Pro, Gly; Ile, Val, Leu, Phe. The other amino acids stand alone. Analysis of the conservation among amino acids indicates proline to be consistently, by far, the most conserved. Next are Asp, ,  and Gly. Cys is also highly conserved. Interestingly, oppositely charged amino acids are interchanged roughly as frequently as those of the same charge. These observations can be explained in terms of the three-dimensional structures of the proteins. Most of all, there is a clear distinction between residues which prefer to be on the protein surfaces, compared to those frequently buried in the interiors. Analysis of the interchanges indicates their low information content. This, together with the separation into two groups, suggest that the predictive value of the spatial positions of the Calpha+ atoms is not much greater than the sequence alone, aside from their hydrophobicity/hydrophillicity classification.</p><p>Keyword: energy</p><h2><a href=https://www.ncbi.nlm.nih.gov/pubmed/7410430>Allophycocyanin from Nostoc sp. phycobilisomes. Properties and amino acid sequence at the NH2 terminus of the alpha and beta subunits of allophycocyanins I, II, and III.</a></h2><p>Allophycocyanin from Nostoc sp. phycobilisomes was separated into four spectrally distinct components designated allophycocyanin I, B, II, and III by adsorption chromatography on brushite columns. Allophycocyanins I and B had red-shifted fluorescence emission maxima, and on this basis, may function in transfer of excitation  from phycobilisomes to chlorophyll a. Allophycyanins II and III, which together comprise 70% of the total allophycocyanin, have absorption maxima at 648 nm and 650 nm, respectively, and probably transfer excitation  from phycocyanin to allophycocyanins I and B, in addition to serving a structural function. Allophycocyanin I was resolved into alpha, beta, and gamma subunits with apparent molecular weights of 18,000, 17,000, and 35,000, respectively, whereas allophycocyanin B was resolved into two subunits with apparent molecular weights of 16,100 and 15,300, using a modified Weber and Osborn gel electrophoresis system (Brown, A. S., and Troxler, R. F. (1977) Biochem. J. 163, 571-581). In the same gel system, allophycocyanins II and III were each resolved into alpha and beta subunits with apparent molecular weights of 18,000 and 17,000, respectively. The subunits of allophycocyanins I, II, and III were isolated by gel filtration and ion exchange chromatography and the amino acid compositions determined. Automated sequence analysis demonstrated that the first 30 amino acids at the NH2 terminus of alpha subunits, and the beta subunits, of allophycocyanins I to III were identical. alpha Subunits: Ser-Ile-Val-Thr--Ser-Ile-Val--Ala-Asp-Ala--Ala-Arg--Leu-Ser-Pro-Gly --Leu-Asp-Arg-Ile--Ser-Phe-Val-Thr- beta Subunits: Ala-Gln-Asp-Ala-Ile-Thr-Ala-Val-Ile--Ala-Ala-Asp-Val-Gln-Gly---Leu-Asp-Ala-Thr-Ala-Leu-Ser--Leu--Ala-- The gamma subunit of allophycocyanin I and both subunits of allophycocyanin B appeared to be blocked at the NH2 terminus, suggesting that the allophycocyanin B subunits may be different gene products than those of allophycocyanins I to III, or if the same, the subunits of allophycocyanin B undergo proteolytic modification after initial synthesis.</p><p>Keyword: energy</p><h2><a href=https://www.ncbi.nlm.nih.gov/pubmed/20969427>Sequence composition and environment effects on residue fluctuations in protein structures.</a></h2><p>Structure fluctuations in proteins affect a broad range of cell phenomena, including stability of proteins and their fragments, allosteric transitions, and  transfer. This study presents a statistical-thermodynamic analysis of relationship between the sequence composition and the distribution of residue fluctuations in protein-protein complexes. A one-node-per-residue elastic network model accounting for the nonhomogeneous protein mass distribution and the interatomic interactions through the renormalized inter-residue potential is developed. Two factors, a protein mass distribution and a residue environment, were found to determine the scale of residue fluctuations. Surface residues undergo larger fluctuations than core residues in agreement with experimental observations. Ranking residues over the normalized scale of fluctuations yields a distinct classification of amino acids into three groups: (i) highly fluctuating-Gly, Ala, Ser, Pro, and Asp, (ii) moderately fluctuating-Thr, , Gln, , , Arg, Val, and Cys, and (iii) weakly fluctuating-Ile, Leu, Met, Phe, , Trp, and His. The structural instability in proteins possibly relates to the high content of the highly fluctuating residues and a deficiency of the weakly fluctuating residues in irregular secondary structure elements (loops), chameleon sequences, and disordered proteins. Strong correlation between residue fluctuations and the sequence composition of protein loops supports this hypothesis. Comparing fluctuations of binding site residues (interface residues) with other surface residues shows that, on average, the interface is more rigid than the rest of the protein surface and Gly, Ala, Ser, Cys, Leu, and Trp have a propensity to form more stable docking patches on the interface. The findings have broad implications for understanding mechanisms of protein association and stability of protein structures.</p><p>Keyword: energy</p><h2><a href=https://www.ncbi.nlm.nih.gov/pubmed/9271505>Functional effects on the acetylcholine receptor of multiple mutations of gamma Asp174 and delta Asp180.</a></h2><p>Residues gamma Asp174 and delta Asp180, previously implicated in the binding of acetylcholine (ACh) by the mouse muscle ACh receptor, were each mutated to nine other residues, , , Thr, Ala, Cys, His, Val, , and . The effects of the mutations on ACh-induced current was determined on surface receptors containing wild-type alpha and beta subunits and mutant gamma and delta subunits. The mutations increased the concentration of ACh eliciting half-maximal current (EC50) by factors from 22 for the  mutant to 660 for the  mutant. Analysis of the effects in terms of the difference in the accessible surface areas of the mutant and wild-type side chains and the difference in side-chain charges indicated that, per binding site, Delta DeltaG0 for activation was a sum of 10 cal mol-1 A-2 of change in side-chain accessible surface area and of 0.95 kcal mol-1 positive step-1 in side-chain charge, equivalent to 1 mol of charge falling through 42 mV. The effects on the concentration of ACh (IC50, ACh) and of d-tubocurarine (IC50,dTC) causing half-maximal retardation of alpha-bungarotoxin binding were determined on complexes containing wild-type alpha and beta subunits and either mutant gamma or mutant delta subunit. The effects on IC50,ACh correlated well with the effects on EC50, with a similar magnitude for the influence of side-chain charge on the free  of binding (in this case to the desensitized state) and on the electrostatic potential at the binding site. The effects on IC50,dTC were in all cases less than the effects on IC50,ACh, and the two sets of effects were poorly correlated. In line with the higher ACh affinity and lower d-tubocurarine affinity of the alpha-delta binding site compared to the alpha-gamma binding site, mutations of delta Asp180 had a greater effect on IC50,ACh than did the same mutations of gamma Asp174, and vice versa for effects on IC50,dTC. Consequently, all mutations decreased the asymmetry in the binding properties of the two types of sites.</p><p>Keyword: energy</p><h2><a href=https://www.ncbi.nlm.nih.gov/pubmed/10413483>Effects of structure on alpha C-H bond enthalpies of amino acid residues: relevance to H transfers in enzyme mechanisms and in protein oxidation.</a></h2><p>The bond dissociation enthalpies (BDE) of all of the amino acid residues, modeled by HC(O)NHCH(R)C(O)NH(2) (PH(res)), were determined at the B3LYP/6-31G//B3LYP/6-31G level, coupled with isodesmic reactions. The results for neutral side chains with phi, psi angles approximately 180 degrees, approximately 180 degrees in ascending order, to an expected accuracy of +/-10 kJ mol(-)(1), are  326; cystine 330; Asp 332; Gln 334; Trp 337; Arg 340;  340; Met 343; His 344; Phe 344;  344; Leu 344; Ala 345; Cys 346; Ser 349; Gly 350; Ile 351; Val 352;  354; Thr 357; Pro-cis 358; Pro-trans 369. BDEs calculated at the ROMP2/6-31G//B3LYP/6-31G level exhibit the same trends but are approximately 7 kJ mol(-)(1) higher. All BDEs are smaller than those of typical secondary or tertiary C-H bonds due to the phenomenon of captodative stabilization. The stabilization is reduced by changes in the phi,psi angles. As a result the BDEs increase by about 10 kJ mol(-)(1) in beta-sheet and 40 kJ mol(-)(1) in alpha-helical environments, respectively. In effect the alpha C-H BDEs can be "tuned" from about 345 to 400 kJ mol(-)(1) by adjusting the local environment. Some very significant effects of this are seen in the current literature on H-transfer processes in enzyme mechanisms and in oxidative damage to proteins. These observations are discussed in terms of the findings of the present study.</p><p>Keyword: energy</p><h2><a href=https://www.ncbi.nlm.nih.gov/pubmed/10091663>Comparison of protein-protein interactions in serine protease-inhibitor and antibody-antigen complexes: implications for the protein docking problem.</a></h2><p>The protein-protein interaction  of 12 nonhomologous serine protease-inhibitor and 15 antibody-antigen complexes is calculated using a molecular mechanics formalism and dissected in terms of the main-chain vs. side-chain contribution, nonrotameric side-chain contributions, and amino acid residue type involvement in the interface interaction. There are major differences in the interactions of the two types of protein-protein complex. Protease-inhibitor complexes interact predominantly through a main-chain-main-chain mechanism while antibody-antigen complexes interact predominantly through a side-chain-side-chain or a side-chain-main-chain mechanism. However, there is no simple correlation between the main-chain-main-chain interaction  and the percentage of main-chain surface area buried on binding. The interaction  is equally effected by the presence of nonrotameric side-chain conformations, which constitute approximately 20% of the interaction . The ability to reproduce the interface interaction  of the crystal structure if original side-chain conformations are removed from the calculation is much greater in the protease-inhibitor complexes than the antibody-antigen complexes. The success of a rotameric model for protein-protein docking appears dependent on the extent of the main-chain-main-chain contribution to binding. Analysis of (1) residue type and (2) residue pair interactions at the interface show that antibody-antigen interactions are very restricted with over 70% of the antibody  attributable to just six residue types ( > Asp >  > Ser >  > Trp) in agreement with previous studies on residue propensity. However, it is found here that 50% of the antigen  is attributable to just four residue types (Arg =  >  > Asp). On average just 12 residue pair interactions (6%) contribute over 40% of the favorable interaction  in the antibody-antigen complexes, with charge-charge and charge/polar-tyrosine interactions being prominent. In contrast protease inhibitors use a diverse set of residue types and residue pair interactions.</p><p>Keyword: energy</p><h2><a href=https://www.ncbi.nlm.nih.gov/pubmed/23976829>Three dimensional (3D) structure prediction and substrate-protein interaction study of the chitin binding protein CBP24 from B. thuringiensis.</a></h2><p>Bacillus thuringiensis is an insecticidal bacterium whose chitinolytic system has been exploited to improve insect resistance in crops. In the present study, we studied the CBP24 from B. thuringiensis using homology modeling and molecular docking. The primary and secondary structure analyses showed CBP24 is a positively charged protein and contains single domain that belongs to family CBM33. The 3D model after refinement was used to explore the chitin binding characteristics of CBP24 using AUTODOCK. The docking analyses have shown that the surface exposed hydrophilic amino acid residues Thr-103, -112 and Ser-162 interact with substrate through H-bonding. While, the amino acids resides -39, -46, Ser-104 and -109 were shown to have polar interactions with the substrate. The binding  values evaluation of docking depicts a stable intermolecular conformation of the docked complex. The functional characterization of the CBP24 will elucidate the substrate-interaction pathway of the protein in specific and the carbohydrate binding proteins in general leading towards the exploration and exploitation of the prokaryotic substrate utilization pathways.</p><p>Keyword: energy</p><h2><a href=https://www.ncbi.nlm.nih.gov/pubmed/7668386>Increase in fluorescence upon the hydrolysis of tyrosine peptides: application to proteinase assays.</a></h2><p>The intrinsic fluorescence of tyrosine increases by a factor of approximately two when the carboxy group is liberated from a peptide bond by hydrolysis. The increase in fluorescence provides a novel way to monitor the hydrolysis of native tyrosine peptides that contain only proteinogenic amino acids. Thus, for example, the hydrolysis by HIV-1 proteinase of a heptapeptide viral protein fragment gag129-135, Ser-Gln---Pro-Ile-Val, was followed continuously at excitation and emission wavelengths 275 and 305 nm. The fluorescence increase is magnified by at least a factor of a thousand when a resonance  quencher, such as paranitrophenylalanine, is in the vicinity. For example, the peptide -Ala-Arg-Val--Phe(p-NO2)--Ala-Nle-NH2 [Richards et al. (1990) J. Biol. Chem. 265, 7733], widely used for spectrophotometric assays of the HIV-1 proteinase, yields a substrate:product fluorescence ratio greater than 1:1000. Tyrosine-containing substrates of pepsin and trypsin showed similar behavior. The detection limit of the present method is at least one order of magnitude lower than absorbance assays of p-nitrophenylalanine peptides.</p><p>Keyword: energy</p><h2><a href=https://www.ncbi.nlm.nih.gov/pubmed/9050852>Site-specific mutagenesis reveals differences in the structural bases for tight binding of RNase inhibitor to angiogenin and RNase A.</a></h2><p>RNase inhibitor (RI) binds with extraordinary affinity (Ki approximately 10(-13)-10(-16) M) to diverse proteins in the pancreatic RNase superfamily. In the present study, the structural basis for the recognition of two RI ligands, human angiogenin (Ang) and bovine RNase A, has been investigated by site-specific mutagenesis of human RI and Ang. The RI residues examined were those that appear to form strong contacts with RNase A in the crystal structure of the porcine RI x RNase A complex [Kobe, B. & Deisenhofer, J. (1995) Nature (London) 374, 183-186] that would not be replicated in the Ang complex. Ala substitutions of five of these residues (-287, -320, -401, Cys-408, and Arg-457) were found to have little or no effect on binding of RNase A. In contrast, replacements of -434, Asp-435, and -437 and deletion of the C-terminal residue Ser-460 substantially weakened affinity for RNase A: the losses of binding  associated with the mutations were 5.9, 3.6, 2.6, and 3.5 kcal/mol, respectively. Thus these four residues, which are neighbors in the tertiary structure, appear to constitute a "hot spot" for the RNase A interaction. However, only one of them, Asp-435, was equally important for binding of Ang; the Ki increases produced by mutations of the others were 20- to 93-fold smaller for Ang than for RNase A. Consequently, -434 plays a significant but lesser role in the Ang complex, whereas -437 and Ser-460 make only minor contributions. Ala mutations of four Ang residues (His-8, Gln-12, -68, and -108) that correspond to RI contacts on RNase A produced no major changes in affinity for RI. These findings indicate that RI uses largely different interactions to achieve its extremely tight binding of RNase A and Ang.</p><p>Keyword: energy</p><h2><a href=https://www.ncbi.nlm.nih.gov/pubmed/14691219>Effect of the N3 residue on the stability of the alpha-helix.</a></h2><p>N3 is the third position from the N terminus in the alpha-helix with helical backbone dihedral angles. All 20 amino acids have been placed in the N3 position of a synthetic helical peptide (CH(3)CO-[AAX AAAAKAAAAKAGY]-NH(2)) and the helix content measured by circular dichroism spectroscopy at 273 K. The dependence of peptide helicity on N3 residue identity has been used to determine a free  scale by analysis with a modified Lifson-Roig helix coil theory that includes a parameter for the N3  (n3). The most stabilizing residues at N3 in rank order are Ala, , Met/Ile, Leu, , Ser, Gln, Thr, , Phe, Asp, His, and Trp. Free energies for the most destabilizing residues (Cys, Gly, , Arg, and Pro) could not be fitted. The results correlate with N1, N2, and helix interior energies and not at all with N-cap preferences. This completes our work on studying the structural and energetic preferences of the amino acids for the N-terminal positions of the alpha-helix. These results can be used to rationally modify protein stability, help design helices, and improve prediction of helix location and stability.</p><p>Keyword: energy</p><h2><a href=https://www.ncbi.nlm.nih.gov/pubmed/15854661>Are acidic and basic groups in buried proteins predicted to be ionized?</a></h2><p>Ionizable residues play essential roles in proteins, modulating protein stability, fold and function. Asp, , Arg, and  make up about a quarter of the residues in an average protein. Multi-conformation continuum electrostatic (MCCE) calculations were used to predict the ionization states of all acidic and basic residues in 490 proteins. Of all 36,192 ionizable residues, 93.5% were predicted to be ionized. Thirty-five percent have lost 4.08 kcal/mol solvation  (DeltaDeltaG(rxn)) sufficient to shift a pK(a) by three pH units in the absence of other interactions and 17% have DeltaDeltaG(rxn) sufficient to shift pK(a) by five pH units. Overall 85% of these buried residues (DeltaDeltaG(rxn)>5DeltapK units) are ionized, including 92% of the Arg, 86% of the Asp, 77% of the , and 75% of the . Ion-pair interactions stabilize the ionization of both acids and bases. The backbone dipoles stabilize anions more than cations. The interactions with polar side-chains are also different for acids and bases.  and Gln stabilize all charges, Ser and Thr stabilize only acids while  rarely stabilize . Thus, hydroxyls are better hydrogen bond donors than acceptors. Buried ionized residues are more likely to be conserved than those on the surface. There are 3.95 residues buried per 100 residues in an average protein.</p><p>Keyword: energy</p><h2><a href=https://www.ncbi.nlm.nih.gov/pubmed/8289284>Determination of alpha-helix propensity within the context of a folded protein. Sites 44 and 131 in bacteriophage T4 lysozyme.</a></h2><p>To determine the effects of different amino acids on the structure and stability of an alpha-helix in the context of a globular protein, all 19 naturally-occurring amino acids were substituted for Ser44 in phage T4 lysozyme. A more restricted set of nine replacements was also made for Val131. Ser44 and Val131 are two of a very limited number of possible sites in T4 lysozyme that are well within alpha-helices, are solvent-exposed and relatively free of interactions with neighboring residues, and are not involved in crystal contacts. High resolution structures for the majority of the mutants, some of which crystallized non-isomorphously with wild-type, were determined. With the exception of proline, the amino acid substitutions caused little if any perturbation of the alpha-helix backbone. Also the beta-branched residues Thr, Val and Ile show no indication of either side-chain or backbone distortion. Therefore, other than proline, there is no evidence that differences in helix propensities are associated with different amounts of strain introduced into the helix. For reference, and also to allow estimates of side-chain entropy, a survey was made of side-chain conformations in 100 well-refined protein structures. As noted previously all side-chains within alpha-helices strongly avoid the g- conformation (chi 1 approximately 60 degrees). This restricts the beta-branched residues Thr, Val and Ile to a single conformer (g+, chi 1 approximately -60 degrees). Asp, , Met and Ser within helices also overwhelmingly prefer the g+ conformation. For Arg, Cys, Gln, , Leu and  the t (chi 1 approximately 180 degrees) and g+ conformers are populated roughly equally. Only the aromatic residues, His, , Trp and Phe prefer the t conformation. These preferences are the same whether the side-chain is buried or solvent-exposed. In general, the side-chain conformations adopted by the residues substituted at positions 44 and 131 correspond to the most commonly observed conformation for the same amino acid in helices in known protein structures. The changes in protein stability for the replacements at site 131 in general agree well with those at site 44 (correlation r = 0.97), suggesting that these may be representative of substitutions at fully solvent-exposed sites in the middle of alpha-helices. The free  values also agree quite well with those observed for equivalent replacements in a number of soluble alpha-helical model peptides and with data from "host-guest" studies and statistical surveys (r = 0.69 to 0.93).(ABSTRACT TRUNCATED AT 400 WORDS).</p><p>Keyword: energy</p><h2><a href=https://www.ncbi.nlm.nih.gov/pubmed/26733187>Molecular dynamics of sialic acid analogues complex with cholera toxin and DFT optimization of ethylene glycol-mediated zinc nanocluster conjugation.</a></h2><p>Cholera is an infectious disease caused by cholera toxin (CT) protein of bacterium Vibrio cholerae. A sequence of sialic acid (N-acetylneuraminic acid, NeuNAc or Neu5Ac) analogues modified in its C-5 position is modelled using molecular modelling techniques and docked against the CT followed by molecular dynamics simulations. Docking results suggest better binding affinity of NeuNAc analogue towards the binding site of CT. The NeuNAc analogues interact with the active site residues :11, :12, HIS:13, GLY:33, :34, :51, GLN:56, HIE:57, ILE:58, GLN:61, TRP:88, :90 and :91 through intermolecular hydrogen bonding. Analogues N-glycolyl-NeuNAc, N-Pentanoyl-NeuNAc and N-Propanoyl-NeuNAc show the least XPGscore (docking score) of -9.90, -9.16, and -8.91, respectively, and glide  of -45.99, -42.14 and -41.66 kcal/mol, respectively. Stable nature of CT-N-glycolyl-NeuNAc, CT-N-Pentanoyl-NeuNAc and CT-N-Propanoyl-NeuNAc complexes was verified through molecular dynamics simulations, each for 40 ns using the software Desmond. All the nine NeuNAc analogues show better score for drug-like properties, so could be considered as suitable candidates for drug development for cholera infection. To improve the enhanced binding mode of NeuNAc analogues towards CT, the nine NeuNAc analogues are conjugated with Zn nanoclusters through ethylene glycol (EG) as carriers. The NeuNAc analogues conjugated with EG-Zn nanoclusters show better binding  towards CT than the unconjugated nine NeuNAc analogues. The electronic structural optimization of EG-Zn nanoclusters was carried out for optimizing their performance as better delivery vehicles for NeuNAc analogues through density functional theory calculations. These sialic acid analogues may be considered as novel leads for the design of drug against cholera and the EG-Zn nanocluster may be a suitable carrier for sialic acid analogues.</p><p>Keyword: energy</p><h2><a href=https://www.ncbi.nlm.nih.gov/pubmed/31079216>Novel stable analogues of the neurotensin C-terminal hexapeptide containing unnatural amino acids.</a></h2><p>Neurotensin (NT) (pGlu-Leu-----Pro-Arg-Arg-Pro--Ile-Leu) exerts a dual function as a neurotransmitter/neuromodulator in the central nervous system and as a hormone/cellular mediator in periphery. This dual function of NT establishes a connection between brain and peripheral tissues that renders this peptide a central player in  homeostasis. Many biological actions of NT are mediated through its interaction with three types of NT receptors (NTS receptors). Despite its role in  homeostasis, NT has a short half-life that hampers further determination of the biological actions of this peptide and its receptors in brain and periphery. The short half-life of NT is due to the proteolytic degradation of its C-terminal side by several endopeptidases. Therefore, it is important to synthesize NT analogues with resistant bonds against metabolic deactivation. Based on these findings, we herein report the synthesis of ten linear, two cyclic and two dimeric analogues of NT with modifications in its structure that improve their metabolic stability, while retaining the ability to bind to NTS receptors. Modifications at position 11 (introduction of D-Tyrosine (OEthyl) [D-(Et)] or D-1-naphtylalanine [D-1-Nal] were combined with introduction of a L-Lysine or a D-Arginine at positions 8 or 9, and 1-[2-(aminophenyl)-2-oxoethyl]-1H-pyrrole-2-carboxylic acid (AOPC) at positions 7 or 8, resulting in compounds NT4-NT21. AOPC is an unnatural amino acid with promise in applications as a building block for the synthesis of peptidomimetic compounds. To biologically evaluate these analogues, we determined their plasma stability and their binding affinities to type 1 NT receptor (NTS1), endogenously expressed in HT-29 cells, Among the fourteen NT analogues, compounds, NT5, NT6, and NT8, which have D-(Et) at position 11, bound to NTS1 in a dose-response manner and with relatively high affinity but still lower than that of the natural peptide. Despite their lower binding affinities compared to NT, the NT5, NT6, and NT8 exhibited a remarkably higher stability, as a result of their chemistry, which provides protection from enzymatic activity. These results will set the basis for the rational design of novel NT molecules with improved pharmacological properties and enhanced enzymatic stability.</p><p>Keyword: energy</p><h2><a href=https://www.ncbi.nlm.nih.gov/pubmed/23603742>Mutations in Escherichia coli ExbB transmembrane domains identify scaffolding and signal transduction functions and exclude participation in a proton pathway.</a></h2><p>The TonB system couples cytoplasmic membrane proton motive force (pmf) to active transport of diverse nutrients across the outer membrane. Current data suggest that cytoplasmic membrane proteins ExbB and ExbD harness pmf . Transmembrane domain (TMD) interactions between TonB and ExbD allow the ExbD C terminus to modulate conformational rearrangements of the periplasmic TonB C terminus in vivo. These conformational changes somehow allow energization of high-affinity TonB-gated transporters by direct interaction with TonB. While ExbB is essential for  transduction, its role is not well understood. ExbB has N-terminus-out, C-terminus-in topology with three TMDs. TMDs 1 and 2 are punctuated by a cytoplasmic loop, with the C-terminal tail also occupying the cytoplasm. We tested the hypothesis that ExbB TMD residues play roles in proton translocation. Reassessment of TMD boundaries based on hydrophobic character and residue conservation among distantly related ExbB proteins brought earlier widely divergent predictions into congruence. All TMD residues with potentially function-specific side chains (, Cys, Ser, Thr, , , and ) and residues with probable structure-specific side chains (Trp, Gly, and Pro) were substituted with Ala and evaluated in multiple assays. While all three TMDs were essential, they had different roles: TMD1 was a region through which ExbB interacted with the TonB TMD. TMD2 and TMD3, the most conserved among the ExbB/TolQ/MotA/PomA family, played roles in signal transduction between cytoplasm and periplasm and the transition from ExbB homodimers to homotetramers. Consideration of combined data excludes ExbB TMD residues from direct participation in a proton pathway.</p><p>Keyword: energy</p><h2><a href=https://www.ncbi.nlm.nih.gov/pubmed/11420432>Effect of the N2 residue on the stability of the alpha-helix for all 20 amino acids.</a></h2><p>N2 is the second position in the alpha-helix. All 20 amino acids were placed in the N2 position of a synthetic helical peptide (CH(3)CO-[AXAAAAKAAAAKAAGY]-NH(2)) and the helix content was measured by circular dichroism spectroscopy at 273K. The dependence of peptide helicity on N2 residue identity has been used to determine a free- scale by analysis with a modified Lifson-Roig helix-coil theory that includes a parameter for the N2  (n2). The rank order of DeltaDeltaG((relative to Ala)) is (-), Asp(-) > Ala > (0), Leu, Val, Gln, Thr, Ile, Ser, Met, Asp(0), His(0), Arg, Cys, , Phe > , > Gly, His(+), Pro, . The results correlate very well with N2 propensities in proteins, moderately well with N1 and helix interior preferences, and not at all with N-cap preferences. The strongest energetic effects result from interactions with the helix dipole, which favors negative charges at the helix N terminus. Hydrogen bonds to side chains at N2, such as Gln, Ser, and Thr, are weak, despite occurring frequently in protein crystal structures, in contrast to the N-cap position. This is because N-cap hydrogen bonds are close to linear, whereas N2 hydrogen bonds have poor geometry. These results can be used to modify protein stability rationally, help design helices, and improve prediction of helix location and stability.</p><p>Keyword: energy</p><h2><a href=https://www.ncbi.nlm.nih.gov/pubmed/28798365>Mass & secondary structure propensity of amino acids explain their mutability and evolutionary replacements.</a></h2><p>Why is an amino acid replacement in a protein accepted during evolution? The answer given by bioinformatics relies on the frequency of change of each amino acid by another one and the propensity of each to remain unchanged. We propose that these replacement rules are recoverable from the secondary structural trends of amino acids. A distance measure between high-resolution Ramachandran distributions reveals that structurally similar residues coincide with those found in substitution matrices such as BLOSUM:  ↔ Asp, P™he ↔ ,  ↔ Arg, Gln ↔ , Ile ↔ Val, Met → Leu; with Ala, Cys, His, Gly, Ser, Pro, and Thr, as structurally idiosyncratic residues. We also found a high average correlation ([Formula: see text] = 0.85) between thirty amino acid mutability scales and the mutational inertia (I  ), which measures the energetic cost weighted by the number of observations at the most probable amino acid conformation. These results indicate that amino acid substitutions follow two optimally-efficient principles: (a) amino acids interchangeability privileges their secondary structural similarity, and (b) the amino acid mutability depends directly on its biosynthetic  cost, and inversely with its frequency. These two principles are the underlying rules governing the observed amino acid substitutions.</p><p>Keyword: energy</p></html>